<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Clarke Iakovakis" />


<title>rcrossref</title>

<script src="site_libs/header-attrs-2.9/header-attrs.js"></script>
<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/spacelab.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<script src="site_libs/htmlwidgets-1.5.3/htmlwidgets.js"></script>
<link href="site_libs/jsoneditor-5.25.6/jsoneditor.min.css" rel="stylesheet" />
<script src="site_libs/jsoneditor-5.25.6/jsoneditor.min.js"></script>
<script src="site_libs/jsonedit-binding-3.0.0/jsonedit.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>





<link rel="stylesheet" href="styles.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}

.tocify-subheader {
  display: inline;
}
.tocify-subheader .tocify-item {
  font-size: 0.95em;
}

</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Clarke Iakovakis</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    FSCI 2021 W25 Session
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="fsci_syllabus.html">Course syllabus</a>
    </li>
    <li>
      <a href="using_jupyter_notebooks.html">Jupyter Notebooks and Binder</a>
    </li>
    <li>
      <a href="IntroductionToR.html">Introduction to R</a>
    </li>
    <li>
      <a href="RBasics.html">R Basics</a>
    </li>
    <li>
      <a href="DataExploration.html">Data Exploration</a>
    </li>
    <li>
      <a href="jennybc_lists_lesson.html">Exploring Data in Lists</a>
    </li>
    <li>
      <a href="rcrossref.html">rcrossref</a>
    </li>
    <li>
      <a href="rorcid.html">rorcid</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">rcrossref</h1>
<h4 class="author">Clarke Iakovakis</h4>

</div>


<div id="licensing" class="section level1">
<h1>Licensing</h1>
<p>This walkthrough is distributed under a <a href="Creative%20Commons%20Attribution%204.0%20International%20(CC%20BY%204.0)%20License">https://creativecommons.org/licenses/by/4.0/</a>.</p>
</div>
<div id="binder-link-to-this-notebook" class="section level1">
<h1>Binder link to this notebook:</h1>
<p><a href="https://mybinder.org/v2/gh/ciakovx/ciakovx.github.io/master?filepath=rcrossref.ipynb"><img src="https://mybinder.org/badge_logo.svg" alt="Binder" /></a></p>
<p><a href="https://mybinder.org/badge_logo.svg)%5D(https://mybinder.org/v2/gh/ciakovx/ciakovx.github.io/master?filepath=rcrossref.ipynb" class="uri">https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/ciakovx/ciakovx.github.io/master?filepath=rcrossref.ipynb</a></p>
<div id="setting-up-rcrossref" class="section level2">
<h2>Setting up <code>rcrossref</code></h2>
<p>First install and load <code>rcrossref</code> in R. Also install <code>usethis</code> in order to send your email to Crossref, as well as <code>tidyverse</code>, which is a package that contains many packages we’ll use throughout.</p>
<pre class="r"><code>install.packages(&quot;rcrossref&quot;)
install.packages(&quot;usethis&quot;)
install.packages(&quot;tidyverse&quot;)
install.packages(&quot;listviewer&quot;)
library(rcrossref)
library(usethis)
library(tidyverse)
library(listviewer)</code></pre>
<p>As described in the documentation, the Crossref team encourages requests with appropriate contact information, and will forward you to a dedicated API cluster for improved performance when you share your email with them. Learn more at <a href="https://github.com/CrossRef/rest-api-doc#good-manners--more-reliable-service" class="uri">https://github.com/CrossRef/rest-api-doc#good-manners--more-reliable-service</a>. To do so, first open your R environment using the <code>edit_r_environ()</code> function from the <code>usethis</code> package.</p>
<pre class="r"><code>usethis::edit_r_environ()</code></pre>
<p>A new window will open in R Studio. Type <code>crossref_email=name@example.com</code>, replacing the example with your own email address. Then press enter to create a new line, and leave it blank. Press Ctrl/Cmd + S to save the API key to your R environment and close the window. Your email address will now be shared with Crossref whenever you send them an API request.</p>
</div>
</div>
<div id="crossref-and-rcrossref" class="section level1">
<h1>Crossref and <code>rcrossref</code></h1>
<div id="crossref" class="section level2">
<h2>Crossref</h2>
<p>Crossref is a a not-for-profit membership organization dedicated to interlinking scholarly metadata, including journals, books, conference proceedings, working papers, technical reports, data sets, authors, funders, and more. The <a href="https://github.com/CrossRef/rest-api-doc">Crossref REST API</a> allows anybody to search and reuse members’ metadata in a variety of ways. Read <a href="https://www.crossref.org/services/metadata-delivery/user-stories/">examples of user stories</a>.</p>
</div>
<div id="rcrossref" class="section level2">
<h2><code>rcrossref</code></h2>
<p><code>rcrossref</code> is a package developed by <a href="https://scottchamberlain.info/">Scott Chamberlain</a>, Hao Zhu, Najko Jahn, Carl Boettiger, and Karthik Ram, part of the <a href="https://ropensci.org/">rOpenSci</a> set of packages. rOpenSci is an incredible organization dedicated to open and reproducible research using shared data and reusable software. I strongly recommend you browse their set of packages at <a href="https://ropensci.org/packages/" class="uri">https://ropensci.org/packages/</a>.</p>
<p><code>rcrossref</code> serves as an interface to the Crossref API.</p>
<p><strong>Key links</strong></p>
<ul>
<li><a href="https://cran.r-project.org/web/packages/rcrossref/rcrossref.pdf">rcrossref documentation</a></li>
<li><a href="https://github.com/ropensci/rcrossref">Crossref REST API documentation</a></li>
<li><a href="https://github.com/CrossRef/rest-api-doc/blob/master/api_format.md">Crossref Metadata API JSON Format</a></li>
<li>Crossref and rcrossref tutorials
<ul>
<li><a href="https://ciakovx.github.io/rcrossref.html">my <code>rcrossref</code> tutorial</a></li>
<li>rOpenSci <a href="https://ropensci.org/tutorials/rcrossref_tutorial/">rcrossref tutorial</a></li>
<li>Paul Oldham: <a href="https://poldham.github.io/abs/crossref.html">“Accessing the Scientific Literature with Crossref”</a></li>
<li><a href="https://cran.r-project.org/web/packages/rcrossref/vignettes/crossref_vignette.html">rcrossref vignette</a></li>
</ul></li>
</ul>
</div>
</div>
<div id="getting-publications-from-journals-with-cr_journals" class="section level1">
<h1>Getting publications from journals with <code>cr_journals</code></h1>
<p><code>cr_journals()</code> takes either an ISSN or a general keyword query, and returns metadata for articles published in the journal, including DOI, title, volume, issue, pages, publisher, authors, etc. A full list of publications in Crossref is <a href="https://support.crossref.org/hc/en-us/articles/213197226-Browsable-title-list">available on their website</a>.</p>
<div id="getting-journal-details" class="section level2">
<h2>Getting journal details</h2>
<p>Crossref is entirely dependent on publishers to supply the metadata. Some fields are required, while others are optional. You may therefore first be interested in what metadata publishers have submitted to Crossref for a given journal. By using <code>cr_journals</code> with <code>works = FALSE</code>, you can determine who publishes the journal, the total number of articles for the journal in Crossref, whether abstracts are included, if the full text of articles is deposited, if author ORCIDs are provided, and if the publisher supplies author affiliations, author ORCID iDs, article licensing data, funders for the article, article references, and a few other items.</p>
<p>First we will create a new vector <code>plosone_issn</code> with the ISSN for the journal <em>PLoS ONE</em>.</p>
<pre class="r"><code># assign the PLoS ISSN
plosone_issn &lt;- &quot;1932-6203&quot;</code></pre>
<p>We will then run <code>rcrossref::cr_journals()</code>, setting the ISSN equal to the <code>plosone_issn</code> we just created, and print the results.</p>
<pre class="r"><code># get information about the journal
plosone_details &lt;- rcrossref::cr_journals(issn = plosone_issn, works = FALSE)
plosone_details</code></pre>
<pre><code>## $meta
## NULL
## 
## $data
##      title                 publisher      issn last_status_check_time
## 1 PLoS ONE Public Library of Science 1932-6203             2021-07-20
##   deposits_abstracts_current deposits_orcids_current deposits
## 1                       TRUE                    TRUE     TRUE
##   deposits_affiliations_backfile deposits_update_policies_backfile
## 1                          FALSE                              TRUE
##   deposits_similarity_checking_backfile deposits_award_numbers_current
## 1                                  TRUE                           TRUE
##   deposits_resource_links_current deposits_articles
## 1                           FALSE              TRUE
##   deposits_affiliations_current deposits_funders_current
## 1                         FALSE                     TRUE
##   deposits_references_backfile deposits_abstracts_backfile
## 1                         TRUE                        TRUE
##   deposits_licenses_backfile deposits_award_numbers_backfile
## 1                       TRUE                            TRUE
##   deposits_open_references_backfile deposits_open_references_current
## 1                              TRUE                             TRUE
##   deposits_references_current deposits_resource_links_backfile
## 1                        TRUE                            FALSE
##   deposits_orcids_backfile deposits_funders_backfile
## 1                     TRUE                      TRUE
##   deposits_update_policies_current deposits_similarity_checking_current
## 1                             TRUE                                 TRUE
##   deposits_licenses_current affiliations_current similarity_checking_current
## 1                      TRUE                    0                           1
##   funders_backfile licenses_backfile funders_current affiliations_backfile
## 1        0.1731985         0.9924777       0.5762672                     0
##   resource_links_backfile orcids_backfile update_policies_current
## 1                       0       0.1729574                       1
##   open_references_backfile orcids_current similarity_checking_backfile
## 1                        1      0.9389957                    0.9135426
##   references_backfile award_numbers_backfile update_policies_backfile
## 1           0.9135426              0.1471421                0.9999527
##   licenses_current award_numbers_current abstracts_backfile
## 1                1             0.4964669       3.782452e-05
##   resource_links_current abstracts_current open_references_current
## 1                      0          0.284577                       1
##   references_current total_dois current_dois backfile_dois
## 1                  1     253959        42456        211503
## 
## $facets
## NULL</code></pre>
<p>This actually comes back as a list of three items: <code>meta</code>, <code>data</code>, and <code>facets</code>. The good stuff is in <code>data</code>.</p>
<p>We use the <code>pluck()</code> function from the <code>purrr</code> package to pull that data only. We will be using <code>pluck</code> throughout this tutorial; it’s an easy way of indexing deeply and flexibly into lists to extract information.</p>
<p>We don’t have time in this tutorial to discuss list items and purr. For an excellent in-depth tutorial, see Jenny Bryan’s <a href="https://jennybc.github.io/purrr-tutorial/ls01_map-name-position-shortcuts.html">Introduction to map(): extract elements</a>, also provided on this website.</p>
<pre class="r"><code># get information about the journal and pluck the data
plosone_details &lt;- rcrossref::cr_journals(issn = plosone_issn, works = FALSE) %&gt;%
    purrr::pluck(&quot;data&quot;)</code></pre>
<p>The <code>purrr::pluck()</code> function is connected to <code>plosone_details</code> with something called a <a href="https://www.datacamp.com/community/tutorials/pipe-r-tutorial">Pipe Operator</a> <code>%&amp;gt;%</code>, which we will be using throughout the tutorial. A pipe takes the output of one statement and immediately makes it the input of the next statement. It helps so that you don’t have to write every intermediate, processing data to your R environment. You can think of it as “then” in natural language. So the above script first makes the API call with <code>cr_journals()</code>, then it applies <code>pluck()</code> to extract only the list element called <code>"data"</code>, and returns it to the <code>plosone_details</code> value.</p>
<p>We now have a <strong>data frame</strong> including the details Croassref has on file about PLoS ONE. Scroll to the right to see all the columns.</p>
<pre class="r"><code>plosone_details</code></pre>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["title"],"name":[1],"type":["chr"],"align":["left"]},{"label":["publisher"],"name":[2],"type":["chr"],"align":["left"]},{"label":["issn"],"name":[3],"type":["chr"],"align":["left"]},{"label":["last_status_check_time"],"name":[4],"type":["date"],"align":["right"]},{"label":["deposits_abstracts_current"],"name":[5],"type":["lgl"],"align":["right"]},{"label":["deposits_orcids_current"],"name":[6],"type":["lgl"],"align":["right"]},{"label":["deposits"],"name":[7],"type":["lgl"],"align":["right"]},{"label":["deposits_affiliations_backfile"],"name":[8],"type":["lgl"],"align":["right"]},{"label":["deposits_update_policies_backfile"],"name":[9],"type":["lgl"],"align":["right"]},{"label":["deposits_similarity_checking_backfile"],"name":[10],"type":["lgl"],"align":["right"]},{"label":["deposits_award_numbers_current"],"name":[11],"type":["lgl"],"align":["right"]},{"label":["deposits_resource_links_current"],"name":[12],"type":["lgl"],"align":["right"]},{"label":["deposits_articles"],"name":[13],"type":["lgl"],"align":["right"]},{"label":["deposits_affiliations_current"],"name":[14],"type":["lgl"],"align":["right"]},{"label":["deposits_funders_current"],"name":[15],"type":["lgl"],"align":["right"]},{"label":["deposits_references_backfile"],"name":[16],"type":["lgl"],"align":["right"]},{"label":["deposits_abstracts_backfile"],"name":[17],"type":["lgl"],"align":["right"]},{"label":["deposits_licenses_backfile"],"name":[18],"type":["lgl"],"align":["right"]},{"label":["deposits_award_numbers_backfile"],"name":[19],"type":["lgl"],"align":["right"]},{"label":["deposits_open_references_backfile"],"name":[20],"type":["lgl"],"align":["right"]},{"label":["deposits_open_references_current"],"name":[21],"type":["lgl"],"align":["right"]},{"label":["deposits_references_current"],"name":[22],"type":["lgl"],"align":["right"]},{"label":["deposits_resource_links_backfile"],"name":[23],"type":["lgl"],"align":["right"]},{"label":["deposits_orcids_backfile"],"name":[24],"type":["lgl"],"align":["right"]},{"label":["deposits_funders_backfile"],"name":[25],"type":["lgl"],"align":["right"]},{"label":["deposits_update_policies_current"],"name":[26],"type":["lgl"],"align":["right"]},{"label":["deposits_similarity_checking_current"],"name":[27],"type":["lgl"],"align":["right"]},{"label":["deposits_licenses_current"],"name":[28],"type":["lgl"],"align":["right"]},{"label":["affiliations_current"],"name":[29],"type":["dbl"],"align":["right"]},{"label":["similarity_checking_current"],"name":[30],"type":["dbl"],"align":["right"]},{"label":["funders_backfile"],"name":[31],"type":["dbl"],"align":["right"]},{"label":["licenses_backfile"],"name":[32],"type":["dbl"],"align":["right"]},{"label":["funders_current"],"name":[33],"type":["dbl"],"align":["right"]},{"label":["affiliations_backfile"],"name":[34],"type":["dbl"],"align":["right"]},{"label":["resource_links_backfile"],"name":[35],"type":["dbl"],"align":["right"]},{"label":["orcids_backfile"],"name":[36],"type":["dbl"],"align":["right"]},{"label":["update_policies_current"],"name":[37],"type":["dbl"],"align":["right"]},{"label":["open_references_backfile"],"name":[38],"type":["dbl"],"align":["right"]},{"label":["orcids_current"],"name":[39],"type":["dbl"],"align":["right"]},{"label":["similarity_checking_backfile"],"name":[40],"type":["dbl"],"align":["right"]},{"label":["references_backfile"],"name":[41],"type":["dbl"],"align":["right"]},{"label":["award_numbers_backfile"],"name":[42],"type":["dbl"],"align":["right"]},{"label":["update_policies_backfile"],"name":[43],"type":["dbl"],"align":["right"]},{"label":["licenses_current"],"name":[44],"type":["dbl"],"align":["right"]},{"label":["award_numbers_current"],"name":[45],"type":["dbl"],"align":["right"]},{"label":["abstracts_backfile"],"name":[46],"type":["dbl"],"align":["right"]},{"label":["resource_links_current"],"name":[47],"type":["dbl"],"align":["right"]},{"label":["abstracts_current"],"name":[48],"type":["dbl"],"align":["right"]},{"label":["open_references_current"],"name":[49],"type":["dbl"],"align":["right"]},{"label":["references_current"],"name":[50],"type":["dbl"],"align":["right"]},{"label":["total_dois"],"name":[51],"type":["int"],"align":["right"]},{"label":["current_dois"],"name":[52],"type":["int"],"align":["right"]},{"label":["backfile_dois"],"name":[53],"type":["int"],"align":["right"]}],"data":[{"1":"PLoS ONE","2":"Public Library of Science","3":"1932-6203","4":"2021-07-20","5":"TRUE","6":"TRUE","7":"TRUE","8":"FALSE","9":"TRUE","10":"TRUE","11":"TRUE","12":"FALSE","13":"TRUE","14":"FALSE","15":"TRUE","16":"TRUE","17":"TRUE","18":"TRUE","19":"TRUE","20":"TRUE","21":"TRUE","22":"TRUE","23":"FALSE","24":"TRUE","25":"TRUE","26":"TRUE","27":"TRUE","28":"TRUE","29":"0","30":"1","31":"0.1731985","32":"0.9924777","33":"0.5762672","34":"0","35":"0","36":"0.1729574","37":"1","38":"1","39":"0.9389957","40":"0.9135426","41":"0.9135426","42":"0.1471421","43":"0.9999527","44":"1","45":"0.4964669","46":"3.782452e-05","47":"0","48":"0.284577","49":"1","50":"1","51":"253959","52":"42456","53":"211503"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>There are a number of ways to explore this data frame:</p>
<pre class="r"><code># display information about the data frame
str(plosone_details)</code></pre>
<pre><code>## &#39;data.frame&#39;:    1 obs. of  53 variables:
##  $ title                                : chr &quot;PLoS ONE&quot;
##  $ publisher                            : chr &quot;Public Library of Science&quot;
##  $ issn                                 : chr &quot;1932-6203&quot;
##  $ last_status_check_time               : Date, format: &quot;2021-07-20&quot;
##  $ deposits_abstracts_current           : logi TRUE
##  $ deposits_orcids_current              : logi TRUE
##  $ deposits                             : logi TRUE
##  $ deposits_affiliations_backfile       : logi FALSE
##  $ deposits_update_policies_backfile    : logi TRUE
##  $ deposits_similarity_checking_backfile: logi TRUE
##  $ deposits_award_numbers_current       : logi TRUE
##  $ deposits_resource_links_current      : logi FALSE
##  $ deposits_articles                    : logi TRUE
##  $ deposits_affiliations_current        : logi FALSE
##  $ deposits_funders_current             : logi TRUE
##  $ deposits_references_backfile         : logi TRUE
##  $ deposits_abstracts_backfile          : logi TRUE
##  $ deposits_licenses_backfile           : logi TRUE
##  $ deposits_award_numbers_backfile      : logi TRUE
##  $ deposits_open_references_backfile    : logi TRUE
##  $ deposits_open_references_current     : logi TRUE
##  $ deposits_references_current          : logi TRUE
##  $ deposits_resource_links_backfile     : logi FALSE
##  $ deposits_orcids_backfile             : logi TRUE
##  $ deposits_funders_backfile            : logi TRUE
##  $ deposits_update_policies_current     : logi TRUE
##  $ deposits_similarity_checking_current : logi TRUE
##  $ deposits_licenses_current            : logi TRUE
##  $ affiliations_current                 : num 0
##  $ similarity_checking_current          : num 1
##  $ funders_backfile                     : num 0.173
##  $ licenses_backfile                    : num 0.992
##  $ funders_current                      : num 0.576
##  $ affiliations_backfile                : num 0
##  $ resource_links_backfile              : num 0
##  $ orcids_backfile                      : num 0.173
##  $ update_policies_current              : num 1
##  $ open_references_backfile             : num 1
##  $ orcids_current                       : num 0.939
##  $ similarity_checking_backfile         : num 0.914
##  $ references_backfile                  : num 0.914
##  $ award_numbers_backfile               : num 0.147
##  $ update_policies_backfile             : num 1
##  $ licenses_current                     : num 1
##  $ award_numbers_current                : num 0.496
##  $ abstracts_backfile                   : num 3.78e-05
##  $ resource_links_current               : num 0
##  $ abstracts_current                    : num 0.285
##  $ open_references_current              : num 1
##  $ references_current                   : num 1
##  $ total_dois                           : int 253959
##  $ current_dois                         : int 42456
##  $ backfile_dois                        : int 211503</code></pre>
<p>Type <code>?str</code> into the console to read the description of the <code>str</code> function. You can call <code>str()</code> on an R object to compactly display information about it, including the data type, the number of elements, and a printout of the first few elements.</p>
<pre class="r"><code># dimensions: 1 row, 53 columns
dim(plosone_details)</code></pre>
<pre><code>## [1]  1 53</code></pre>
<pre class="r"><code># number of rows
nrow(plosone_details)</code></pre>
<pre><code>## [1] 1</code></pre>
<pre class="r"><code># number of columns
ncol(plosone_details)</code></pre>
<pre><code>## [1] 53</code></pre>
<pre class="r"><code># column names
names(plosone_details)</code></pre>
<pre><code>##  [1] &quot;title&quot;                                
##  [2] &quot;publisher&quot;                            
##  [3] &quot;issn&quot;                                 
##  [4] &quot;last_status_check_time&quot;               
##  [5] &quot;deposits_abstracts_current&quot;           
##  [6] &quot;deposits_orcids_current&quot;              
##  [7] &quot;deposits&quot;                             
##  [8] &quot;deposits_affiliations_backfile&quot;       
##  [9] &quot;deposits_update_policies_backfile&quot;    
## [10] &quot;deposits_similarity_checking_backfile&quot;
## [11] &quot;deposits_award_numbers_current&quot;       
## [12] &quot;deposits_resource_links_current&quot;      
## [13] &quot;deposits_articles&quot;                    
## [14] &quot;deposits_affiliations_current&quot;        
## [15] &quot;deposits_funders_current&quot;             
## [16] &quot;deposits_references_backfile&quot;         
## [17] &quot;deposits_abstracts_backfile&quot;          
## [18] &quot;deposits_licenses_backfile&quot;           
## [19] &quot;deposits_award_numbers_backfile&quot;      
## [20] &quot;deposits_open_references_backfile&quot;    
## [21] &quot;deposits_open_references_current&quot;     
## [22] &quot;deposits_references_current&quot;          
## [23] &quot;deposits_resource_links_backfile&quot;     
## [24] &quot;deposits_orcids_backfile&quot;             
## [25] &quot;deposits_funders_backfile&quot;            
## [26] &quot;deposits_update_policies_current&quot;     
## [27] &quot;deposits_similarity_checking_current&quot; 
## [28] &quot;deposits_licenses_current&quot;            
## [29] &quot;affiliations_current&quot;                 
## [30] &quot;similarity_checking_current&quot;          
## [31] &quot;funders_backfile&quot;                     
## [32] &quot;licenses_backfile&quot;                    
## [33] &quot;funders_current&quot;                      
## [34] &quot;affiliations_backfile&quot;                
## [35] &quot;resource_links_backfile&quot;              
## [36] &quot;orcids_backfile&quot;                      
## [37] &quot;update_policies_current&quot;              
## [38] &quot;open_references_backfile&quot;             
## [39] &quot;orcids_current&quot;                       
## [40] &quot;similarity_checking_backfile&quot;         
## [41] &quot;references_backfile&quot;                  
## [42] &quot;award_numbers_backfile&quot;               
## [43] &quot;update_policies_backfile&quot;             
## [44] &quot;licenses_current&quot;                     
## [45] &quot;award_numbers_current&quot;                
## [46] &quot;abstracts_backfile&quot;                   
## [47] &quot;resource_links_current&quot;               
## [48] &quot;abstracts_current&quot;                    
## [49] &quot;open_references_current&quot;              
## [50] &quot;references_current&quot;                   
## [51] &quot;total_dois&quot;                           
## [52] &quot;current_dois&quot;                         
## [53] &quot;backfile_dois&quot;</code></pre>
<p>We see this data frame includes one observation of 53 different variables. This includes the total number of DOIs, whether the abstracts, orcids, article references are current; and other information.</p>
<p>You can use the $ symbol to work with particular variables. For example, the <code>publisher</code> column:</p>
<pre class="r"><code># print the publisher variable
plosone_details$publisher</code></pre>
<pre><code>## [1] &quot;Public Library of Science&quot;</code></pre>
<p>The total number of DOIs on file:</p>
<pre class="r"><code># print the total number of DOIs
plosone_details$total_dois</code></pre>
<pre><code>## [1] 253959</code></pre>
<p>Whether the data publishers provide on funders of articles they publish is current in Crossref (a TRUE/FALSE value–called “logical” in R):</p>
<pre class="r"><code># is funder data current on deposits?
plosone_details$deposits_funders_current</code></pre>
<pre><code>## [1] TRUE</code></pre>
<hr />
<p><strong>TRY IT YOURSELF</strong></p>
<ol style="list-style-type: decimal">
<li>Assign an ISSN for a well-known journal to a new variable in R. Name it whatever you like. You can use the <a href="https://www.scimagojr.com/journalrank.php">Scimago Journal Rank</a> to look up the ISSN. If you need a couple examples, try <a href="https://www.scimagojr.com/journalsearch.php?q=16004&amp;tip=sid&amp;clean=0">RUSA</a> or <a href="https://www.scimagojr.com/journalsearch.php?q=144908&amp;tip=sid&amp;clean=0">Library Hi Tech</a>. Make sure to put the ISSN in quotes to create a character vector.</li>
<li>Look up the journal details using <code>cr_journals</code>. Make sure to pass the argument <code>works = FALSE</code>.</li>
<li>Print the data to your console by typing in the value name.</li>
</ol>
<p><strong>Does it matter if the ISSN has a hyphen or not? Try both methods.</strong></p>
<pre class="r"><code># assign an ISSN to a value. Call the value what you want (e.g. plosone_issn)</code></pre>
<pre class="r"><code># look up journal details using the cr_journals function and assign it to a new
# value (e.g. plosone_details).  Remember to include a %&gt;% pipe and call
# purrr::pluck(&#39;data&#39;)</code></pre>
<pre class="r"><code># print the journal details to the console by typing in the value name</code></pre>
</div>
<div id="getting-journal-publications-by-issn" class="section level2">
<h2>Getting journal publications by ISSN</h2>
<p>To get metadata for the publications themselves rather than data about the journal, we will again use the <code>plosone_issn</code> value in the <code>issn =</code> argument to <code>cr_journals</code>, but we now set <code>works = TRUE</code>.</p>
<pre class="r"><code># get metadata on articles by setting works = TRUE
plosone_publications &lt;- cr_journals(issn = plosone_issn, works = TRUE, limit = 25) %&gt;%
    pluck(&quot;data&quot;)</code></pre>
<p>Let’s walk through this step by step:</p>
<ul>
<li>First, we are creating a new value called <code>plosone_publications</code></li>
<li>We are using the assignment operator <code>&amp;lt;-</code> to assign the results of an operation to this new value</li>
<li>We are running the function <code>cr_journals()</code>. It is not necessary to add <code>rcrossref::</code> to the beginning of the function.</li>
<li>We <strong>pass</strong> three arguments to the function:
<ul>
<li><code>issn = plosone_issn</code> : We defined <code>plosone_issn</code> earlier in the session as ‘1932-6203’. We are reusing that value here to tell the <code>cr_journals()</code> function what journal we want information on</li>
<li><code>works = TRUE</code> : When we earlier specified <code>works = FALSE</code>, we got back information on the publication. When <code>works = TRUE</code>, we will get back article level metadata</li>
<li><code>limit = 25</code> : We will get back 25 articles. The default number of articles returned is 20, but you can increase or decrease that with the <code>limit</code> argument. The max limit is 1000, but you can get more using the <code>cursor</code> argument (see below).</li>
</ul></li>
<li><code>%&amp;gt;%</code> : Pipe operator says to R to take the results of this function and use it as the input for what follows</li>
<li><code>pluck("data")</code> : This will grab only the contents of the list item “data” and return it to <code>plosone_publications</code>.</li>
</ul>
<p>Let’s explore the data frame:</p>
<pre class="r"><code># print dimensions of this data frame
dim(plosone_publications)</code></pre>
<pre><code>## [1] 25 33</code></pre>
<p>When we run <code>dim()</code> (dimensions) on this result, we now see a different number of rows and columns: 25 rows and 28 columns. This is therefore a different dataset than <code>plosone_details</code>. Let’s call <code>names()</code> to see what the column names are:</p>
<pre class="r"><code># print column names
names(plosone_publications)</code></pre>
<pre><code>##  [1] &quot;container.title&quot;        &quot;created&quot;                &quot;deposited&quot;             
##  [4] &quot;published.online&quot;       &quot;doi&quot;                    &quot;indexed&quot;               
##  [7] &quot;issn&quot;                   &quot;issue&quot;                  &quot;issued&quot;                
## [10] &quot;member&quot;                 &quot;page&quot;                   &quot;prefix&quot;                
## [13] &quot;publisher&quot;              &quot;score&quot;                  &quot;source&quot;                
## [16] &quot;reference.count&quot;        &quot;references.count&quot;       &quot;is.referenced.by.count&quot;
## [19] &quot;title&quot;                  &quot;type&quot;                   &quot;update.policy&quot;         
## [22] &quot;url&quot;                    &quot;volume&quot;                 &quot;language&quot;              
## [25] &quot;short.container.title&quot;  &quot;author&quot;                 &quot;link&quot;                  
## [28] &quot;content_domain&quot;         &quot;license&quot;                &quot;reference&quot;             
## [31] &quot;update_to&quot;              &quot;clinical-trial-number&quot;  &quot;funder&quot;</code></pre>
<p>We view the entire data frame below. Because there are some nested lists within the data, we will use the <code>select()</code> function from the <code>dplyr</code> package to select only a few columns. This will make it easier for us to view here in the Azure Notebook environment. You can also use the <code>select()</code> function to rearrange the columns.</p>
<pre class="r"><code># print select columns from the data frame
plosone_publications %&gt;%
    dplyr::select(title, doi, volume, issue, page, issued, url, publisher, reference.count,
        type, issn)</code></pre>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["title"],"name":[1],"type":["chr"],"align":["left"]},{"label":["doi"],"name":[2],"type":["chr"],"align":["left"]},{"label":["volume"],"name":[3],"type":["chr"],"align":["left"]},{"label":["issue"],"name":[4],"type":["chr"],"align":["left"]},{"label":["page"],"name":[5],"type":["chr"],"align":["left"]},{"label":["issued"],"name":[6],"type":["chr"],"align":["left"]},{"label":["url"],"name":[7],"type":["chr"],"align":["left"]},{"label":["publisher"],"name":[8],"type":["chr"],"align":["left"]},{"label":["reference.count"],"name":[9],"type":["chr"],"align":["left"]},{"label":["type"],"name":[10],"type":["chr"],"align":["left"]},{"label":["issn"],"name":[11],"type":["chr"],"align":["left"]}],"data":[{"1":"Multiple Sporadic Colorectal Cancers Display a Unique Methylation Phenotype","2":"10.1371/journal.pone.0091033","3":"9","4":"3","5":"e91033","6":"2014-03-18","7":"http://dx.doi.org/10.1371/journal.pone.0091033","8":"Public Library of Science (PLoS)","9":"42","10":"journal-article","11":"1932-6203"},{"1":"Need for Space: The Key Distance Effect Depends on Spatial Stimulus Configurations","2":"10.1371/journal.pone.0091432","3":"9","4":"3","5":"e91432","6":"2014-03-18","7":"http://dx.doi.org/10.1371/journal.pone.0091432","8":"Public Library of Science (PLoS)","9":"24","10":"journal-article","11":"1932-6203"},{"1":"Correction: MicroRNA-130b Suppresses Migration and Invasion of Colorectal Cancer Cells through Downregulation of Integrin β1","2":"10.1371/journal.pone.0091498","3":"9","4":"3","5":"e91498","6":"2014-03-18","7":"http://dx.doi.org/10.1371/journal.pone.0091498","8":"Public Library of Science (PLoS)","9":"0","10":"journal-article","11":"1932-6203"},{"1":"Oral Health Training Programs for Community and Professional Health Care Workers in Nairobi East District Increases Identification of HIV-Infected Patients","2":"10.1371/journal.pone.0090927","3":"9","4":"3","5":"e90927","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0090927","8":"Public Library of Science (PLoS)","9":"22","10":"journal-article","11":"1932-6203"},{"1":"Correction: Association of Polymorphisms and Haplotypes in the Insulin-Like Growth Factor 1 Receptor (IGF1R) Gene with the Risk of Breast Cancer in Korean Women","2":"10.1371/journal.pone.0091218","3":"9","4":"3","5":"e91218","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0091218","8":"Public Library of Science (PLoS)","9":"0","10":"journal-article","11":"1932-6203"},{"1":"Epidemiological and Etiological Characteristics of Fever, Thrombocytopenia and Leukopenia Syndrome in Henan Province, China, 2011–2012","2":"10.1371/journal.pone.0091166","3":"9","4":"3","5":"e91166","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0091166","8":"Public Library of Science (PLoS)","9":"26","10":"journal-article","11":"1932-6203"},{"1":"Correction: Network Analysis Reveals Ecological Links between N-Fixing Bacteria and Wood-Decaying Fungi","2":"10.1371/journal.pone.0091389","3":"9","4":"3","5":"e91389","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0091389","8":"Public Library of Science (PLoS)","9":"0","10":"journal-article","11":"1932-6203"},{"1":"Correction: CXCR7 Is Highly Expressed in Acute Lymphoblastic Leukemia and Potentiates CXCR4 Response to CXCL12","2":"10.1371/journal.pone.0091365","3":"9","4":"3","5":"e91365","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0091365","8":"Public Library of Science (PLoS)","9":"0","10":"journal-article","11":"1932-6203"},{"1":"Stochastic Resonance with Colored Noise for Neural Signal Detection","2":"10.1371/journal.pone.0091345","3":"9","4":"3","5":"e91345","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0091345","8":"Public Library of Science (PLoS)","9":"63","10":"journal-article","11":"1932-6203"},{"1":"Scaling and Predictability in Stock Markets: A Comparative Study","2":"10.1371/journal.pone.0091707","3":"9","4":"3","5":"e91707","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0091707","8":"Public Library of Science (PLoS)","9":"26","10":"journal-article","11":"1932-6203"},{"1":"Correction: Density, Destinations or Both? A Comparison of Measures of Walkability in Relation to Transportation Behaviors, Obesity and Diabetes in Toronto, Canada","2":"10.1371/journal.pone.0091485","3":"9","4":"3","5":"e91485","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0091485","8":"Public Library of Science (PLoS)","9":"0","10":"journal-article","11":"1932-6203"},{"1":"Molecular Characterization of Adipose Tissue in the African Elephant (Loxodonta africana)","2":"10.1371/journal.pone.0091717","3":"9","4":"3","5":"e91717","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0091717","8":"Public Library of Science (PLoS)","9":"74","10":"journal-article","11":"1932-6203"},{"1":"Immunization with H7-HCP-Tir-Intimin Significantly Reduces Colonization and Shedding of Escherichia coli O157:H7 in Goats","2":"10.1371/journal.pone.0091632","3":"9","4":"3","5":"e91632","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0091632","8":"Public Library of Science (PLoS)","9":"34","10":"journal-article","11":"1932-6203"},{"1":"Characterization of the Altered Gene Expression Profile in Early Porcine Embryos Generated from Parthenogenesis and Somatic Cell Chromatin Transfer","2":"10.1371/journal.pone.0091728","3":"9","4":"3","5":"e91728","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0091728","8":"Public Library of Science (PLoS)","9":"71","10":"journal-article","11":"1932-6203"},{"1":"Preferential Occupancy of R2 Retroelements on the B Chromosomes of the Grasshopper Eyprepocnemis plorans","2":"10.1371/journal.pone.0091820","3":"9","4":"3","5":"e91820","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0091820","8":"Public Library of Science (PLoS)","9":"81","10":"journal-article","11":"1932-6203"},{"1":"Quantitative Analysis of Cone Photoreceptor Distribution and Its Relationship with Axial Length, Age, and Early Age-Related Macular Degeneration","2":"10.1371/journal.pone.0091873","3":"9","4":"3","5":"e91873","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0091873","8":"Public Library of Science (PLoS)","9":"51","10":"journal-article","11":"1932-6203"},{"1":"Can Training Enhance Face Cognition Abilities in Middle-Aged Adults?","2":"10.1371/journal.pone.0090249","3":"9","4":"3","5":"e90249","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0090249","8":"Public Library of Science (PLoS)","9":"84","10":"journal-article","11":"1932-6203"},{"1":"A Spiking Network Model of Decision Making Employing Rewarded STDP","2":"10.1371/journal.pone.0090821","3":"9","4":"3","5":"e90821","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0090821","8":"Public Library of Science (PLoS)","9":"85","10":"journal-article","11":"1932-6203"},{"1":"Association between a C8orf13–BLK Polymorphism and Polymyositis/Dermatomyositis in the Japanese Population: An Additive Effect with STAT4 on Disease Susceptibility","2":"10.1371/journal.pone.0090019","3":"9","4":"3","5":"e90019","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0090019","8":"Public Library of Science (PLoS)","9":"28","10":"journal-article","11":"1932-6203"},{"1":"Signaling Role of Prokineticin 2 on the Estrous Cycle of Female Mice","2":"10.1371/journal.pone.0090860","3":"9","4":"3","5":"e90860","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0090860","8":"Public Library of Science (PLoS)","9":"45","10":"journal-article","11":"1932-6203"},{"1":"Correction: Morphological Characterization and Quantification of the Mycelial Growth of the Brown-Rot Fungus Postia placenta for Modeling Purposes","2":"10.1371/journal.pone.0189548","3":"13","4":"5","5":"e0189548","6":"2018-05-30","7":"http://dx.doi.org/10.1371/journal.pone.0189548","8":"Public Library of Science (PLoS)","9":"1","10":"journal-article","11":"1932-6203"},{"1":"Correction: Effects of resource-dependent cannibalism on population size distribution and individual life history in a case-bearing caddisfly","2":"10.1371/journal.pone.0197014","3":"13","4":"5","5":"e0197014","6":"2018-05-30","7":"http://dx.doi.org/10.1371/journal.pone.0197014","8":"Public Library of Science (PLoS)","9":"1","10":"journal-article","11":"1932-6203"},{"1":"Internists’ dilemmas in their interactions with chronically ill patients; A comparison of their interaction strategies and dilemmas in two different medical contexts","2":"10.1371/journal.pone.0194133","3":"13","4":"5","5":"e0194133","6":"2018-05-30","7":"http://dx.doi.org/10.1371/journal.pone.0194133","8":"Public Library of Science (PLoS)","9":"71","10":"journal-article","11":"1932-6203"},{"1":"Managing the 1920s’ Chilean educational crisis: A historical view combined with machine learning","2":"10.1371/journal.pone.0197429","3":"13","4":"5","5":"e0197429","6":"2018-05-30","7":"http://dx.doi.org/10.1371/journal.pone.0197429","8":"Public Library of Science (PLoS)","9":"42","10":"journal-article","11":"1932-6203"},{"1":"Can excreted thiocyanate be used to detect cyanide exposure in live reef fish?","2":"10.1371/journal.pone.0196841","3":"13","4":"5","5":"e0196841","6":"2018-05-30","7":"http://dx.doi.org/10.1371/journal.pone.0196841","8":"Public Library of Science (PLoS)","9":"24","10":"journal-article","11":"1932-6203"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>Here we are just getting back the last 25 articles that have been indexed in Crossref by PLoS ONE. However, this gives you a taste of how rich the metadata is. We have the dates the article was deposited and published online, the title, DOI, the ISSN, the volume, issue, and page numbers, the number of references, the URL, and for some items, the subjects. The omitted columns include information on licensing, authors, and more. We will deal with those columns further down.</p>
</div>
<div id="getting-multiple-publications-by-issn" class="section level2">
<h2>Getting multiple publications by ISSN</h2>
<p>You can also pass multiple ISSNs to <code>cr_journals</code>. Here we create 2 new values, <code>jama_issn</code> and <code>jah_issn</code>. These are ISSNs for the <em>Journal of American History</em> and <em>JAMA: The Journal of the American Medical Association</em>. We then pass them to <code>cr_journals</code> by passing them to the <code>c()</code> function, which will combine them (it’s like CONCATENATE in Excel). We set <code>works</code> to <code>TRUE</code> so we’ll get the publications metadata, and we set the <code>limit</code> to 50, so we’ll get 50 publications per journal.</p>
<pre class="r"><code># assign the JAMA and JAH ISSNs
jama_issn &lt;- &quot;1538-3598&quot;
jah_issn &lt;- &quot;0021-8723&quot;

# get the last 10 publications on deposit from each journal. For multiple
# ISSNs, use c() to combine them
jah_jama_publications &lt;- rcrossref::cr_journals(issn = c(jama_issn, jah_issn), works = T,
    limit = 10) %&gt;%
    purrr::pluck(&quot;data&quot;)</code></pre>
<p>Here we used <code>c()</code> to combine <code>jama_issn</code> and <code>jah_issn</code>. <code>c()</code> is used to create a <strong>vector</strong> in R. A vector is a sequence of elements of the same type. In this case, even though the ISSNs are numbers, we created them as <code>character</code> vectors by surrounding them in quotation marks. You can use single or double quotes. Above, when we assigned 5 to <code>y</code>, we created a <code>numeric</code> vector.</p>
<p>Vectors can only contain “homogenous” data–in other words, all data must be of the same type. The type of a vector determines what kind of analysis you can do on it. For example, you can perform mathematical operations on <code>numeric</code> objects, but not on <code>character</code> objects. You can think of vectors as columns in an Excel spreadsheet: for example, in a name column, you want every value to be a character; in a date column, you want every value to be a date; etc.</p>
<p>Going back to our <code>jah_jama_publications</code> object, we have a dataframe composed of 20 observations of 24 variables. This is a rich set of metadata for the articles in the given publications. The fields are detailed in the <a href="https://github.com/CrossRef/rest-api-doc/blob/master/api_format.md#work">Crossref documentation</a>, including the field name, type, description, and whether or not it’s required. Some of these fields are title, DOI, DOI prefix identifer, ISSN, volume, issue, publisher, abstract (if provided), reference count (if provided–i.e., the number of references <em>in</em> the given article), link (if provided), subject (if provided), and other information. The number of citations <em>to</em> the article are not pulled, but these can be gathered separately with <code>cr_citation_count()</code> (see below).</p>
<pre class="r"><code># print column names
names(jah_jama_publications)</code></pre>
<pre><code>##  [1] &quot;container.title&quot;        &quot;created&quot;                &quot;deposited&quot;             
##  [4] &quot;published.print&quot;        &quot;doi&quot;                    &quot;indexed&quot;               
##  [7] &quot;issn&quot;                   &quot;issue&quot;                  &quot;issued&quot;                
## [10] &quot;member&quot;                 &quot;page&quot;                   &quot;prefix&quot;                
## [13] &quot;publisher&quot;              &quot;score&quot;                  &quot;source&quot;                
## [16] &quot;reference.count&quot;        &quot;references.count&quot;       &quot;is.referenced.by.count&quot;
## [19] &quot;title&quot;                  &quot;type&quot;                   &quot;url&quot;                   
## [22] &quot;volume&quot;                 &quot;author&quot;                 &quot;content_domain&quot;        
## [25] &quot;language&quot;               &quot;short.container.title&quot;  &quot;link&quot;                  
## [28] &quot;subject&quot;                &quot;published.online&quot;</code></pre>
<pre class="r"><code># print data frame with select columns
jah_jama_publications %&gt;%
    dplyr::select(title, doi, volume, issue, page, issued, url, publisher, reference.count,
        type, issn)</code></pre>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["title"],"name":[1],"type":["chr"],"align":["left"]},{"label":["doi"],"name":[2],"type":["chr"],"align":["left"]},{"label":["volume"],"name":[3],"type":["chr"],"align":["left"]},{"label":["issue"],"name":[4],"type":["chr"],"align":["left"]},{"label":["page"],"name":[5],"type":["chr"],"align":["left"]},{"label":["issued"],"name":[6],"type":["chr"],"align":["left"]},{"label":["url"],"name":[7],"type":["chr"],"align":["left"]},{"label":["publisher"],"name":[8],"type":["chr"],"align":["left"]},{"label":["reference.count"],"name":[9],"type":["chr"],"align":["left"]},{"label":["type"],"name":[10],"type":["chr"],"align":["left"]},{"label":["issn"],"name":[11],"type":["chr"],"align":["left"]}],"data":[{"1":"Acetaminophen overdose in the third trimester of pregnancy","2":"10.1001/jama.247.22.3114","3":"247","4":"22","5":"3114-3115","6":"1982-06-11","7":"http://dx.doi.org/10.1001/jama.247.22.3114","8":"American Medical Association (AMA)","9":"0","10":"journal-article","11":"0098-7484,1538-3598"},{"1":"First-trimester drug use and congenital disorders","2":"10.1001/jama.246.4.343","3":"246","4":"4","5":"343-346","6":"1981-07-24","7":"http://dx.doi.org/10.1001/jama.246.4.343","8":"American Medical Association (AMA)","9":"0","10":"journal-article","11":"0098-7484,1538-3598"},{"1":"Normal serum free thyroid hormone concentrations in patients treated with phenytoin or carbamazepine. A paradox resolved","2":"10.1001/jama.275.19.1495","3":"275","4":"19","5":"1495-1498","6":"1996-05-15","7":"http://dx.doi.org/10.1001/jama.275.19.1495","8":"American Medical Association (AMA)","9":"0","10":"journal-article","11":"0098-7484,1538-3598"},{"1":"Eosinophilic Meningitis Following Myelography","2":"10.1001/jama.1980.03300490014009","3":"243","4":"23","5":"2396-2397","6":"1980-06-20","7":"http://dx.doi.org/10.1001/jama.1980.03300490014009","8":"American Medical Association (AMA)","9":"0","10":"journal-article","11":"0098-7484,1538-3598"},{"1":"Do condom catheter collecting systems cause urinary tract infection?","2":"10.1001/jama.242.4.340","3":"242","4":"4","5":"340-341","6":"1979-07-27","7":"http://dx.doi.org/10.1001/jama.242.4.340","8":"American Medical Association (AMA)","9":"0","10":"journal-article","11":"0098-7484,1538-3598"},{"1":"Ribavirin treatment of respiratory syncytial viral infection in infants with underlying cardiopulmonary disease","2":"10.1001/jama.254.21.3047","3":"254","4":"21","5":"3047-3051","6":"1985-12-06","7":"http://dx.doi.org/10.1001/jama.254.21.3047","8":"American Medical Association (AMA)","9":"0","10":"journal-article","11":"0098-7484,1538-3598"},{"1":"Molecular Epidemiology and Tuberculosis Control","2":"10.1001/jama.284.3.305","3":"284","4":"3","5":"305-307","6":"2000-07-19","7":"http://dx.doi.org/10.1001/jama.284.3.305","8":"American Medical Association (AMA)","9":"0","10":"journal-article","11":"0098-7484,1538-3598"},{"1":"Guidelines for essential trace element preparations for parenteral use. A statement by an expert panel. AMA Department of Foods and Nutrition","2":"10.1001/jama.241.19.2051","3":"241","4":"19","5":"2051-2054","6":"1979-05-11","7":"http://dx.doi.org/10.1001/jama.241.19.2051","8":"American Medical Association (AMA)","9":"0","10":"journal-article","11":"0098-7484,1538-3598"},{"1":"Severe Acute Respiratory Syndrome--Singapore, 2003","2":"10.1001/jama.289.24.3231-b","3":"289","4":"24","5":"3231-a-3234","6":"2003-06-25","7":"http://dx.doi.org/10.1001/jama.289.24.3231-b","8":"American Medical Association (AMA)","9":"0","10":"journal-article","11":"0098-7484,1538-3598"},{"1":"Recent Trends in Disability and Functioning Among Older Adults in the United States: A Systematic Review--Correction","2":"10.1001/jama.289.24.3242-a","3":"289","4":"24","5":"3242-a-3242","6":"2003-06-25","7":"http://dx.doi.org/10.1001/jama.289.24.3242-a","8":"American Medical Association (AMA)","9":"0","10":"journal-article","11":"0098-7484,1538-3598"},{"1":"Retraction","2":"10.1093/jahist/jar440","3":"98","4":"1","5":"NP-NP","6":"2011-06-01","7":"http://dx.doi.org/10.1093/jahist/jar440","8":"Oxford University Press (OUP)","9":"0","10":"journal-article","11":"0021-8723"},{"1":"Ernie Pyle's War: America's Eyewitness to World War II.","2":"10.2307/2568210","3":"84","4":"4","5":"1568","6":"1998-03","7":"http://dx.doi.org/10.2307/2568210","8":"Oxford University Press (OUP)","9":"0","10":"journal-article","11":"0021-8723"},{"1":"Mr. Polk's Army: The American Military Experience in the Mexican War.","2":"10.2307/2568147","3":"84","4":"4","5":"1510","6":"1998-03","7":"http://dx.doi.org/10.2307/2568147","8":"Oxford University Press (OUP)","9":"0","10":"journal-article","11":"0021-8723"},{"1":"Huguenot Traditions in the Mountains of Kentucky: Daniel Trabue's Memories","2":"10.2307/2568083","3":"84","4":"4","5":"1313","6":"1998-03","7":"http://dx.doi.org/10.2307/2568083","8":"Oxford University Press (OUP)","9":"0","10":"journal-article","11":"0021-8723"},{"1":"Black Jacks: African American Seamen in the Age of Sail.","2":"10.2307/2568136","3":"84","4":"4","5":"1500","6":"1998-03","7":"http://dx.doi.org/10.2307/2568136","8":"Oxford University Press (OUP)","9":"0","10":"journal-article","11":"0021-8723"},{"1":"Guardians of Empire: The U.S. Army and the Pacific, 1902-1940.","2":"10.2307/2568161","3":"84","4":"4","5":"1522","6":"1998-03","7":"http://dx.doi.org/10.2307/2568161","8":"Oxford University Press (OUP)","9":"0","10":"journal-article","11":"0021-8723"},{"1":"A Cautious Patriotism: The American Churches & the Second World War.","2":"10.2307/2568206","3":"84","4":"4","5":"1564","6":"1998-03","7":"http://dx.doi.org/10.2307/2568206","8":"Oxford University Press (OUP)","9":"0","10":"journal-article","11":"0021-8723"},{"1":"Designing Modern America: The Regional Planning Association of America and Its Members.","2":"10.2307/2568177","3":"84","4":"4","5":"1536","6":"1998-03","7":"http://dx.doi.org/10.2307/2568177","8":"Oxford University Press (OUP)","9":"0","10":"journal-article","11":"0021-8723"},{"1":"Trade and Gunboats: The United States and Brazil in the Age of Empire.","2":"10.2307/2568160","3":"84","4":"4","5":"1521","6":"1998-03","7":"http://dx.doi.org/10.2307/2568160","8":"Oxford University Press (OUP)","9":"0","10":"journal-article","11":"0021-8723"},{"1":"What Do We Need a Union For? The TWUA in the South, 1945-1955.","2":"10.2307/2568203","3":"84","4":"4","5":"1561","6":"1998-03","7":"http://dx.doi.org/10.2307/2568203","8":"Oxford University Press (OUP)","9":"0","10":"journal-article","11":"0021-8723"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="filtering-the-cr_journals-query-with-the-filter-argument" class="section level2">
<h2>Filtering the <code>cr_journals</code> query with the <code>filter</code> argument</h2>
<p>You can use the <code>filter</code> argument within <code>cr_journals</code> to specify some parameters as the query is executing. This filter is built into the Crossref API query. See the available filters by calling <code>rcrossref::filter_names()</code>, and details by calling <code>rcrossref::filter_details</code>. It’s also in <a href="https://github.com/CrossRef/rest-api-doc#filter-names">the API documentation</a>.</p>
<table class="table table-striped table-hover table-condensed table-responsive" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
x
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
award_funder
</td>
</tr>
<tr>
<td style="text-align:left;">
award_number
</td>
</tr>
<tr>
<td style="text-align:left;">
content_domain
</td>
</tr>
<tr>
<td style="text-align:left;">
from_accepted_date
</td>
</tr>
<tr>
<td style="text-align:left;">
from_online_pub_date
</td>
</tr>
<tr>
<td style="text-align:left;">
from_posted_date
</td>
</tr>
<tr>
<td style="text-align:left;">
from_print_pub_date
</td>
</tr>
<tr>
<td style="text-align:left;">
has_assertion
</td>
</tr>
<tr>
<td style="text-align:left;">
has_authenticated_orcid
</td>
</tr>
<tr>
<td style="text-align:left;">
has_crossmark_restriction
</td>
</tr>
<tr>
<td style="text-align:left;">
has_relation
</td>
</tr>
<tr>
<td style="text-align:left;">
location
</td>
</tr>
<tr>
<td style="text-align:left;">
relation_object
</td>
</tr>
<tr>
<td style="text-align:left;">
relation_object_type
</td>
</tr>
<tr>
<td style="text-align:left;">
relation_type
</td>
</tr>
<tr>
<td style="text-align:left;">
until_accepted_date
</td>
</tr>
<tr>
<td style="text-align:left;">
until_online_pub_date
</td>
</tr>
<tr>
<td style="text-align:left;">
until_posted_date
</td>
</tr>
<tr>
<td style="text-align:left;">
until_print_pub_date
</td>
</tr>
<tr>
<td style="text-align:left;">
has_funder
</td>
</tr>
<tr>
<td style="text-align:left;">
funder
</td>
</tr>
<tr>
<td style="text-align:left;">
prefix
</td>
</tr>
<tr>
<td style="text-align:left;">
member
</td>
</tr>
<tr>
<td style="text-align:left;">
from_index_date
</td>
</tr>
<tr>
<td style="text-align:left;">
until_index_date
</td>
</tr>
<tr>
<td style="text-align:left;">
from_deposit_date
</td>
</tr>
<tr>
<td style="text-align:left;">
until_deposit_date
</td>
</tr>
<tr>
<td style="text-align:left;">
from_update_date
</td>
</tr>
<tr>
<td style="text-align:left;">
until_update_date
</td>
</tr>
<tr>
<td style="text-align:left;">
from_pub_date
</td>
</tr>
<tr>
<td style="text-align:left;">
until_pub_date
</td>
</tr>
<tr>
<td style="text-align:left;">
has_license
</td>
</tr>
<tr>
<td style="text-align:left;">
license_url
</td>
</tr>
<tr>
<td style="text-align:left;">
license_version
</td>
</tr>
<tr>
<td style="text-align:left;">
license_delay
</td>
</tr>
<tr>
<td style="text-align:left;">
has_full_text
</td>
</tr>
<tr>
<td style="text-align:left;">
full_text_version
</td>
</tr>
<tr>
<td style="text-align:left;">
full_text_type
</td>
</tr>
<tr>
<td style="text-align:left;">
full_text_application
</td>
</tr>
<tr>
<td style="text-align:left;">
has_references
</td>
</tr>
<tr>
<td style="text-align:left;">
reference_visibility
</td>
</tr>
<tr>
<td style="text-align:left;">
has_archive
</td>
</tr>
<tr>
<td style="text-align:left;">
archive
</td>
</tr>
<tr>
<td style="text-align:left;">
has_orcid
</td>
</tr>
<tr>
<td style="text-align:left;">
orcid
</td>
</tr>
<tr>
<td style="text-align:left;">
issn
</td>
</tr>
<tr>
<td style="text-align:left;">
isbn
</td>
</tr>
<tr>
<td style="text-align:left;">
type
</td>
</tr>
<tr>
<td style="text-align:left;">
directory
</td>
</tr>
<tr>
<td style="text-align:left;">
doi
</td>
</tr>
<tr>
<td style="text-align:left;">
updates
</td>
</tr>
<tr>
<td style="text-align:left;">
is_update
</td>
</tr>
<tr>
<td style="text-align:left;">
has_update_policy
</td>
</tr>
<tr>
<td style="text-align:left;">
container_title
</td>
</tr>
<tr>
<td style="text-align:left;">
category_name
</td>
</tr>
<tr>
<td style="text-align:left;">
type_name
</td>
</tr>
<tr>
<td style="text-align:left;">
from_created_date
</td>
</tr>
<tr>
<td style="text-align:left;">
until_created_date
</td>
</tr>
<tr>
<td style="text-align:left;">
has_affiliation
</td>
</tr>
<tr>
<td style="text-align:left;">
assertion_group
</td>
</tr>
<tr>
<td style="text-align:left;">
assertion
</td>
</tr>
<tr>
<td style="text-align:left;">
article_number
</td>
</tr>
<tr>
<td style="text-align:left;">
alternative_id
</td>
</tr>
<tr>
<td style="text-align:left;">
has_clinical_trial_number
</td>
</tr>
<tr>
<td style="text-align:left;">
has_abstract
</td>
</tr>
<tr>
<td style="text-align:left;">
has_content_domain
</td>
</tr>
<tr>
<td style="text-align:left;">
has_domain_restriction
</td>
</tr>
</tbody>
</table>
<div id="filtering-by-publication-date-with-from_pub_date-and-until_pub_date" class="section level3">
<h3>Filtering by publication date with <code>from_pub_date</code> and <code>until_pub_date</code></h3>
<p>For example, you may only want to pull publications from a given year, or within a date range. Remember to increase the limit or use <code>cursor</code> if you need to. Also notice three things about the <code>filter</code> argument:</p>
<ul>
<li>The query parameter is in backticks (the key next to the 1 on the keyboard)</li>
<li>The query itself is in single quotes</li>
<li>The whole thing is wrapped in <code>c()</code></li>
</ul>
<p>Here, we will get all articles from the <em>Journal of Librarianship and Scholarly Communication</em> published after January 1, 2019:</p>
<pre class="r"><code># assign the JLSC ISSN
jlsc_issn &lt;- &quot;2162-3309&quot;

# get articles published since January 1, 2019
jlsc_publications_2019 &lt;- rcrossref::cr_journals(issn = jlsc_issn, works = T, filter = c(from_pub_date = &quot;2019-01-01&quot;)) %&gt;%
    purrr::pluck(&quot;data&quot;)

# print the dataframe with select column
jlsc_publications_2019 %&gt;%
    dplyr::select(title, doi, volume, issue, issued, url, publisher, reference.count,
        type, issn)</code></pre>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["title"],"name":[1],"type":["chr"],"align":["left"]},{"label":["doi"],"name":[2],"type":["chr"],"align":["left"]},{"label":["volume"],"name":[3],"type":["chr"],"align":["left"]},{"label":["issue"],"name":[4],"type":["chr"],"align":["left"]},{"label":["issued"],"name":[5],"type":["chr"],"align":["left"]},{"label":["url"],"name":[6],"type":["chr"],"align":["left"]},{"label":["publisher"],"name":[7],"type":["chr"],"align":["left"]},{"label":["reference.count"],"name":[8],"type":["chr"],"align":["left"]},{"label":["type"],"name":[9],"type":["chr"],"align":["left"]},{"label":["issn"],"name":[10],"type":["chr"],"align":["left"]}],"data":[{"1":"Review of Creative Commons for Educators and Librarians","2":"10.31274/jlsc.12904","3":"9","4":"1","5":"2021-07-16","6":"http://dx.doi.org/10.31274/jlsc.12904","7":"Iowa State University","8":"0","9":"journal-article","10":"2162-3309"},{"1":"The Changing Landscape of Open Access Publishing: Can Open Access Publishing Make the Scholarly World More Equitable and Productive?","2":"10.7710/2162-3309.2345","3":"9","4":"1","5":"2021-02-21","6":"http://dx.doi.org/10.7710/2162-3309.2345","7":"Iowa State University","8":"0","9":"journal-article","10":"2162-3309"},{"1":"Knowledge and Practice changes following a student data focused data management education program.","2":"10.31274/jlsc.12906","3":"NA","4":"NA","5":"2021-07-07","6":"http://dx.doi.org/10.31274/jlsc.12906","7":"Iowa State University","8":"0","9":"journal-article","10":"2162-3309"},{"1":"Using Law School Faculty Author Profiles to Promote Impact: The U.S. News & World Report Saga Continues","2":"10.7710/2162-3309.2386","3":"8","4":"1","5":"2020-12-14","6":"http://dx.doi.org/10.7710/2162-3309.2386","7":"Iowa State University","8":"0","9":"journal-article","10":"2162-3309"},{"1":"Creating a Library Publishing Program: Your Options Are Limited","2":"10.7710/2162-3309.2262","3":"7","4":"1","5":"2019-01-10","6":"http://dx.doi.org/10.7710/2162-3309.2262","7":"Iowa State University","8":"0","9":"journal-article","10":"2162-3309"},{"1":"Bibliometrics and Research Evaluation: Uses and Abuses","2":"10.7710/2162-3309.2286","3":"7","4":"1","5":"2019-01-10","6":"http://dx.doi.org/10.7710/2162-3309.2286","7":"Iowa State University","8":"0","9":"journal-article","10":"2162-3309"},{"1":"Meaningful Metrics: A 21st Century Librarian's Guide to Bibliometrics, Altmetrics, and Research Impact [Book Review]","2":"10.7710/2162-3309.2290","3":"7","4":"1","5":"2019-01-15","6":"http://dx.doi.org/10.7710/2162-3309.2290","7":"Iowa State University","8":"0","9":"journal-article","10":"2162-3309"},{"1":"Online Safety and Academic Scholarship: Exploring Researchers’ Concerns from Ghana","2":"10.7710/2162-3309.2263","3":"7","4":"1","5":"2019-01-15","6":"http://dx.doi.org/10.7710/2162-3309.2263","7":"Iowa State University","8":"0","9":"journal-article","10":"2162-3309"},{"1":"When a Repository Is Not Enough: Redesigning a Digital Ecosystem to Serve Scholarly Communication","2":"10.7710/2162-3309.2225","3":"7","4":"1","5":"2019-02-26","6":"http://dx.doi.org/10.7710/2162-3309.2225","7":"Iowa State University","8":"0","9":"journal-article","10":"2162-3309"},{"1":"Strategies for Supporting OER Adoption through Faculty and Instructor Use of a Federated Search Tool","2":"10.7710/2162-3309.2279","3":"7","4":"1","5":"2019-02-22","6":"http://dx.doi.org/10.7710/2162-3309.2279","7":"Iowa State University","8":"0","9":"journal-article","10":"2162-3309"},{"1":"Opening Up Open Access Institutional Repositories to Demonstrate Value: Two Universities’ Pilots on Including Metadata-Only Records","2":"10.7710/2162-3309.2220","3":"7","4":"1","5":"2019-03-08","6":"http://dx.doi.org/10.7710/2162-3309.2220","7":"Iowa State University","8":"0","9":"journal-article","10":"2162-3309"},{"1":"Course Review: CopyrightX","2":"10.7710/2162-3309.2297","3":"7","4":"1","5":"2019-04-11","6":"http://dx.doi.org/10.7710/2162-3309.2297","7":"Iowa State University","8":"0","9":"journal-article","10":"2162-3309"},{"1":"Dynamic Research Support for Academic Libraries [Book Review]","2":"10.7710/2162-3309.2307","3":"7","4":"1","5":"2019-04-15","6":"http://dx.doi.org/10.7710/2162-3309.2307","7":"Iowa State University","8":"0","9":"journal-article","10":"2162-3309"},{"1":"Whose Research is it Anyway? Academic Social Networks Versus Institutional Repositories","2":"10.7710/2162-3309.2243","3":"7","4":"1","5":"2019-04-25","6":"http://dx.doi.org/10.7710/2162-3309.2243","7":"Iowa State University","8":"0","9":"journal-article","10":"2162-3309"},{"1":"Library Publishers as Educators: Crafting Curriculum for Undergraduate Research Journals","2":"10.7710/2162-3309.2296","3":"7","4":"1","5":"2019-05-06","6":"http://dx.doi.org/10.7710/2162-3309.2296","7":"Iowa State University","8":"0","9":"journal-article","10":"2162-3309"},{"1":"Avoiding the “Axe”: Advancing Affordable and Open Education Resources at a Midsize University","2":"10.7710/2162-3309.2259","3":"7","4":"1","5":"2019-05-20","6":"http://dx.doi.org/10.7710/2162-3309.2259","7":"Iowa State University","8":"0","9":"journal-article","10":"2162-3309"},{"1":"The Development of the Journal Evaluation Tool to Evaluate the Credibility of Publication Venues","2":"10.7710/2162-3309.2250","3":"7","4":"1","5":"2019-06-04","6":"http://dx.doi.org/10.7710/2162-3309.2250","7":"Iowa State University","8":"0","9":"journal-article","10":"2162-3309"},{"1":"Embracing New Trends in Scholarly Communication: From Competency Requirements in the Workplace to LIS Curriculum Presence","2":"10.7710/2162-3309.2291","3":"7","4":"1","5":"2019-07-19","6":"http://dx.doi.org/10.7710/2162-3309.2291","7":"Iowa State University","8":"0","9":"journal-article","10":"2162-3309"},{"1":"The Five Laws of OER: Observations from Ranganathan","2":"10.7710/2162-3309.2299","3":"7","4":"1","5":"2019-08-20","6":"http://dx.doi.org/10.7710/2162-3309.2299","7":"Iowa State University","8":"0","9":"journal-article","10":"2162-3309"},{"1":"We Can Do I.T.: Women in Library Information Technology","2":"10.7710/2162-3309.2320","3":"7","4":"1","5":"2019-09-16","6":"http://dx.doi.org/10.7710/2162-3309.2320","7":"Iowa State University","8":"0","9":"journal-article","10":"2162-3309"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="filtering-by-funder-with-award.funder" class="section level3">
<h3>Filtering by funder with <code>award.funder</code></h3>
<p>You can also return all articles funded by a specific funder. See the <a href="https://gitlab.com/crossref/open_funder_registry">Crossref Funder Registry</a> for a list of funders and their DOIs.</p>
<p>Here, we will combine two filters: <code>award.funder</code> and <code>from_pub_date</code> to return all articles published in PLoS ONE where a) at least one funder is the National Institutes of Health, and b) the article was published after March 1, 2020. Note that we set a <code>limit</code> here of 25 because we are doing a teaching activity and we don’t want to send heavy queries. If you were doing this on your own, you would likely want to remove the limit.</p>
<pre class="r"><code># assign the PLoS ONE ISSN and the NIH Funder DOI
plosone_issn &lt;- &quot;1932-6203&quot;
nih_funder_doi &lt;- &quot;10.13039/100000002&quot;

# get articles published in PLoS since 3/1 funded by NIH
plosone_publications_nih &lt;- rcrossref::cr_journals(issn = plosone_issn, works = T,
    limit = 25, filter = c(award.funder = nih_funder_doi, from_pub_date = &quot;2020-03-01&quot;)) %&gt;%
    purrr::pluck(&quot;data&quot;)</code></pre>
<p>We will use <code>unnest()</code> from the <code>tidyr</code> package to view the data frame here. This is described below in <a href="https://rcrossref2-clarkeiakovakis.notebooks.azure.com/j/notebooks/rcrossref_20200305.ipynb#Unnesting-list-columns">Unnesting List Columns</a>.</p>
<pre class="r"><code># print the dataframe, first unnesting the funder column
plosone_publications_nih %&gt;%
    tidyr::unnest(funder)</code></pre>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["container.title"],"name":[1],"type":["chr"],"align":["left"]},{"label":["created"],"name":[2],"type":["chr"],"align":["left"]},{"label":["deposited"],"name":[3],"type":["chr"],"align":["left"]},{"label":["published.online"],"name":[4],"type":["chr"],"align":["left"]},{"label":["doi"],"name":[5],"type":["chr"],"align":["left"]},{"label":["indexed"],"name":[6],"type":["chr"],"align":["left"]},{"label":["issn"],"name":[7],"type":["chr"],"align":["left"]},{"label":["issue"],"name":[8],"type":["chr"],"align":["left"]},{"label":["issued"],"name":[9],"type":["chr"],"align":["left"]},{"label":["member"],"name":[10],"type":["chr"],"align":["left"]},{"label":["page"],"name":[11],"type":["chr"],"align":["left"]},{"label":["prefix"],"name":[12],"type":["chr"],"align":["left"]},{"label":["publisher"],"name":[13],"type":["chr"],"align":["left"]},{"label":["score"],"name":[14],"type":["chr"],"align":["left"]},{"label":["source"],"name":[15],"type":["chr"],"align":["left"]},{"label":["reference.count"],"name":[16],"type":["chr"],"align":["left"]},{"label":["references.count"],"name":[17],"type":["chr"],"align":["left"]},{"label":["is.referenced.by.count"],"name":[18],"type":["chr"],"align":["left"]},{"label":["subject"],"name":[19],"type":["chr"],"align":["left"]},{"label":["title"],"name":[20],"type":["chr"],"align":["left"]},{"label":["type"],"name":[21],"type":["chr"],"align":["left"]},{"label":["update.policy"],"name":[22],"type":["chr"],"align":["left"]},{"label":["url"],"name":[23],"type":["chr"],"align":["left"]},{"label":["volume"],"name":[24],"type":["chr"],"align":["left"]},{"label":["language"],"name":[25],"type":["chr"],"align":["left"]},{"label":["short.container.title"],"name":[26],"type":["chr"],"align":["left"]},{"label":["author"],"name":[27],"type":["list"],"align":["right"]},{"label":["DOI"],"name":[28],"type":["chr"],"align":["left"]},{"label":["name"],"name":[29],"type":["chr"],"align":["left"]},{"label":["doi.asserted.by"],"name":[30],"type":["chr"],"align":["left"]},{"label":["award"],"name":[31],"type":["chr"],"align":["left"]},{"label":["award1"],"name":[32],"type":["chr"],"align":["left"]},{"label":["award2"],"name":[33],"type":["chr"],"align":["left"]},{"label":["award3"],"name":[34],"type":["chr"],"align":["left"]},{"label":["link"],"name":[35],"type":["list"],"align":["right"]},{"label":["content_domain"],"name":[36],"type":["list"],"align":["right"]},{"label":["license"],"name":[37],"type":["list"],"align":["right"]},{"label":["reference"],"name":[38],"type":["list"],"align":["right"]},{"label":["abstract"],"name":[39],"type":["chr"],"align":["left"]}],"data":[{"1":"PLOS ONE","2":"2020-08-26","3":"2020-08-26","4":"2020-08-26","5":"10.1371/journal.pone.0238172","6":"2020-08-26","7":"1932-6203","8":"8","9":"2020-08-26","10":"340","11":"e0238172","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"30","17":"30","18":"0","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"A simple predictive model for estimating relative e-cigarette toxic carbonyl levels","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0238172","24":"15","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000002","29":"National Institutes of Health","30":"publisher","31":"R01ES025257","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,11]>","39":"NA"},{"1":"PLOS ONE","2":"2020-12-22","3":"2020-12-22","4":"2020-12-22","5":"10.1371/journal.pone.0238552","6":"2021-06-23","7":"1932-6203","8":"12","9":"2020-12-22","10":"340","11":"e0238552","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"37","17":"37","18":"5","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Obesity and smoking as risk factors for invasive mechanical ventilation in COVID-19: A retrospective, observational cohort study","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0238552","24":"15","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000002","29":"National Institutes of Health","30":"publisher","31":"T32 5T32HL072752-1","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,12]>","39":"<jats:sec id=\"sec001\">\\n<jats:title>Purpose<\/jats:title>\\n<jats:p>To describe the trajectory of respiratory failure in COVID-19 and explore factors associated with risk of invasive mechanical ventilation (IMV).<\/jats:p>\\n<\/jats:sec>\\n<jats:sec id=\"sec002\">\\n<jats:title>Materials and methods<\/jats:title>\\n<jats:p>A retrospective, observational cohort study of 112 inpatient adults diagnosed with COVID-19 between March 12 and April 16, 2020. Data were manually extracted from electronic medical records. Multivariable and Univariable regression were used to evaluate association between baseline characteristics, initial serum markers and the outcome of IMV.<\/jats:p>\\n<\/jats:sec>\\n<jats:sec id=\"sec003\">\\n<jats:title>Results<\/jats:title>\\n<jats:p>Our cohort had median age of 61 (IQR 45–74) and was 66% male. In-hospital mortality was 6% (7/112). ICU mortality was 12.8% (6/47), and 18% (5/28) for those requiring IMV. Obesity (OR 5.82, CI 1.74–19.48), former (OR 8.06, CI 1.51–43.06) and current smoking status (OR 10.33, CI 1.43–74.67) were associated with IMV after adjusting for age, sex, and high prevalence comorbidities by multivariable analysis. Initial absolute lymphocyte count (OR 0.33, CI 0.11–0.96), procalcitonin (OR 1.27, CI 1.02–1.57), IL-6 (OR 1.17, CI 1.03–1.33), ferritin (OR 1.05, CI 1.005–1.11), LDH (OR 1.57, 95% CI 1.13–2.17) and CRP (OR 1.13, CI 1.06–1.21), were associated with IMV by univariate analysis.<\/jats:p>\\n<\/jats:sec>\\n<jats:sec id=\"sec004\">\\n<jats:title>Conclusions<\/jats:title>\\n<jats:p>Obesity, smoking history, and elevated inflammatory markers were associated with increased need for IMV in patients with COVID-19.<\/jats:p>\\n<\/jats:sec>"},{"1":"PLOS ONE","2":"2021-01-06","3":"2021-01-06","4":"2021-01-06","5":"10.1371/journal.pone.0244747","6":"2021-06-23","7":"1932-6203","8":"1","9":"2021-01-06","10":"340","11":"e0244747","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"60","17":"60","18":"1","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Using Fitbit data to examine factors that affect daily activity levels of college students","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0244747","24":"16","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000002","29":"National Institutes of Health","30":"publisher","31":"#1 R01 HL117757-01A1","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,11]>","39":"<jats:p>To date, the effect of both fixed and time-varying individual, social, psychological, environmental, and behavioral characteristics on temporal growth trends in physical activity (PA) among younger individuals remains an under-studied topic. In this paper, we address this gap in previous work by examining how temporal growth trends in PA respond to changing social, environmental, and behavioral characteristics using a large sample of college students (N = 692) who participated in the NetHealth project at the University of Notre Dame and from which fine-grained longitudinal data on physical activity and social interaction were collected unobtrusively via the use of wearables for 637 days (August 16, 2015 to May 13, 2017). These data are augmented by periodic survey data on fixed sociodemographic and psychological variables. We estimate latent growth-curve models for daily activity status, steps, active minutes, and activity calories. We find evidence of both a generalized friendship paradox and a peer effect for PA, with the average PA level of study participants’ contacts being on average larger than their own, and with this average level exerting a statistically significant effect on individual PA levels. Notably, there was limited evidence of temporal growth in PA across the 637 days of observation with null temporal effects for three out of the four PA indicators, except for daily steps taken. Finally, we find that social, psychological, and behavioral factors (e.g., large network size, high extroversion levels, and more courses taken) are systematically associated with higher PA levels in this sample. Overall, our findings highlight the importance of social, environmental, and behavioral factors (such as peer networks and daily sociability) in modulating the dynamics of PA levels among college students.<\/jats:p>"},{"1":"PLOS ONE","2":"2020-09-17","3":"2020-09-17","4":"2020-09-17","5":"10.1371/journal.pone.0239175","6":"2021-06-01","7":"1932-6203","8":"9","9":"2020-09-17","10":"340","11":"e0239175","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"37","17":"37","18":"4","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"COVID-19 pandemic and Farr’s law: A global comparison and prediction of outbreak acceleration and deceleration rates","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0239175","24":"15","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000002","29":"National Institutes of Health","30":"crossref","31":"R01 AT009491-01A1","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,12]>","39":"NA"},{"1":"PLOS ONE","2":"2021-07-01","3":"2021-07-01","4":"2021-07-01","5":"10.1371/journal.pone.0251342","6":"2021-07-02","7":"1932-6203","8":"7","9":"2021-07-01","10":"340","11":"e0251342","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"17","17":"17","18":"0","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Development and validation of a quantitative, non-invasive, highly sensitive and specific, electrochemical assay for anti-SARS-CoV-2 IgG antibodies in saliva","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0251342","24":"16","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000002","29":"National Institutes of Health","30":"publisher","31":"NA","32":"F30DE27615","33":"U54L119893","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,11]>","39":"<jats:p>Amperial™ is a novel assay platform that uses immobilized antigen in a conducting polymer gel followed by detection via electrochemical measurement of oxidation-reduction reaction between H<jats:sub>2<\/jats:sub>O<jats:sub>2<\/jats:sub>/Tetrametylbenzidine and peroxidase enzyme in a completed assay complex. A highly specific and sensitive assay was developed to quantify levels of IgG antibodies to SARS-CoV-2 in saliva. After establishing linearity and limit of detection we established a reference range of 5 standard deviations above the mean. There were no false positives in 667 consecutive saliva samples obtained prior to 2019. Saliva was obtained from 34 patients who had recovered from documented COVID-19 or had documented positive serologies. All of the patients with symptoms severe enough to seek medical attention had positive antibody tests and 88% overall had positive results. We obtained blinded paired saliva and plasma samples from 14 individuals. The plasma was analyzed using an EUA-FDA cleared ELISA kit and the saliva was analyzed by our Amperial™ assay. All 5 samples with negative plasma titers were negative in saliva testing. Eight of the 9 positive plasma samples were positive in saliva and 1 had borderline results. A CLIA validation was performed as a laboratory developed test in a high complexity laboratory. A quantitative non-invasive saliva based SARS-CoV-2 antibody test was developed and validated with sufficient specificity to be useful for population-based monitoring and monitoring of individuals following vaccination.<\/jats:p>"},{"1":"PLOS ONE","2":"2021-07-01","3":"2021-07-01","4":"2021-07-01","5":"10.1371/journal.pone.0251342","6":"2021-07-02","7":"1932-6203","8":"7","9":"2021-07-01","10":"340","11":"e0251342","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"17","17":"17","18":"0","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Development and validation of a quantitative, non-invasive, highly sensitive and specific, electrochemical assay for anti-SARS-CoV-2 IgG antibodies in saliva","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0251342","24":"16","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"NA","29":"Liquid Diagnostics, LLC","30":"NA","31":"NA","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,11]>","39":"<jats:p>Amperial™ is a novel assay platform that uses immobilized antigen in a conducting polymer gel followed by detection via electrochemical measurement of oxidation-reduction reaction between H<jats:sub>2<\/jats:sub>O<jats:sub>2<\/jats:sub>/Tetrametylbenzidine and peroxidase enzyme in a completed assay complex. A highly specific and sensitive assay was developed to quantify levels of IgG antibodies to SARS-CoV-2 in saliva. After establishing linearity and limit of detection we established a reference range of 5 standard deviations above the mean. There were no false positives in 667 consecutive saliva samples obtained prior to 2019. Saliva was obtained from 34 patients who had recovered from documented COVID-19 or had documented positive serologies. All of the patients with symptoms severe enough to seek medical attention had positive antibody tests and 88% overall had positive results. We obtained blinded paired saliva and plasma samples from 14 individuals. The plasma was analyzed using an EUA-FDA cleared ELISA kit and the saliva was analyzed by our Amperial™ assay. All 5 samples with negative plasma titers were negative in saliva testing. Eight of the 9 positive plasma samples were positive in saliva and 1 had borderline results. A CLIA validation was performed as a laboratory developed test in a high complexity laboratory. A quantitative non-invasive saliva based SARS-CoV-2 antibody test was developed and validated with sufficient specificity to be useful for population-based monitoring and monitoring of individuals following vaccination.<\/jats:p>"},{"1":"PLOS ONE","2":"2021-07-01","3":"2021-07-01","4":"2021-07-01","5":"10.1371/journal.pone.0249691","6":"2021-07-02","7":"1932-6203","8":"7","9":"2021-07-01","10":"340","11":"e0249691","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"128","17":"128","18":"0","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Profiling DNA break sites and transcriptional changes in response to contextual fear learning","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0249691","24":"16","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000002","29":"National Institutes of Health","30":"publisher","31":"R01NS102730-01","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,11]>","39":"<jats:p>Neuronal activity generates DNA double-strand breaks (DSBs) at specific loci <jats:italic>in vitro<\/jats:italic> and this facilitates the rapid transcriptional induction of early response genes (ERGs). Physiological neuronal activity, including exposure of mice to learning behaviors, also cause the formation of DSBs, yet the distribution of these breaks and their relation to brain function remains unclear. Here, following contextual fear conditioning (CFC) in mice, we profiled the locations of DSBs genome-wide in the medial prefrontal cortex and hippocampus using γH2AX ChIP-Seq. Remarkably, we found that DSB formation is widespread in the brain compared to cultured primary neurons and they are predominately involved in synaptic processes. We observed increased DNA breaks at genes induced by CFC in neuronal and non-neuronal nuclei. Activity-regulated and proteostasis-related transcription factors appear to govern some of these gene expression changes across cell types. Finally, we find that glia but not neurons have a robust transcriptional response to glucocorticoids, and many of these genes are sites of DSBs. Our results indicate that learning behaviors cause widespread DSB formation in the brain that are associated with experience-driven transcriptional changes across both neuronal and glial cells.<\/jats:p>"},{"1":"PLOS ONE","2":"2021-07-01","3":"2021-07-01","4":"2021-07-01","5":"10.1371/journal.pone.0249691","6":"2021-07-02","7":"1932-6203","8":"7","9":"2021-07-01","10":"340","11":"e0249691","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"128","17":"128","18":"0","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Profiling DNA break sites and transcriptional changes in response to contextual fear learning","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0249691","24":"16","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100001642","29":"Glenn Foundation for Medical Research","30":"publisher","31":"NA","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,11]>","39":"<jats:p>Neuronal activity generates DNA double-strand breaks (DSBs) at specific loci <jats:italic>in vitro<\/jats:italic> and this facilitates the rapid transcriptional induction of early response genes (ERGs). Physiological neuronal activity, including exposure of mice to learning behaviors, also cause the formation of DSBs, yet the distribution of these breaks and their relation to brain function remains unclear. Here, following contextual fear conditioning (CFC) in mice, we profiled the locations of DSBs genome-wide in the medial prefrontal cortex and hippocampus using γH2AX ChIP-Seq. Remarkably, we found that DSB formation is widespread in the brain compared to cultured primary neurons and they are predominately involved in synaptic processes. We observed increased DNA breaks at genes induced by CFC in neuronal and non-neuronal nuclei. Activity-regulated and proteostasis-related transcription factors appear to govern some of these gene expression changes across cell types. Finally, we find that glia but not neurons have a robust transcriptional response to glucocorticoids, and many of these genes are sites of DSBs. Our results indicate that learning behaviors cause widespread DSB formation in the brain that are associated with experience-driven transcriptional changes across both neuronal and glial cells.<\/jats:p>"},{"1":"PLOS ONE","2":"2021-07-01","3":"2021-07-01","4":"2021-07-01","5":"10.1371/journal.pone.0249691","6":"2021-07-02","7":"1932-6203","8":"7","9":"2021-07-01","10":"340","11":"e0249691","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"128","17":"128","18":"0","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Profiling DNA break sites and transcriptional changes in response to contextual fear learning","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0249691","24":"16","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100007457","29":"JPB Foundation","30":"publisher","31":"NA","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,11]>","39":"<jats:p>Neuronal activity generates DNA double-strand breaks (DSBs) at specific loci <jats:italic>in vitro<\/jats:italic> and this facilitates the rapid transcriptional induction of early response genes (ERGs). Physiological neuronal activity, including exposure of mice to learning behaviors, also cause the formation of DSBs, yet the distribution of these breaks and their relation to brain function remains unclear. Here, following contextual fear conditioning (CFC) in mice, we profiled the locations of DSBs genome-wide in the medial prefrontal cortex and hippocampus using γH2AX ChIP-Seq. Remarkably, we found that DSB formation is widespread in the brain compared to cultured primary neurons and they are predominately involved in synaptic processes. We observed increased DNA breaks at genes induced by CFC in neuronal and non-neuronal nuclei. Activity-regulated and proteostasis-related transcription factors appear to govern some of these gene expression changes across cell types. Finally, we find that glia but not neurons have a robust transcriptional response to glucocorticoids, and many of these genes are sites of DSBs. Our results indicate that learning behaviors cause widespread DSB formation in the brain that are associated with experience-driven transcriptional changes across both neuronal and glial cells.<\/jats:p>"},{"1":"PLOS ONE","2":"2020-04-17","3":"2020-04-17","4":"2020-04-17","5":"10.1371/journal.pone.0231791","6":"2021-07-02","7":"1932-6203","8":"4","9":"2020-04-17","10":"340","11":"e0231791","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"60","17":"60","18":"9","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Investigation of synovial fluid induced Staphylococcus aureus aggregate development and its impact on surface attachment and biofilm formation","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0231791","24":"15","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000002","29":"National Institutes of Health","30":"publisher","31":"R01GM124436","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,10]>","39":"NA"},{"1":"PLOS ONE","2":"2021-05-03","3":"2021-05-03","4":"2021-05-03","5":"10.1371/journal.pone.0249798","6":"2021-05-03","7":"1932-6203","8":"5","9":"2021-05-03","10":"340","11":"e0249798","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"72","17":"72","18":"0","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Odor sampling strategies in mice with genetically altered olfactory responses","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0249798","24":"16","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000055","29":"National Institute on Deafness and Other Communication Disorders","30":"publisher","31":"NA","32":"R01DC016647","33":"1P30DC011735","34":"R01DC009613","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,10]>","39":"<jats:p>Peripheral sensory cells and the central neuronal circuits that monitor environmental changes to drive behaviors should be adapted to match the behaviorally relevant kinetics of incoming stimuli, be it the detection of sound frequencies, the speed of moving objects or local temperature changes. Detection of odorants begins with the activation of olfactory receptor neurons in the nasal cavity following inhalation of air and airborne odorants carried therein. Thus, olfactory receptor neurons are stimulated in a rhythmic and repeated fashion that is determined by the breathing or sniffing frequency that can be controlled and altered by the animal. This raises the question of how the response kinetics of olfactory receptor neurons are matched to the imposed stimulation frequency and if, vice versa, the kinetics of olfactory receptor neuron responses determine the sniffing frequency. We addressed this question by using a mouse model that lacks the K<jats:sup>+<\/jats:sup>-dependent Na<jats:sup>+<\/jats:sup>/Ca<jats:sup>2+<\/jats:sup> exchanger 4 (NCKX4), which results in markedly slowed response termination of olfactory receptor neuron responses and hence changes the temporal response kinetics of these neurons. We monitored sniffing behaviors of freely moving wildtype and NCKX4 knockout mice while they performed olfactory Go/NoGo discrimination tasks. Knockout mice performed with similar or, surprisingly, better accuracy compared to wildtype mice, but chose, depending on the task, different odorant sampling durations depending on the behavioral demands of the odorant identification task. Similarly, depending on the demands of the behavioral task, knockout mice displayed a lower basal breathing frequency prior to odorant sampling, a possible mechanism to increase the dynamic range for changes in sniffing frequency during odorant sampling. Overall, changes in sniffing behavior between wildtype and NCKX4 knockout mice were subtle, suggesting that, at least for the particular odorant-driven task we used, slowed response termination of the odorant-induced receptor neuron response either has a limited detrimental effect on odorant-driven behavior or mice are able to compensate via an as yet unknown mechanism.<\/jats:p>"},{"1":"PLOS ONE","2":"2021-05-03","3":"2021-05-03","4":"2021-05-03","5":"10.1371/journal.pone.0249798","6":"2021-05-03","7":"1932-6203","8":"5","9":"2021-05-03","10":"340","11":"e0249798","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"72","17":"72","18":"0","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Odor sampling strategies in mice with genetically altered olfactory responses","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0249798","24":"16","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000182","29":"Medical Research and Materiel Command","30":"crossref","31":"W911NF-11-1-0087","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,10]>","39":"<jats:p>Peripheral sensory cells and the central neuronal circuits that monitor environmental changes to drive behaviors should be adapted to match the behaviorally relevant kinetics of incoming stimuli, be it the detection of sound frequencies, the speed of moving objects or local temperature changes. Detection of odorants begins with the activation of olfactory receptor neurons in the nasal cavity following inhalation of air and airborne odorants carried therein. Thus, olfactory receptor neurons are stimulated in a rhythmic and repeated fashion that is determined by the breathing or sniffing frequency that can be controlled and altered by the animal. This raises the question of how the response kinetics of olfactory receptor neurons are matched to the imposed stimulation frequency and if, vice versa, the kinetics of olfactory receptor neuron responses determine the sniffing frequency. We addressed this question by using a mouse model that lacks the K<jats:sup>+<\/jats:sup>-dependent Na<jats:sup>+<\/jats:sup>/Ca<jats:sup>2+<\/jats:sup> exchanger 4 (NCKX4), which results in markedly slowed response termination of olfactory receptor neuron responses and hence changes the temporal response kinetics of these neurons. We monitored sniffing behaviors of freely moving wildtype and NCKX4 knockout mice while they performed olfactory Go/NoGo discrimination tasks. Knockout mice performed with similar or, surprisingly, better accuracy compared to wildtype mice, but chose, depending on the task, different odorant sampling durations depending on the behavioral demands of the odorant identification task. Similarly, depending on the demands of the behavioral task, knockout mice displayed a lower basal breathing frequency prior to odorant sampling, a possible mechanism to increase the dynamic range for changes in sniffing frequency during odorant sampling. Overall, changes in sniffing behavior between wildtype and NCKX4 knockout mice were subtle, suggesting that, at least for the particular odorant-driven task we used, slowed response termination of the odorant-induced receptor neuron response either has a limited detrimental effect on odorant-driven behavior or mice are able to compensate via an as yet unknown mechanism.<\/jats:p>"},{"1":"PLOS ONE","2":"2021-05-03","3":"2021-05-03","4":"2021-05-03","5":"10.1371/journal.pone.0249798","6":"2021-05-03","7":"1932-6203","8":"5","9":"2021-05-03","10":"340","11":"e0249798","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"72","17":"72","18":"0","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Odor sampling strategies in mice with genetically altered olfactory responses","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0249798","24":"16","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000002","29":"National Institutes of Health","30":"publisher","31":"G20OD020296","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,10]>","39":"<jats:p>Peripheral sensory cells and the central neuronal circuits that monitor environmental changes to drive behaviors should be adapted to match the behaviorally relevant kinetics of incoming stimuli, be it the detection of sound frequencies, the speed of moving objects or local temperature changes. Detection of odorants begins with the activation of olfactory receptor neurons in the nasal cavity following inhalation of air and airborne odorants carried therein. Thus, olfactory receptor neurons are stimulated in a rhythmic and repeated fashion that is determined by the breathing or sniffing frequency that can be controlled and altered by the animal. This raises the question of how the response kinetics of olfactory receptor neurons are matched to the imposed stimulation frequency and if, vice versa, the kinetics of olfactory receptor neuron responses determine the sniffing frequency. We addressed this question by using a mouse model that lacks the K<jats:sup>+<\/jats:sup>-dependent Na<jats:sup>+<\/jats:sup>/Ca<jats:sup>2+<\/jats:sup> exchanger 4 (NCKX4), which results in markedly slowed response termination of olfactory receptor neuron responses and hence changes the temporal response kinetics of these neurons. We monitored sniffing behaviors of freely moving wildtype and NCKX4 knockout mice while they performed olfactory Go/NoGo discrimination tasks. Knockout mice performed with similar or, surprisingly, better accuracy compared to wildtype mice, but chose, depending on the task, different odorant sampling durations depending on the behavioral demands of the odorant identification task. Similarly, depending on the demands of the behavioral task, knockout mice displayed a lower basal breathing frequency prior to odorant sampling, a possible mechanism to increase the dynamic range for changes in sniffing frequency during odorant sampling. Overall, changes in sniffing behavior between wildtype and NCKX4 knockout mice were subtle, suggesting that, at least for the particular odorant-driven task we used, slowed response termination of the odorant-induced receptor neuron response either has a limited detrimental effect on odorant-driven behavior or mice are able to compensate via an as yet unknown mechanism.<\/jats:p>"},{"1":"PLOS ONE","2":"2020-08-25","3":"2020-08-25","4":"2020-08-25","5":"10.1371/journal.pone.0236285","6":"2020-08-25","7":"1932-6203","8":"8","9":"2020-08-25","10":"340","11":"e0236285","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"33","17":"33","18":"0","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Reduced meiotic recombination in rhesus macaques and the origin of the human recombination landscape","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0236285","24":"15","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000002","29":"National Institutes of Health","30":"publisher","31":"NA","32":"5R01HG008115","33":"U54-HG006484-01","34":"R24-OD011173","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,11]>","39":"NA"},{"1":"PLOS ONE","2":"2020-03-11","3":"2020-03-11","4":"2020-03-11","5":"10.1371/journal.pone.0230127","6":"2021-07-03","7":"1932-6203","8":"3","9":"2020-03-11","10":"340","11":"e0230127","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"40","17":"40","18":"7","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Fatal overdose prevention and experience with naloxone: A cross-sectional study from a community-based cohort of people who inject drugs in Baltimore, Maryland","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0230127","24":"15","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000026","29":"National Institute on Drug Abuse","30":"publisher","31":"NA","32":"R01-DA-12568","33":"U01-DA-036297","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,10]>","39":"NA"},{"1":"PLOS ONE","2":"2020-03-11","3":"2020-03-11","4":"2020-03-11","5":"10.1371/journal.pone.0230127","6":"2021-07-03","7":"1932-6203","8":"3","9":"2020-03-11","10":"340","11":"e0230127","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"40","17":"40","18":"7","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Fatal overdose prevention and experience with naloxone: A cross-sectional study from a community-based cohort of people who inject drugs in Baltimore, Maryland","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0230127","24":"15","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000060","29":"National Institute of Allergy and Infectious Diseases","30":"publisher","31":"T32-AI102623","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,10]>","39":"NA"},{"1":"PLOS ONE","2":"2020-03-11","3":"2020-03-11","4":"2020-03-11","5":"10.1371/journal.pone.0230127","6":"2021-07-03","7":"1932-6203","8":"3","9":"2020-03-11","10":"340","11":"e0230127","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"40","17":"40","18":"7","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Fatal overdose prevention and experience with naloxone: A cross-sectional study from a community-based cohort of people who inject drugs in Baltimore, Maryland","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0230127","24":"15","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000002","29":"National Institutes of Health","30":"publisher","31":"P30AI094189","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,10]>","39":"NA"},{"1":"PLOS ONE","2":"2021-03-04","3":"2021-03-04","4":"2021-03-04","5":"10.1371/journal.pone.0247963","6":"2021-03-24","7":"1932-6203","8":"3","9":"2021-03-04","10":"340","11":"e0247963","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"35","17":"35","18":"1","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Construction, characterization, and immunization of nanoparticles that display a diverse array of influenza HA trimers","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0247963","24":"16","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000060","29":"National Institute of Allergy and Infectious Diseases","30":"publisher","31":"1R01AI129784","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,12]>","39":"<jats:p>Current influenza vaccines do not elicit broadly protective immune responses against multiple strains. New strategies to focus the humoral immune response to conserved regions on influenza antigens are therefore required for recognition by broadly neutralizing antibodies. It has been suggested that B-cells with receptors that recognize conserved epitopes would be preferentially stimulated through avidity effects by mosaic particles presenting multiple forms of a variable antigen. We adapted SpyCatcher-based platforms, AP205 virus-like particles (VLPs) and mi3 nanoparticles (NPs), to covalently co-display SpyTagged hemagglutinin (HA) trimers from group 1 and group 2 influenza A strains. Here we show successful homotypic and heterotypic conjugation of up to 8 different HA trimers to both VLPs and NPs. We characterized the HA-VLPs and HA-NPs by cryo-electron tomography to derive the average number of conjugated HAs and their separation distances on particles, and compared immunizations of mosaic and homotypic particles in wild-type mice. Both types of HA particles elicited strong antibody responses, but the mosaic particles did not consistently elicit broader immune responses than mixtures of homotypic particles. We conclude that covalent attachment of HAs from currently-circulating influenza strains represents a viable alternative to current annual influenza vaccine strategies, but in the absence of further modifications, is unlikely to represent a method for making a universal influenza vaccine.<\/jats:p>"},{"1":"PLOS ONE","2":"2021-03-04","3":"2021-03-04","4":"2021-03-04","5":"10.1371/journal.pone.0247963","6":"2021-03-24","7":"1932-6203","8":"3","9":"2021-03-04","10":"340","11":"e0247963","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"35","17":"35","18":"1","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Construction, characterization, and immunization of nanoparticles that display a diverse array of influenza HA trimers","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0247963","24":"16","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000002","29":"National Institutes of Health","30":"publisher","31":"P50 AI150464","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,12]>","39":"<jats:p>Current influenza vaccines do not elicit broadly protective immune responses against multiple strains. New strategies to focus the humoral immune response to conserved regions on influenza antigens are therefore required for recognition by broadly neutralizing antibodies. It has been suggested that B-cells with receptors that recognize conserved epitopes would be preferentially stimulated through avidity effects by mosaic particles presenting multiple forms of a variable antigen. We adapted SpyCatcher-based platforms, AP205 virus-like particles (VLPs) and mi3 nanoparticles (NPs), to covalently co-display SpyTagged hemagglutinin (HA) trimers from group 1 and group 2 influenza A strains. Here we show successful homotypic and heterotypic conjugation of up to 8 different HA trimers to both VLPs and NPs. We characterized the HA-VLPs and HA-NPs by cryo-electron tomography to derive the average number of conjugated HAs and their separation distances on particles, and compared immunizations of mosaic and homotypic particles in wild-type mice. Both types of HA particles elicited strong antibody responses, but the mosaic particles did not consistently elicit broader immune responses than mixtures of homotypic particles. We conclude that covalent attachment of HAs from currently-circulating influenza strains represents a viable alternative to current annual influenza vaccine strategies, but in the absence of further modifications, is unlikely to represent a method for making a universal influenza vaccine.<\/jats:p>"},{"1":"PLOS ONE","2":"2021-02-11","3":"2021-02-11","4":"2021-02-10","5":"10.1371/journal.pone.0244556","6":"2021-02-11","7":"1932-6203","8":"2","9":"2021-02-10","10":"340","11":"e0244556","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"64","17":"64","18":"0","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Pathways of calcium regulation, electron transport, and mitochondrial protein translation are molecular signatures of susceptibility to recurrent exertional rhabdomyolysis in Thoroughbred racehorses","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0244556","24":"16","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100001655","29":"Grayson-Jockey Club Research Foundation","30":"publisher","31":"NA","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,12]>","39":"<jats:p>Recurrent exertional rhabdomyolysis (RER) is a chronic muscle disorder of unknown etiology in racehorses. A potential role of intramuscular calcium (Ca<jats:sup>2+<\/jats:sup>) dysregulation in RER has led to the use of dantrolene to prevent episodes of rhabdomyolysis. We examined differentially expressed proteins (DEP) and gene transcripts (DEG) in gluteal muscle of Thoroughbred race-trained mares after exercise among three groups of 5 horses each; 1) horses susceptible to, but not currently experiencing rhabdomyolysis, 2) healthy horses with no history of RER (control), 3) RER-susceptible horses treated with dantrolene pre-exercise (RER-D). Tandem mass tag LC/MS/MS quantitative proteomics and RNA-seq analysis (FDR &lt;0.05) was followed by gene ontology (GO) and semantic similarity of enrichment terms. Of the 375 proteins expressed, 125 were DEP in RER-susceptible versus control, with 52 ↑DEP mainly involving Ca<jats:sup>2+<\/jats:sup> regulation (N = 11) (e.g. RYR1, calmodulin, calsequestrin, calpain), protein degradation (N = 6), antioxidants (N = 4), plasma membranes (N = 3), glyco(geno)lysis (N = 3) and 21 DEP being blood-borne. ↓DEP (N = 73) were largely mitochondrial (N = 45) impacting the electron transport system (28), enzymes (6), heat shock proteins (4), and contractile proteins (12) including Ca<jats:sup>2+<\/jats:sup> binding proteins. There were 812 DEG in RER-susceptible versus control involving the electron transfer system, the mitochondrial transcription/translational response and notably the pro-apoptotic Ca<jats:sup>2+-<\/jats:sup>activated mitochondrial membrane transition pore (<jats:italic>SLC25A27<\/jats:italic>, <jats:italic>BAX<\/jats:italic>, <jats:italic>ATP5<\/jats:italic> subunits). Upregulated mitochondrial DEG frequently had downregulation of their encoded DEP with semantic similarities highlighting signaling mechanisms regulating mitochondrial protein translation. RER-susceptible horses treated with dantrolene, which slows sarcoplasmic reticulum Ca<jats:sup>2+<\/jats:sup> release, showed no DEG compared to control horses. We conclude that RER-susceptibility is associated with alterations in proteins, genes and pathways impacting myoplasmic Ca<jats:sup>2+<\/jats:sup> regulation, the mitochondrion and protein degradation with opposing effects on mitochondrial transcriptional/translational responses and mitochondrial protein content. RER could potentially arise from excessive sarcoplasmic reticulum Ca<jats:sup>2+<\/jats:sup> release and subsequent mitochondrial buffering of excessive myoplasmic Ca<jats:sup>2+<\/jats:sup>.<\/jats:p>"},{"1":"PLOS ONE","2":"2021-02-11","3":"2021-02-11","4":"2021-02-10","5":"10.1371/journal.pone.0244556","6":"2021-02-11","7":"1932-6203","8":"2","9":"2021-02-10","10":"340","11":"e0244556","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"64","17":"64","18":"0","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Pathways of calcium regulation, electron transport, and mitochondrial protein translation are molecular signatures of susceptibility to recurrent exertional rhabdomyolysis in Thoroughbred racehorses","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0244556","24":"16","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000002","29":"National Institutes of Health","30":"publisher","31":"T32 OD011167","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,12]>","39":"<jats:p>Recurrent exertional rhabdomyolysis (RER) is a chronic muscle disorder of unknown etiology in racehorses. A potential role of intramuscular calcium (Ca<jats:sup>2+<\/jats:sup>) dysregulation in RER has led to the use of dantrolene to prevent episodes of rhabdomyolysis. We examined differentially expressed proteins (DEP) and gene transcripts (DEG) in gluteal muscle of Thoroughbred race-trained mares after exercise among three groups of 5 horses each; 1) horses susceptible to, but not currently experiencing rhabdomyolysis, 2) healthy horses with no history of RER (control), 3) RER-susceptible horses treated with dantrolene pre-exercise (RER-D). Tandem mass tag LC/MS/MS quantitative proteomics and RNA-seq analysis (FDR &lt;0.05) was followed by gene ontology (GO) and semantic similarity of enrichment terms. Of the 375 proteins expressed, 125 were DEP in RER-susceptible versus control, with 52 ↑DEP mainly involving Ca<jats:sup>2+<\/jats:sup> regulation (N = 11) (e.g. RYR1, calmodulin, calsequestrin, calpain), protein degradation (N = 6), antioxidants (N = 4), plasma membranes (N = 3), glyco(geno)lysis (N = 3) and 21 DEP being blood-borne. ↓DEP (N = 73) were largely mitochondrial (N = 45) impacting the electron transport system (28), enzymes (6), heat shock proteins (4), and contractile proteins (12) including Ca<jats:sup>2+<\/jats:sup> binding proteins. There were 812 DEG in RER-susceptible versus control involving the electron transfer system, the mitochondrial transcription/translational response and notably the pro-apoptotic Ca<jats:sup>2+-<\/jats:sup>activated mitochondrial membrane transition pore (<jats:italic>SLC25A27<\/jats:italic>, <jats:italic>BAX<\/jats:italic>, <jats:italic>ATP5<\/jats:italic> subunits). Upregulated mitochondrial DEG frequently had downregulation of their encoded DEP with semantic similarities highlighting signaling mechanisms regulating mitochondrial protein translation. RER-susceptible horses treated with dantrolene, which slows sarcoplasmic reticulum Ca<jats:sup>2+<\/jats:sup> release, showed no DEG compared to control horses. We conclude that RER-susceptibility is associated with alterations in proteins, genes and pathways impacting myoplasmic Ca<jats:sup>2+<\/jats:sup> regulation, the mitochondrion and protein degradation with opposing effects on mitochondrial transcriptional/translational responses and mitochondrial protein content. RER could potentially arise from excessive sarcoplasmic reticulum Ca<jats:sup>2+<\/jats:sup> release and subsequent mitochondrial buffering of excessive myoplasmic Ca<jats:sup>2+<\/jats:sup>.<\/jats:p>"},{"1":"PLOS ONE","2":"2021-02-11","3":"2021-02-11","4":"2021-02-10","5":"10.1371/journal.pone.0244556","6":"2021-02-11","7":"1932-6203","8":"2","9":"2021-02-10","10":"340","11":"e0244556","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"64","17":"64","18":"0","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Pathways of calcium regulation, electron transport, and mitochondrial protein translation are molecular signatures of susceptibility to recurrent exertional rhabdomyolysis in Thoroughbred racehorses","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0244556","24":"16","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"NA","29":"Mary Anne McPhail Endowment","30":"NA","31":"NA","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,12]>","39":"<jats:p>Recurrent exertional rhabdomyolysis (RER) is a chronic muscle disorder of unknown etiology in racehorses. A potential role of intramuscular calcium (Ca<jats:sup>2+<\/jats:sup>) dysregulation in RER has led to the use of dantrolene to prevent episodes of rhabdomyolysis. We examined differentially expressed proteins (DEP) and gene transcripts (DEG) in gluteal muscle of Thoroughbred race-trained mares after exercise among three groups of 5 horses each; 1) horses susceptible to, but not currently experiencing rhabdomyolysis, 2) healthy horses with no history of RER (control), 3) RER-susceptible horses treated with dantrolene pre-exercise (RER-D). Tandem mass tag LC/MS/MS quantitative proteomics and RNA-seq analysis (FDR &lt;0.05) was followed by gene ontology (GO) and semantic similarity of enrichment terms. Of the 375 proteins expressed, 125 were DEP in RER-susceptible versus control, with 52 ↑DEP mainly involving Ca<jats:sup>2+<\/jats:sup> regulation (N = 11) (e.g. RYR1, calmodulin, calsequestrin, calpain), protein degradation (N = 6), antioxidants (N = 4), plasma membranes (N = 3), glyco(geno)lysis (N = 3) and 21 DEP being blood-borne. ↓DEP (N = 73) were largely mitochondrial (N = 45) impacting the electron transport system (28), enzymes (6), heat shock proteins (4), and contractile proteins (12) including Ca<jats:sup>2+<\/jats:sup> binding proteins. There were 812 DEG in RER-susceptible versus control involving the electron transfer system, the mitochondrial transcription/translational response and notably the pro-apoptotic Ca<jats:sup>2+-<\/jats:sup>activated mitochondrial membrane transition pore (<jats:italic>SLC25A27<\/jats:italic>, <jats:italic>BAX<\/jats:italic>, <jats:italic>ATP5<\/jats:italic> subunits). Upregulated mitochondrial DEG frequently had downregulation of their encoded DEP with semantic similarities highlighting signaling mechanisms regulating mitochondrial protein translation. RER-susceptible horses treated with dantrolene, which slows sarcoplasmic reticulum Ca<jats:sup>2+<\/jats:sup> release, showed no DEG compared to control horses. We conclude that RER-susceptibility is associated with alterations in proteins, genes and pathways impacting myoplasmic Ca<jats:sup>2+<\/jats:sup> regulation, the mitochondrion and protein degradation with opposing effects on mitochondrial transcriptional/translational responses and mitochondrial protein content. RER could potentially arise from excessive sarcoplasmic reticulum Ca<jats:sup>2+<\/jats:sup> release and subsequent mitochondrial buffering of excessive myoplasmic Ca<jats:sup>2+<\/jats:sup>.<\/jats:p>"},{"1":"PLOS ONE","2":"2021-07-02","3":"2021-07-02","4":"2021-07-02","5":"10.1371/journal.pone.0254161","6":"2021-07-02","7":"1932-6203","8":"7","9":"2021-07-02","10":"340","11":"e0254161","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"21","17":"21","18":"0","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Oral secretory leukocyte protease inhibitor (SLPI): Associations with oropharyngeal cancer and treatment outcome","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0254161","24":"16","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000002","29":"National Institutes of Health","30":"publisher","31":"NA","32":"R21 DE024816","33":"R01 CA214588","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,10]>","39":"<jats:sec id=\"sec001\">\\n<jats:title>Background<\/jats:title>\\n<jats:p>Rates of oropharyngeal cancer (OPC) associated with alcohol &amp; tobacco use have decreased, while human papillomavirus (HPV) associated OPC has increased among men in the US. Secretory leukocyte protease inhibitor (SLPI), detectable in a variety of secretions, has been implicated in cancers of the head and neck, associated with tumor progression and anti-viral activity. Using the recently verified oral gargle specimen, this study aimed to assess the association of salivary SLPI expression with risk of OPC and response to treatment.<\/jats:p>\\n<\/jats:sec>\\n<jats:sec id=\"sec002\">\\n<jats:title>Methods<\/jats:title>\\n<jats:p>A case-control study design compared levels of salivary SLPI among OPC cases to age and tobacco smoking matched healthy controls. Oral HPV DNA and SLPI was quantified from oral gargle specimens. Logistic regression estimated odds ratios (OR) and 95% confidence intervals (CI) for associations of oral SLPI and risk of OPC and treatment outcomes.<\/jats:p>\\n<\/jats:sec>\\n<jats:sec id=\"sec003\">\\n<jats:title>Results<\/jats:title>\\n<jats:p>In crude and adjusted analyses of 96 OPC cases and 97 age- and smoking-matched controls, OPC was not significantly associated with oral gargle SLPI levels. Among cases, oral SLPI was associated with tonsillectomy (p = 0.018) and among controls oral SLPI was associated with HPV in the oral gargle (p = 0.008). Higher concentrations of SLPI was significantly associated with increased odds of incomplete treatment response (T2: OR: 12.39; 95% CI: 1.44–106.72; T3: OR: 9.86; 95% CI: 1.13–85.90) among all cases, but not among P16+ cases.<\/jats:p>\\n<\/jats:sec>\\n<jats:sec id=\"sec004\">\\n<jats:title>Conclusions<\/jats:title>\\n<jats:p>Salivary SLPI was not associated with OPC risk but was associated with higher odds of an incomplete treatment response.<\/jats:p>\\n<\/jats:sec>"},{"1":"PLOS ONE","2":"2021-07-02","3":"2021-07-02","4":"2021-07-02","5":"10.1371/journal.pone.0254177","6":"2021-07-02","7":"1932-6203","8":"7","9":"2021-07-02","10":"340","11":"e0254177","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"37","17":"37","18":"0","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Viral and immunological markers of HIV-associated Kaposi sarcoma recurrence","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0254177","24":"16","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000002","29":"National Institutes of Health","30":"publisher","31":"NA","32":"U54CA221204","33":"K43TW011095","34":"D43TW010354","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,10]>","39":"<jats:p>Kaposi sarcoma (KS) is an AIDS-defining angio-proliferative malignancy highly prevalent in Sub-Saharan Africa. The main objective of this study was to determine the factors associated with recurrence of HIV-associated KS. We recruited a cohort of individuals on antiretroviral therapy who were in remission for HIV-associated KS after undergoing cytotoxic cancer chemotherapy. Collected variables included sociodemographic and clinical parameters, cytokines and chemokines, HIV viral loads, and CD4 counts. Compared to individuals who had KS recurrence, IL-5 was significantly higher at time of follow-up in individuals who had sustained remission (22.7pg/ml <jats:italic>vs<\/jats:italic>. 2.4pg/ml; p = 0.02); IL-6 was significantly higher at baseline and time of follow-up in individuals who had sustained remission, (18.4pg/ml <jats:italic>vs<\/jats:italic>. 0pg/ml; p = 0.01) and (18.0pg/ml <jats:italic>vs<\/jats:italic>. 0.18pg/ml; p = 0.03) respectively; IP-10 was significantly lower at baseline and at time of follow-up in individuals who had sustained remission, (534pg/ml <jats:italic>vs<\/jats:italic>. 920pg/ml; p = 0.04) and (446pg/ml <jats:italic>vs<\/jats:italic>.1098pg/ml; p = 0.01) respectively; while HIV viral load was significantly lower at baseline and at time of follow-up in individuals who had sustained remission, (0copies/ml <jats:italic>vs<\/jats:italic>. 113copies/ml; p = 0.004) and (0copies/ml <jats:italic>vs<\/jats:italic>. 152copies/ml; p = 0.025) respectively. Plasma levels of IL-5, IL-6, and IP-10 are associated with recurrence of HIV-associated KS, while persistently detectable HIV viral loads increase the risk of KS recurrence.<\/jats:p>"},{"1":"PLOS ONE","2":"2020-04-24","3":"2020-04-24","4":"2020-04-24","5":"10.1371/journal.pone.0231501","6":"2020-06-04","7":"1932-6203","8":"4","9":"2020-04-24","10":"340","11":"e0231501","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"49","17":"49","18":"0","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Periostin interaction with discoidin domain receptor-1 (DDR1) promotes cartilage degeneration","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0231501","24":"15","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000002","29":"National Institutes of Health","30":"publisher","31":"NA","32":"AR070934","33":"AR054817","34":"AR068341","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,10]>","39":"NA"},{"1":"PLOS ONE","2":"2020-11-03","3":"2020-11-03","4":"2020-11-03","5":"10.1371/journal.pone.0240932","6":"2021-06-29","7":"1932-6203","8":"11","9":"2020-11-03","10":"340","11":"e0240932","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"80","17":"80","18":"3","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Bridging of nucleosome-proximal DNA double-strand breaks by PARP2 enhances its interaction with HPF1","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0240932","24":"15","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000054","29":"","30":"publisher","31":"CA218255","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,10]>","39":"NA"},{"1":"PLOS ONE","2":"2020-11-03","3":"2020-11-03","4":"2020-11-03","5":"10.1371/journal.pone.0240932","6":"2021-06-29","7":"1932-6203","8":"11","9":"2020-11-03","10":"340","11":"e0240932","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"80","17":"80","18":"3","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Bridging of nucleosome-proximal DNA double-strand breaks by PARP2 enhances its interaction with HPF1","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0240932","24":"15","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000002","29":"National Institutes of Health","30":"publisher","31":"T32 GM008759","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,10]>","39":"NA"},{"1":"PLOS ONE","2":"2020-11-03","3":"2020-11-03","4":"2020-11-03","5":"10.1371/journal.pone.0240932","6":"2021-06-29","7":"1932-6203","8":"11","9":"2020-11-03","10":"340","11":"e0240932","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"80","17":"80","18":"3","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Bridging of nucleosome-proximal DNA double-strand breaks by PARP2 enhances its interaction with HPF1","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0240932","24":"15","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000011","29":"Howard Hughes Medical Institute","30":"publisher","31":"NA","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,10]>","39":"NA"},{"1":"PLOS ONE","2":"2020-11-03","3":"2020-11-03","4":"2020-11-03","5":"10.1371/journal.pone.0240932","6":"2021-06-29","7":"1932-6203","8":"11","9":"2020-11-03","10":"340","11":"e0240932","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"80","17":"80","18":"3","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Bridging of nucleosome-proximal DNA double-strand breaks by PARP2 enhances its interaction with HPF1","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0240932","24":"15","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000968","29":"American Heart Association","30":"publisher","31":"20POST35211059","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,10]>","39":"NA"},{"1":"PLOS ONE","2":"2020-05-29","3":"2020-05-29","4":"2020-05-29","5":"10.1371/journal.pone.0233672","6":"2021-06-28","7":"1932-6203","8":"5","9":"2020-05-29","10":"340","11":"e0233672","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"47","17":"47","18":"4","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"An exon skipping screen identifies antitumor drugs that are potent modulators of pre-mRNA splicing, suggesting new therapeutic applications","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0233672","24":"15","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000002","29":"National Institutes of Health","30":"crossref","31":"CA2147590","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,10]>","39":"NA"},{"1":"PLOS ONE","2":"2020-06-04","3":"2020-06-04","4":"2020-06-04","5":"10.1371/journal.pone.0233991","6":"2020-06-04","7":"1932-6203","8":"6","9":"2020-06-04","10":"340","11":"e0233991","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"49","17":"49","18":"0","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Neuronal transcriptome analyses reveal novel neuropeptide modulators of excitation and inhibition imbalance in C. elegans","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0233991","24":"15","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000002","29":"National Institutes of Health","30":"publisher","31":"R37 035546","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,10]>","39":"NA"},{"1":"PLOS ONE","2":"2020-11-04","3":"2020-11-04","4":"2020-11-04","5":"10.1371/journal.pone.0241294","6":"2020-11-04","7":"1932-6203","8":"11","9":"2020-11-04","10":"340","11":"e0241294","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"70","17":"70","18":"0","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Secreted protein acidic and rich in cysteine (SPARC) knockout mice have greater outflow facility","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0241294","24":"15","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000053","29":"National Eye Institute","30":"crossref","31":"EY 019654-01","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,10]>","39":"NA"},{"1":"PLOS ONE","2":"2020-11-04","3":"2020-11-04","4":"2020-11-04","5":"10.1371/journal.pone.0241294","6":"2020-11-04","7":"1932-6203","8":"11","9":"2020-11-04","10":"340","11":"e0241294","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"70","17":"70","18":"0","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Secreted protein acidic and rich in cysteine (SPARC) knockout mice have greater outflow facility","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0241294","24":"15","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000002","29":"National Institutes of Health","30":"publisher","31":"P30-EY11373","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,10]>","39":"NA"},{"1":"PLOS ONE","2":"2020-11-04","3":"2020-11-04","4":"2020-11-04","5":"10.1371/journal.pone.0241294","6":"2020-11-04","7":"1932-6203","8":"11","9":"2020-11-04","10":"340","11":"e0241294","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"70","17":"70","18":"0","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Secreted protein acidic and rich in cysteine (SPARC) knockout mice have greater outflow facility","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0241294","24":"15","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100001818","29":"Research to Prevent Blindness","30":"publisher","31":"NA","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,10]>","39":"NA"},{"1":"PLOS ONE","2":"2020-11-04","3":"2020-11-04","4":"2020-11-04","5":"10.1371/journal.pone.0241294","6":"2020-11-04","7":"1932-6203","8":"11","9":"2020-11-04","10":"340","11":"e0241294","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"70","17":"70","18":"0","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Secreted protein acidic and rich in cysteine (SPARC) knockout mice have greater outflow facility","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0241294","24":"15","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"NA","29":"Cleveland Eye Bank Foundation","30":"NA","31":"NA","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,10]>","39":"NA"},{"1":"PLOS ONE","2":"2020-07-30","3":"2020-07-30","4":"2020-07-30","5":"10.1371/journal.pone.0236741","6":"2020-12-16","7":"1932-6203","8":"7","9":"2020-07-30","10":"340","11":"e0236741","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"75","17":"75","18":"1","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Pdgfrα-Cre mediated knockout of the aryl hydrocarbon receptor protects mice from high-fat diet induced obesity and hepatic steatosis","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0236741","24":"15","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000002","29":"National Institutes of Health","30":"publisher","31":"K01DK111758","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,10]>","39":"NA"},{"1":"PLOS ONE","2":"2020-07-30","3":"2020-07-30","4":"2020-07-30","5":"10.1371/journal.pone.0236741","6":"2020-12-16","7":"1932-6203","8":"7","9":"2020-07-30","10":"340","11":"e0236741","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"75","17":"75","18":"1","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Pdgfrα-Cre mediated knockout of the aryl hydrocarbon receptor protects mice from high-fat diet induced obesity and hepatic steatosis","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0236741","24":"15","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"NA","29":"Fraternal Order of Eagles Diabetes Research Center","30":"NA","31":"NA","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,10]>","39":"NA"},{"1":"PLOS ONE","2":"2020-05-21","3":"2020-05-21","4":"2020-05-21","5":"10.1371/journal.pone.0233333","6":"2021-07-19","7":"1932-6203","8":"5","9":"2020-05-21","10":"340","11":"e0233333","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"113","17":"113","18":"6","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"High mitochondrial diversity of domesticated goats persisted among Bronze and Iron Age pastoralists in the Inner Asian Mountain Corridor","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0233333","24":"15","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000002","29":"National Institutes of Health","30":"publisher","31":"Deutsche Forschungsgemeinschaft GSC: 208","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,11]>","39":"NA"},{"1":"PLOS ONE","2":"2020-05-21","3":"2020-05-21","4":"2020-05-21","5":"10.1371/journal.pone.0233333","6":"2021-07-19","7":"1932-6203","8":"5","9":"2020-05-21","10":"340","11":"e0233333","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"113","17":"113","18":"6","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"High mitochondrial diversity of domesticated goats persisted among Bronze and Iron Age pastoralists in the Inner Asian Mountain Corridor","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0233333","24":"15","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100010663","29":"H2020 European Research Council","30":"publisher","31":"772957","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,11]>","39":"NA"},{"1":"PLOS ONE","2":"2020-05-21","3":"2020-05-21","4":"2020-05-21","5":"10.1371/journal.pone.0233333","6":"2021-07-19","7":"1932-6203","8":"5","9":"2020-05-21","10":"340","11":"e0233333","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"113","17":"113","18":"6","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"High mitochondrial diversity of domesticated goats persisted among Bronze and Iron Age pastoralists in the Inner Asian Mountain Corridor","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0233333","24":"15","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/501100004895","29":"European Social Fund","30":"publisher","31":"09.3.3-LMT-K-712","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,11]>","39":"NA"},{"1":"PLOS ONE","2":"2020-05-21","3":"2020-05-21","4":"2020-05-21","5":"10.1371/journal.pone.0233333","6":"2021-07-19","7":"1932-6203","8":"5","9":"2020-05-21","10":"340","11":"e0233333","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"113","17":"113","18":"6","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"High mitochondrial diversity of domesticated goats persisted among Bronze and Iron Age pastoralists in the Inner Asian Mountain Corridor","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0233333","24":"15","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000001","29":"National Science Foundation","30":"publisher","31":"NA","32":"0211431","33":"1132090","34":"0535341","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,11]>","39":"NA"},{"1":"PLOS ONE","2":"2020-05-21","3":"2020-05-21","4":"2020-05-21","5":"10.1371/journal.pone.0233333","6":"2021-07-19","7":"1932-6203","8":"5","9":"2020-05-21","10":"340","11":"e0233333","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"113","17":"113","18":"6","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"High mitochondrial diversity of domesticated goats persisted among Bronze and Iron Age pastoralists in the Inner Asian Mountain Corridor","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0233333","24":"15","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100009498","29":"Western Kentucky University","30":"crossref","31":"NA","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,11]>","39":"NA"},{"1":"PLOS ONE","2":"2020-05-21","3":"2020-05-21","4":"2020-05-21","5":"10.1371/journal.pone.0233333","6":"2021-07-19","7":"1932-6203","8":"5","9":"2020-05-21","10":"340","11":"e0233333","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"113","17":"113","18":"6","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"High mitochondrial diversity of domesticated goats persisted among Bronze and Iron Age pastoralists in the Inner Asian Mountain Corridor","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0233333","24":"15","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100007268","29":"Washington University in St. Louis","30":"crossref","31":"NA","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,11]>","39":"NA"},{"1":"PLOS ONE","2":"2020-05-21","3":"2020-05-21","4":"2020-05-21","5":"10.1371/journal.pone.0233333","6":"2021-07-19","7":"1932-6203","8":"5","9":"2020-05-21","10":"340","11":"e0233333","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"113","17":"113","18":"6","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"High mitochondrial diversity of domesticated goats persisted among Bronze and Iron Age pastoralists in the Inner Asian Mountain Corridor","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0233333","24":"15","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/501100006769","29":"Russian Science Foundation","30":"publisher","31":"16-18-10033","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,11]>","39":"NA"},{"1":"PLOS ONE","2":"2021-03-24","3":"2021-03-24","4":"2021-03-24","5":"10.1371/journal.pone.0245424","6":"2021-07-17","7":"1932-6203","8":"3","9":"2021-03-24","10":"340","11":"e0245424","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"80","17":"80","18":"4","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Influence of obesity on serum levels of SARS-CoV-2-specific antibodies in COVID-19 patients","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0245424","24":"16","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000002","29":"National Institutes of Health","30":"crossref","31":"NA","32":"AG059719","33":"AG32576","34":"AG023717","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,10]>","39":"<jats:p>SARS-CoV-2 (Severe Acute Respiratory Syndrome Corona Virus-2), cause of COVID-19 (Coronavirus Disease of 2019), represents a significant risk to people living with pre-existing conditions associated with exacerbated inflammatory responses and consequent dysfunctional immunity. In this paper, we have evaluated the influence of obesity, a condition associated with chronic systemic inflammation, on the secretion of SARS-CoV-2-specific IgG antibodies in the blood of COVID-19 patients. Our hypothesis is that obesity is associated with reduced amounts of specific IgG antibodies. Results have confirmed our hypothesis and have shown that SARS-CoV-2 IgG antibodies are negatively associated with Body Mass Index (BMI) in COVID-19 obese patients, as expected based on the known influence of obesity on humoral immunity. Antibodies in COVID-19 obese patients are also negatively associated with serum levels of pro-inflammatory and metabolic markers of inflammaging and pulmonary inflammation, such as SAA (serum amyloid A protein), CRP (C-reactive protein), and ferritin, but positively associated with NEFA (nonesterified fatty acids). These results altogether could help to identify an inflammatory signature with strong predictive value for immune dysfunction. Inflammatory markers identified may subsequently be targeted to improve humoral immunity in individuals with obesity and in individuals with other chronic inflammatory conditions.<\/jats:p>"},{"1":"PLOS ONE","2":"2021-03-24","3":"2021-03-24","4":"2021-03-24","5":"10.1371/journal.pone.0245424","6":"2021-07-17","7":"1932-6203","8":"3","9":"2021-03-24","10":"340","11":"e0245424","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"80","17":"80","18":"4","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Influence of obesity on serum levels of SARS-CoV-2-specific antibodies in COVID-19 patients","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0245424","24":"16","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000002","29":"National Institutes of Health","30":"publisher","31":"NA","32":"AG059719","33":"AG32576","34":"AG023717","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,10]>","39":"<jats:p>SARS-CoV-2 (Severe Acute Respiratory Syndrome Corona Virus-2), cause of COVID-19 (Coronavirus Disease of 2019), represents a significant risk to people living with pre-existing conditions associated with exacerbated inflammatory responses and consequent dysfunctional immunity. In this paper, we have evaluated the influence of obesity, a condition associated with chronic systemic inflammation, on the secretion of SARS-CoV-2-specific IgG antibodies in the blood of COVID-19 patients. Our hypothesis is that obesity is associated with reduced amounts of specific IgG antibodies. Results have confirmed our hypothesis and have shown that SARS-CoV-2 IgG antibodies are negatively associated with Body Mass Index (BMI) in COVID-19 obese patients, as expected based on the known influence of obesity on humoral immunity. Antibodies in COVID-19 obese patients are also negatively associated with serum levels of pro-inflammatory and metabolic markers of inflammaging and pulmonary inflammation, such as SAA (serum amyloid A protein), CRP (C-reactive protein), and ferritin, but positively associated with NEFA (nonesterified fatty acids). These results altogether could help to identify an inflammatory signature with strong predictive value for immune dysfunction. Inflammatory markers identified may subsequently be targeted to improve humoral immunity in individuals with obesity and in individuals with other chronic inflammatory conditions.<\/jats:p>"},{"1":"PLOS ONE","2":"2020-11-25","3":"2020-11-25","4":"2020-11-25","5":"10.1371/journal.pone.0241916","6":"2020-11-25","7":"1932-6203","8":"11","9":"2020-11-25","10":"340","11":"e0241916","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"42","17":"42","18":"0","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Evaluation of metronomic chemotherapy response using diffusion and dynamic contrast-enhanced MRI","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0241916","24":"15","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000002","29":"National Institutes of Health","30":"publisher","31":"R01CA160620, R01CA219964, UG3CA228699, and P41EB017183","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,10]>","39":"<jats:sec id=\"sec001\">\\n<jats:title>Purpose<\/jats:title>\\n<jats:p>To investigate the feasibility of using diffusion MRI (dMRI) and dynamic contrast-enhanced (DCE) MRI to evaluate the treatment response of metronomic chemotherapy (MCT) in the 4T1 mammary tumor model of locally advanced breast cancer.<\/jats:p>\\n<\/jats:sec>\\n<jats:sec id=\"sec002\">\\n<jats:title>Methods<\/jats:title>\\n<jats:p>Twelve Balb/c mice with metastatic breast cancer were divided into treated and untreated (control) groups. The treated group (n = 6) received five treatments of anti-metabolite agent 5-Fluorouracil (5FU) in the span of two weeks. dMRI and DCE-MRI were acquired for both treated and control groups before and after MCT. Immunohistochemically staining and measurements were performed after the post-MRI measurements for comparison.<\/jats:p>\\n<\/jats:sec>\\n<jats:sec id=\"sec003\">\\n<jats:title>Results<\/jats:title>\\n<jats:p>The control mice had significantly (<jats:italic>p<\/jats:italic>&lt;0.005) larger tumors than the MCT treated mice. The DCE-MRI analysis showed a decrease in contrast enhancement for the control group, whereas the MCT mice had a more stable enhancement between the pre-chemo and post-chemo time points. This confirms the antiangiogenic effects of 5FU treatment. Comparing amplitude of enhancement revealed a significantly (p&lt;0.05) higher enhancement in the MCT tumors than in the controls. Moreover, the MCT uptake rate was significantly (p&lt;0.001) slower than the controls. dMRI analysis showed the MCT ADC values were significantly larger than the control group at the post-scan time point.<\/jats:p>\\n<\/jats:sec>\\n<jats:sec id=\"sec004\">\\n<jats:title>Conclusion<\/jats:title>\\n<jats:p>dMRI and DCE-MRI can be used as potential biomarkers for assessing the treatment response of MCT. The MRI and pathology observations suggested that in addition to the cytotoxic effect of cell kills, the MCT with a cytotoxic drug, 5FU, induced changes in the tumor vasculature similar to the anti-angiogenic effect.<\/jats:p>\\n<\/jats:sec>"},{"1":"PLOS ONE","2":"2021-03-30","3":"2021-03-30","4":"2021-03-30","5":"10.1371/journal.pone.0245675","6":"2021-03-30","7":"1932-6203","8":"3","9":"2021-03-30","10":"340","11":"e0245675","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"76","17":"76","18":"0","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"POT1 stability and binding measured by fluorescence thermal shift assays","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0245675","24":"16","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000002","29":"National Institutes of Health","30":"crossref","31":"GM07742","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,12]>","39":"<jats:p>The protein POT1 (<jats:underline>P<\/jats:underline>rotection <jats:underline>o<\/jats:underline>f <jats:underline>T<\/jats:underline>elomeres 1) is an integral part of the shelterin complex that protects the ends of human chromosomes from degradation or end fusions. It is the only component of shelterin that binds single-stranded DNA. We describe here the application of two separate fluorescent thermal shift assays (FTSA) that provide quantitative biophysical characterization of POT1 stability and its interactions. The first assay uses Sypro Orange™ and monitors the thermal stability of POT1 and its binding under a variety of conditions. This assay is useful for the quality control of POT1 preparations, for biophysical characterization of its DNA binding and, potentially, as an efficient screening tool for binding of small molecule drug candidates. The second assay uses a FRET-labeled human telomeric G-quadruplex structure that reveals the effects of POT1 binding on thermal stability from the DNA frame of reference. These complementary assays provide efficient biophysical approaches for the quantitative characterization of multiple aspects of POT1 structure and function. The results from these assays provide thermodynamics details of POT1 folding, the sequence selectivity of its DNA binding and the thermodynamic profile for its binding to its preferred DNA binding sequence. Most significantly, results from these assays elucidate two mechanisms for the inhibition of POT1 –DNA interactions. The first is by competitive inhibition at the POT1 DNA binding site. The second is indirect and is by stabilization of G-quadruplex formation within the normal POT1 single-stranded DNA sequence to prevent POT1 binding.<\/jats:p>"},{"1":"PLOS ONE","2":"2021-03-30","3":"2021-03-30","4":"2021-03-30","5":"10.1371/journal.pone.0245675","6":"2021-03-30","7":"1932-6203","8":"3","9":"2021-03-30","10":"340","11":"e0245675","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"76","17":"76","18":"0","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"POT1 stability and binding measured by fluorescence thermal shift assays","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0245675","24":"16","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000002","29":"National Institutes of Health","30":"publisher","31":"GM07742","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,12]>","39":"<jats:p>The protein POT1 (<jats:underline>P<\/jats:underline>rotection <jats:underline>o<\/jats:underline>f <jats:underline>T<\/jats:underline>elomeres 1) is an integral part of the shelterin complex that protects the ends of human chromosomes from degradation or end fusions. It is the only component of shelterin that binds single-stranded DNA. We describe here the application of two separate fluorescent thermal shift assays (FTSA) that provide quantitative biophysical characterization of POT1 stability and its interactions. The first assay uses Sypro Orange™ and monitors the thermal stability of POT1 and its binding under a variety of conditions. This assay is useful for the quality control of POT1 preparations, for biophysical characterization of its DNA binding and, potentially, as an efficient screening tool for binding of small molecule drug candidates. The second assay uses a FRET-labeled human telomeric G-quadruplex structure that reveals the effects of POT1 binding on thermal stability from the DNA frame of reference. These complementary assays provide efficient biophysical approaches for the quantitative characterization of multiple aspects of POT1 structure and function. The results from these assays provide thermodynamics details of POT1 folding, the sequence selectivity of its DNA binding and the thermodynamic profile for its binding to its preferred DNA binding sequence. Most significantly, results from these assays elucidate two mechanisms for the inhibition of POT1 –DNA interactions. The first is by competitive inhibition at the POT1 DNA binding site. The second is indirect and is by stabilization of G-quadruplex formation within the normal POT1 single-stranded DNA sequence to prevent POT1 binding.<\/jats:p>"},{"1":"PLOS ONE","2":"2021-03-30","3":"2021-03-30","4":"2021-03-30","5":"10.1371/journal.pone.0248941","6":"2021-03-30","7":"1932-6203","8":"3","9":"2021-03-30","10":"340","11":"e0248941","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"80","17":"80","18":"0","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Selective killing of homologous recombination-deficient cancer cell lines by inhibitors of the RPA:RAD52 protein-protein interaction","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0248941","24":"16","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000002","29":"National Institutes of Health","30":"publisher","31":"P30CA036727","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,10]>","39":"<jats:p>Synthetic lethality is a successful strategy employed to develop selective chemotherapeutics against cancer cells. Inactivation of RAD52 is synthetically lethal to homologous recombination (HR) deficient cancer cell lines. Replication protein A (RPA) recruits RAD52 to repair sites, and the formation of this protein-protein complex is critical for RAD52 activity. To discover small molecules that inhibit the RPA:RAD52 protein-protein interaction (PPI), we screened chemical libraries with our newly developed Fluorescence-based protein-protein Interaction Assay (FluorIA). Eleven compounds were identified, including FDA-approved drugs (quinacrine, mitoxantrone, and doxorubicin). The FluorIA was used to rank the compounds by their ability to inhibit the RPA:RAD52 PPI and showed mitoxantrone and doxorubicin to be the most effective. Initial studies using the three FDA-approved drugs showed selective killing of BRCA1-mutated breast cancer cells (HCC1937), BRCA2-mutated ovarian cancer cells (PE01), and BRCA1-mutated ovarian cancer cells (UWB1.289). It was noteworthy that selective killing was seen in cells known to be resistant to PARP inhibitors (HCC1937 and UWB1 SYr13). A cell-based double-strand break (DSB) repair assay indicated that mitoxantrone significantly suppressed RAD52-dependent single-strand annealing (SSA) and mitoxantrone treatment disrupted the RPA:RAD52 PPI in cells. Furthermore, mitoxantrone reduced radiation-induced foci-formation of RAD52 with no significant activity against RAD51 foci formation. The results indicate that the RPA:RAD52 PPI could be a therapeutic target for HR-deficient cancers. These data also suggest that RAD52 is one of the targets of mitoxantrone and related compounds.<\/jats:p>"},{"1":"PLOS ONE","2":"2021-03-30","3":"2021-03-30","4":"2021-03-30","5":"10.1371/journal.pone.0248941","6":"2021-03-30","7":"1932-6203","8":"3","9":"2021-03-30","10":"340","11":"e0248941","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"80","17":"80","18":"0","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Selective killing of homologous recombination-deficient cancer cell lines by inhibitors of the RPA:RAD52 protein-protein interaction","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0248941","24":"16","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100010537","29":"Nebraska Department of Health and Human Services","30":"publisher","31":"NA","32":"2015","33":"2018-11","34":"2017-08","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,10]>","39":"<jats:p>Synthetic lethality is a successful strategy employed to develop selective chemotherapeutics against cancer cells. Inactivation of RAD52 is synthetically lethal to homologous recombination (HR) deficient cancer cell lines. Replication protein A (RPA) recruits RAD52 to repair sites, and the formation of this protein-protein complex is critical for RAD52 activity. To discover small molecules that inhibit the RPA:RAD52 protein-protein interaction (PPI), we screened chemical libraries with our newly developed Fluorescence-based protein-protein Interaction Assay (FluorIA). Eleven compounds were identified, including FDA-approved drugs (quinacrine, mitoxantrone, and doxorubicin). The FluorIA was used to rank the compounds by their ability to inhibit the RPA:RAD52 PPI and showed mitoxantrone and doxorubicin to be the most effective. Initial studies using the three FDA-approved drugs showed selective killing of BRCA1-mutated breast cancer cells (HCC1937), BRCA2-mutated ovarian cancer cells (PE01), and BRCA1-mutated ovarian cancer cells (UWB1.289). It was noteworthy that selective killing was seen in cells known to be resistant to PARP inhibitors (HCC1937 and UWB1 SYr13). A cell-based double-strand break (DSB) repair assay indicated that mitoxantrone significantly suppressed RAD52-dependent single-strand annealing (SSA) and mitoxantrone treatment disrupted the RPA:RAD52 PPI in cells. Furthermore, mitoxantrone reduced radiation-induced foci-formation of RAD52 with no significant activity against RAD51 foci formation. The results indicate that the RPA:RAD52 PPI could be a therapeutic target for HR-deficient cancers. These data also suggest that RAD52 is one of the targets of mitoxantrone and related compounds.<\/jats:p>"},{"1":"PLOS ONE","2":"2021-03-30","3":"2021-03-30","4":"2021-03-30","5":"10.1371/journal.pone.0248941","6":"2021-03-30","7":"1932-6203","8":"3","9":"2021-03-30","10":"340","11":"e0248941","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"80","17":"80","18":"0","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Selective killing of homologous recombination-deficient cancer cell lines by inhibitors of the RPA:RAD52 protein-protein interaction","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0248941","24":"16","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100000138","29":"U.S. Department of Education","30":"publisher","31":"P200A120231","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,10]>","39":"<jats:p>Synthetic lethality is a successful strategy employed to develop selective chemotherapeutics against cancer cells. Inactivation of RAD52 is synthetically lethal to homologous recombination (HR) deficient cancer cell lines. Replication protein A (RPA) recruits RAD52 to repair sites, and the formation of this protein-protein complex is critical for RAD52 activity. To discover small molecules that inhibit the RPA:RAD52 protein-protein interaction (PPI), we screened chemical libraries with our newly developed Fluorescence-based protein-protein Interaction Assay (FluorIA). Eleven compounds were identified, including FDA-approved drugs (quinacrine, mitoxantrone, and doxorubicin). The FluorIA was used to rank the compounds by their ability to inhibit the RPA:RAD52 PPI and showed mitoxantrone and doxorubicin to be the most effective. Initial studies using the three FDA-approved drugs showed selective killing of BRCA1-mutated breast cancer cells (HCC1937), BRCA2-mutated ovarian cancer cells (PE01), and BRCA1-mutated ovarian cancer cells (UWB1.289). It was noteworthy that selective killing was seen in cells known to be resistant to PARP inhibitors (HCC1937 and UWB1 SYr13). A cell-based double-strand break (DSB) repair assay indicated that mitoxantrone significantly suppressed RAD52-dependent single-strand annealing (SSA) and mitoxantrone treatment disrupted the RPA:RAD52 PPI in cells. Furthermore, mitoxantrone reduced radiation-induced foci-formation of RAD52 with no significant activity against RAD51 foci formation. The results indicate that the RPA:RAD52 PPI could be a therapeutic target for HR-deficient cancers. These data also suggest that RAD52 is one of the targets of mitoxantrone and related compounds.<\/jats:p>"},{"1":"PLOS ONE","2":"2021-03-30","3":"2021-03-30","4":"2021-03-30","5":"10.1371/journal.pone.0248941","6":"2021-03-30","7":"1932-6203","8":"3","9":"2021-03-30","10":"340","11":"e0248941","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"80","17":"80","18":"0","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Selective killing of homologous recombination-deficient cancer cell lines by inhibitors of the RPA:RAD52 protein-protein interaction","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0248941","24":"16","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"10.13039/100005749","29":"Nebraska Space Grant Consortium","30":"publisher","31":"NA","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,10]>","39":"<jats:p>Synthetic lethality is a successful strategy employed to develop selective chemotherapeutics against cancer cells. Inactivation of RAD52 is synthetically lethal to homologous recombination (HR) deficient cancer cell lines. Replication protein A (RPA) recruits RAD52 to repair sites, and the formation of this protein-protein complex is critical for RAD52 activity. To discover small molecules that inhibit the RPA:RAD52 protein-protein interaction (PPI), we screened chemical libraries with our newly developed Fluorescence-based protein-protein Interaction Assay (FluorIA). Eleven compounds were identified, including FDA-approved drugs (quinacrine, mitoxantrone, and doxorubicin). The FluorIA was used to rank the compounds by their ability to inhibit the RPA:RAD52 PPI and showed mitoxantrone and doxorubicin to be the most effective. Initial studies using the three FDA-approved drugs showed selective killing of BRCA1-mutated breast cancer cells (HCC1937), BRCA2-mutated ovarian cancer cells (PE01), and BRCA1-mutated ovarian cancer cells (UWB1.289). It was noteworthy that selective killing was seen in cells known to be resistant to PARP inhibitors (HCC1937 and UWB1 SYr13). A cell-based double-strand break (DSB) repair assay indicated that mitoxantrone significantly suppressed RAD52-dependent single-strand annealing (SSA) and mitoxantrone treatment disrupted the RPA:RAD52 PPI in cells. Furthermore, mitoxantrone reduced radiation-induced foci-formation of RAD52 with no significant activity against RAD51 foci formation. The results indicate that the RPA:RAD52 PPI could be a therapeutic target for HR-deficient cancers. These data also suggest that RAD52 is one of the targets of mitoxantrone and related compounds.<\/jats:p>"},{"1":"PLOS ONE","2":"2021-03-30","3":"2021-03-30","4":"2021-03-30","5":"10.1371/journal.pone.0248941","6":"2021-03-30","7":"1932-6203","8":"3","9":"2021-03-30","10":"340","11":"e0248941","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"80","17":"80","18":"0","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Selective killing of homologous recombination-deficient cancer cell lines by inhibitors of the RPA:RAD52 protein-protein interaction","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0248941","24":"16","25":"en","26":"PLoS ONE","27":"<tibble[,5]>","28":"NA","29":"Nebraska Research Initiative","30":"NA","31":"NA","32":"NA","33":"NA","34":"NA","35":"<tibble[,4]>","36":"<df[,2] [1 × 2]>","37":"<tibble[,4]>","38":"<tibble[,10]>","39":"<jats:p>Synthetic lethality is a successful strategy employed to develop selective chemotherapeutics against cancer cells. Inactivation of RAD52 is synthetically lethal to homologous recombination (HR) deficient cancer cell lines. Replication protein A (RPA) recruits RAD52 to repair sites, and the formation of this protein-protein complex is critical for RAD52 activity. To discover small molecules that inhibit the RPA:RAD52 protein-protein interaction (PPI), we screened chemical libraries with our newly developed Fluorescence-based protein-protein Interaction Assay (FluorIA). Eleven compounds were identified, including FDA-approved drugs (quinacrine, mitoxantrone, and doxorubicin). The FluorIA was used to rank the compounds by their ability to inhibit the RPA:RAD52 PPI and showed mitoxantrone and doxorubicin to be the most effective. Initial studies using the three FDA-approved drugs showed selective killing of BRCA1-mutated breast cancer cells (HCC1937), BRCA2-mutated ovarian cancer cells (PE01), and BRCA1-mutated ovarian cancer cells (UWB1.289). It was noteworthy that selective killing was seen in cells known to be resistant to PARP inhibitors (HCC1937 and UWB1 SYr13). A cell-based double-strand break (DSB) repair assay indicated that mitoxantrone significantly suppressed RAD52-dependent single-strand annealing (SSA) and mitoxantrone treatment disrupted the RPA:RAD52 PPI in cells. Furthermore, mitoxantrone reduced radiation-induced foci-formation of RAD52 with no significant activity against RAD51 foci formation. The results indicate that the RPA:RAD52 PPI could be a therapeutic target for HR-deficient cancers. These data also suggest that RAD52 is one of the targets of mitoxantrone and related compounds.<\/jats:p>"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>If you scroll all the way to the right, you can see the funder information. Look at the <code>title</code> column and you will notice that some article titles are now duplicated, however you will see different funders in the <code>name</code> column. This is because a single article may have multiple funders, and a new row is created for each funder, with data including the <code>award</code> number.</p>
</div>
<div id="filtering-by-license-with-has_license" class="section level3">
<h3>Filtering by license with <code>has_license</code></h3>
<p>You may be interested in licensing information for articles; for instance, gathering publications in a given journal that are licensed under Creative Commons. First run <code>cr_journals</code> with <code>works</code> set to <code>FALSE</code> in order to return journal details so you can check if the publisher even sends article licensing information to Crossref–it’s not required. We will use PLOS ONE again as an example.</p>
<pre class="r"><code># assign the PLoS ONE ISSN and get journal details by setting works = FALSE
plosone_issn &lt;- &quot;1932-6203&quot;
plosone_details &lt;- rcrossref::cr_journals(issn = plosone_issn, works = FALSE) %&gt;%
    purrr::pluck(&quot;data&quot;)</code></pre>
<p>We can check the <code>deposits_licenses_current</code> variables to see if license data on file is current. If it is <code>TRUE</code>, PLoS ONE does send licensing information and it is current.</p>
<pre class="r"><code># is article licensing data on file current?
plosone_details$deposits_licenses_current</code></pre>
<pre><code>## [1] TRUE</code></pre>
<p>We can now rerun the query but set <code>works = TRUE</code>, and set the <code>has_license</code> to <code>TRUE</code>. This will therefore return only articles that have license information. We will set our <code>limit</code> to 25.</p>
<pre class="r"><code># get last 25 articles on file where has_license is TRUE
plosone_license &lt;- rcrossref::cr_journals(issn = plosone_issn, works = T, limit = 25,
    filter = c(has_license = TRUE)) %&gt;%
    pluck(&quot;data&quot;)</code></pre>
<pre class="r"><code># print the data with select columns
plosone_license %&gt;%
    dplyr::select(title, doi, volume, issue, page, issued, url, publisher, reference.count,
        type, issn, license)</code></pre>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["title"],"name":[1],"type":["chr"],"align":["left"]},{"label":["doi"],"name":[2],"type":["chr"],"align":["left"]},{"label":["volume"],"name":[3],"type":["chr"],"align":["left"]},{"label":["issue"],"name":[4],"type":["chr"],"align":["left"]},{"label":["page"],"name":[5],"type":["chr"],"align":["left"]},{"label":["issued"],"name":[6],"type":["chr"],"align":["left"]},{"label":["url"],"name":[7],"type":["chr"],"align":["left"]},{"label":["publisher"],"name":[8],"type":["chr"],"align":["left"]},{"label":["reference.count"],"name":[9],"type":["chr"],"align":["left"]},{"label":["type"],"name":[10],"type":["chr"],"align":["left"]},{"label":["issn"],"name":[11],"type":["chr"],"align":["left"]},{"label":["license"],"name":[12],"type":["list"],"align":["right"]}],"data":[{"1":"Multiple Sporadic Colorectal Cancers Display a Unique Methylation Phenotype","2":"10.1371/journal.pone.0091033","3":"9","4":"3","5":"e91033","6":"2014-03-18","7":"http://dx.doi.org/10.1371/journal.pone.0091033","8":"Public Library of Science (PLoS)","9":"42","10":"journal-article","11":"1932-6203","12":"<tibble[,4]>"},{"1":"Need for Space: The Key Distance Effect Depends on Spatial Stimulus Configurations","2":"10.1371/journal.pone.0091432","3":"9","4":"3","5":"e91432","6":"2014-03-18","7":"http://dx.doi.org/10.1371/journal.pone.0091432","8":"Public Library of Science (PLoS)","9":"24","10":"journal-article","11":"1932-6203","12":"<tibble[,4]>"},{"1":"Correction: MicroRNA-130b Suppresses Migration and Invasion of Colorectal Cancer Cells through Downregulation of Integrin β1","2":"10.1371/journal.pone.0091498","3":"9","4":"3","5":"e91498","6":"2014-03-18","7":"http://dx.doi.org/10.1371/journal.pone.0091498","8":"Public Library of Science (PLoS)","9":"0","10":"journal-article","11":"1932-6203","12":"<tibble[,4]>"},{"1":"Oral Health Training Programs for Community and Professional Health Care Workers in Nairobi East District Increases Identification of HIV-Infected Patients","2":"10.1371/journal.pone.0090927","3":"9","4":"3","5":"e90927","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0090927","8":"Public Library of Science (PLoS)","9":"22","10":"journal-article","11":"1932-6203","12":"<tibble[,4]>"},{"1":"Correction: Association of Polymorphisms and Haplotypes in the Insulin-Like Growth Factor 1 Receptor (IGF1R) Gene with the Risk of Breast Cancer in Korean Women","2":"10.1371/journal.pone.0091218","3":"9","4":"3","5":"e91218","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0091218","8":"Public Library of Science (PLoS)","9":"0","10":"journal-article","11":"1932-6203","12":"<tibble[,4]>"},{"1":"Epidemiological and Etiological Characteristics of Fever, Thrombocytopenia and Leukopenia Syndrome in Henan Province, China, 2011–2012","2":"10.1371/journal.pone.0091166","3":"9","4":"3","5":"e91166","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0091166","8":"Public Library of Science (PLoS)","9":"26","10":"journal-article","11":"1932-6203","12":"<tibble[,4]>"},{"1":"Correction: Network Analysis Reveals Ecological Links between N-Fixing Bacteria and Wood-Decaying Fungi","2":"10.1371/journal.pone.0091389","3":"9","4":"3","5":"e91389","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0091389","8":"Public Library of Science (PLoS)","9":"0","10":"journal-article","11":"1932-6203","12":"<tibble[,4]>"},{"1":"Correction: CXCR7 Is Highly Expressed in Acute Lymphoblastic Leukemia and Potentiates CXCR4 Response to CXCL12","2":"10.1371/journal.pone.0091365","3":"9","4":"3","5":"e91365","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0091365","8":"Public Library of Science (PLoS)","9":"0","10":"journal-article","11":"1932-6203","12":"<tibble[,4]>"},{"1":"Stochastic Resonance with Colored Noise for Neural Signal Detection","2":"10.1371/journal.pone.0091345","3":"9","4":"3","5":"e91345","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0091345","8":"Public Library of Science (PLoS)","9":"63","10":"journal-article","11":"1932-6203","12":"<tibble[,4]>"},{"1":"Scaling and Predictability in Stock Markets: A Comparative Study","2":"10.1371/journal.pone.0091707","3":"9","4":"3","5":"e91707","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0091707","8":"Public Library of Science (PLoS)","9":"26","10":"journal-article","11":"1932-6203","12":"<tibble[,4]>"},{"1":"Correction: Density, Destinations or Both? A Comparison of Measures of Walkability in Relation to Transportation Behaviors, Obesity and Diabetes in Toronto, Canada","2":"10.1371/journal.pone.0091485","3":"9","4":"3","5":"e91485","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0091485","8":"Public Library of Science (PLoS)","9":"0","10":"journal-article","11":"1932-6203","12":"<tibble[,4]>"},{"1":"Molecular Characterization of Adipose Tissue in the African Elephant (Loxodonta africana)","2":"10.1371/journal.pone.0091717","3":"9","4":"3","5":"e91717","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0091717","8":"Public Library of Science (PLoS)","9":"74","10":"journal-article","11":"1932-6203","12":"<tibble[,4]>"},{"1":"Immunization with H7-HCP-Tir-Intimin Significantly Reduces Colonization and Shedding of Escherichia coli O157:H7 in Goats","2":"10.1371/journal.pone.0091632","3":"9","4":"3","5":"e91632","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0091632","8":"Public Library of Science (PLoS)","9":"34","10":"journal-article","11":"1932-6203","12":"<tibble[,4]>"},{"1":"Characterization of the Altered Gene Expression Profile in Early Porcine Embryos Generated from Parthenogenesis and Somatic Cell Chromatin Transfer","2":"10.1371/journal.pone.0091728","3":"9","4":"3","5":"e91728","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0091728","8":"Public Library of Science (PLoS)","9":"71","10":"journal-article","11":"1932-6203","12":"<tibble[,4]>"},{"1":"Preferential Occupancy of R2 Retroelements on the B Chromosomes of the Grasshopper Eyprepocnemis plorans","2":"10.1371/journal.pone.0091820","3":"9","4":"3","5":"e91820","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0091820","8":"Public Library of Science (PLoS)","9":"81","10":"journal-article","11":"1932-6203","12":"<tibble[,4]>"},{"1":"Quantitative Analysis of Cone Photoreceptor Distribution and Its Relationship with Axial Length, Age, and Early Age-Related Macular Degeneration","2":"10.1371/journal.pone.0091873","3":"9","4":"3","5":"e91873","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0091873","8":"Public Library of Science (PLoS)","9":"51","10":"journal-article","11":"1932-6203","12":"<tibble[,4]>"},{"1":"Can Training Enhance Face Cognition Abilities in Middle-Aged Adults?","2":"10.1371/journal.pone.0090249","3":"9","4":"3","5":"e90249","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0090249","8":"Public Library of Science (PLoS)","9":"84","10":"journal-article","11":"1932-6203","12":"<tibble[,4]>"},{"1":"A Spiking Network Model of Decision Making Employing Rewarded STDP","2":"10.1371/journal.pone.0090821","3":"9","4":"3","5":"e90821","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0090821","8":"Public Library of Science (PLoS)","9":"85","10":"journal-article","11":"1932-6203","12":"<tibble[,4]>"},{"1":"Association between a C8orf13–BLK Polymorphism and Polymyositis/Dermatomyositis in the Japanese Population: An Additive Effect with STAT4 on Disease Susceptibility","2":"10.1371/journal.pone.0090019","3":"9","4":"3","5":"e90019","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0090019","8":"Public Library of Science (PLoS)","9":"28","10":"journal-article","11":"1932-6203","12":"<tibble[,4]>"},{"1":"Signaling Role of Prokineticin 2 on the Estrous Cycle of Female Mice","2":"10.1371/journal.pone.0090860","3":"9","4":"3","5":"e90860","6":"2014-03-14","7":"http://dx.doi.org/10.1371/journal.pone.0090860","8":"Public Library of Science (PLoS)","9":"45","10":"journal-article","11":"1932-6203","12":"<tibble[,4]>"},{"1":"Correction: Morphological Characterization and Quantification of the Mycelial Growth of the Brown-Rot Fungus Postia placenta for Modeling Purposes","2":"10.1371/journal.pone.0189548","3":"13","4":"5","5":"e0189548","6":"2018-05-30","7":"http://dx.doi.org/10.1371/journal.pone.0189548","8":"Public Library of Science (PLoS)","9":"1","10":"journal-article","11":"1932-6203","12":"<tibble[,4]>"},{"1":"Correction: Effects of resource-dependent cannibalism on population size distribution and individual life history in a case-bearing caddisfly","2":"10.1371/journal.pone.0197014","3":"13","4":"5","5":"e0197014","6":"2018-05-30","7":"http://dx.doi.org/10.1371/journal.pone.0197014","8":"Public Library of Science (PLoS)","9":"1","10":"journal-article","11":"1932-6203","12":"<tibble[,4]>"},{"1":"Internists’ dilemmas in their interactions with chronically ill patients; A comparison of their interaction strategies and dilemmas in two different medical contexts","2":"10.1371/journal.pone.0194133","3":"13","4":"5","5":"e0194133","6":"2018-05-30","7":"http://dx.doi.org/10.1371/journal.pone.0194133","8":"Public Library of Science (PLoS)","9":"71","10":"journal-article","11":"1932-6203","12":"<tibble[,4]>"},{"1":"Managing the 1920s’ Chilean educational crisis: A historical view combined with machine learning","2":"10.1371/journal.pone.0197429","3":"13","4":"5","5":"e0197429","6":"2018-05-30","7":"http://dx.doi.org/10.1371/journal.pone.0197429","8":"Public Library of Science (PLoS)","9":"42","10":"journal-article","11":"1932-6203","12":"<tibble[,4]>"},{"1":"Can excreted thiocyanate be used to detect cyanide exposure in live reef fish?","2":"10.1371/journal.pone.0196841","3":"13","4":"5","5":"e0196841","6":"2018-05-30","7":"http://dx.doi.org/10.1371/journal.pone.0196841","8":"Public Library of Science (PLoS)","9":"24","10":"journal-article","11":"1932-6203","12":"<tibble[,4]>"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>The license data comes in as a nested column. We can unnest it using <code>tidyr::unnest</code>, which we used above with funders and will be discussed more below.</p>
<pre class="r"><code># print the data frame with license unnested. The .drop argument will drop all
# other list columns.
plosone_license %&gt;%
    tidyr::unnest(license, .drop = TRUE)</code></pre>
<pre><code>## Warning: The `.drop` argument of `unnest()` is deprecated as of tidyr 1.0.0.
## All list-columns are now preserved.</code></pre>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["container.title"],"name":[1],"type":["chr"],"align":["left"]},{"label":["created"],"name":[2],"type":["chr"],"align":["left"]},{"label":["deposited"],"name":[3],"type":["chr"],"align":["left"]},{"label":["published.online"],"name":[4],"type":["chr"],"align":["left"]},{"label":["doi"],"name":[5],"type":["chr"],"align":["left"]},{"label":["indexed"],"name":[6],"type":["chr"],"align":["left"]},{"label":["issn"],"name":[7],"type":["chr"],"align":["left"]},{"label":["issue"],"name":[8],"type":["chr"],"align":["left"]},{"label":["issued"],"name":[9],"type":["chr"],"align":["left"]},{"label":["member"],"name":[10],"type":["chr"],"align":["left"]},{"label":["page"],"name":[11],"type":["chr"],"align":["left"]},{"label":["prefix"],"name":[12],"type":["chr"],"align":["left"]},{"label":["publisher"],"name":[13],"type":["chr"],"align":["left"]},{"label":["score"],"name":[14],"type":["chr"],"align":["left"]},{"label":["source"],"name":[15],"type":["chr"],"align":["left"]},{"label":["reference.count"],"name":[16],"type":["chr"],"align":["left"]},{"label":["references.count"],"name":[17],"type":["chr"],"align":["left"]},{"label":["is.referenced.by.count"],"name":[18],"type":["chr"],"align":["left"]},{"label":["title"],"name":[19],"type":["chr"],"align":["left"]},{"label":["type"],"name":[20],"type":["chr"],"align":["left"]},{"label":["update.policy"],"name":[21],"type":["chr"],"align":["left"]},{"label":["url"],"name":[22],"type":["chr"],"align":["left"]},{"label":["volume"],"name":[23],"type":["chr"],"align":["left"]},{"label":["language"],"name":[24],"type":["chr"],"align":["left"]},{"label":["short.container.title"],"name":[25],"type":["chr"],"align":["left"]},{"label":["author"],"name":[26],"type":["list"],"align":["right"]},{"label":["link"],"name":[27],"type":["list"],"align":["right"]},{"label":["content_domain"],"name":[28],"type":["list"],"align":["right"]},{"label":["date"],"name":[29],"type":["chr"],"align":["left"]},{"label":["URL"],"name":[30],"type":["chr"],"align":["left"]},{"label":["delay.in.days"],"name":[31],"type":["int"],"align":["right"]},{"label":["content.version"],"name":[32],"type":["chr"],"align":["left"]},{"label":["reference"],"name":[33],"type":["list"],"align":["right"]},{"label":["update_to"],"name":[34],"type":["list"],"align":["right"]},{"label":["clinical-trial-number"],"name":[35],"type":["list"],"align":["right"]},{"label":["funder"],"name":[36],"type":["list"],"align":["right"]}],"data":[{"1":"PLoS ONE","2":"2014-03-18","3":"2019-08-08","4":"2014-03-18","5":"10.1371/journal.pone.0091033","6":"2020-03-30","7":"1932-6203","8":"3","9":"2014-03-18","10":"340","11":"e91033","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"42","17":"42","18":"6","19":"Multiple Sporadic Colorectal Cancers Display a Unique Methylation Phenotype","20":"journal-article","21":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","22":"http://dx.doi.org/10.1371/journal.pone.0091033","23":"9","24":"en","25":"PLoS ONE","26":"<tibble[,4]>","27":"<tibble[,4]>","28":"<df[,2] [1 × 2]>","29":"2014-03-18","30":"http://creativecommons.org/licenses/by/4.0/","31":"0","32":"unspecified","33":"<tibble[,9]>","34":"<NULL>","35":"<NULL>","36":"<NULL>"},{"1":"PLoS ONE","2":"2014-03-18","3":"2018-10-22","4":"2014-03-18","5":"10.1371/journal.pone.0091432","6":"2020-03-30","7":"1932-6203","8":"3","9":"2014-03-18","10":"340","11":"e91432","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"24","17":"24","18":"3","19":"Need for Space: The Key Distance Effect Depends on Spatial Stimulus Configurations","20":"journal-article","21":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","22":"http://dx.doi.org/10.1371/journal.pone.0091432","23":"9","24":"en","25":"PLoS ONE","26":"<tibble[,3]>","27":"<tibble[,4]>","28":"<df[,2] [1 × 2]>","29":"2014-03-18","30":"http://creativecommons.org/licenses/by/4.0/","31":"0","32":"unspecified","33":"<tibble[,10]>","34":"<NULL>","35":"<NULL>","36":"<NULL>"},{"1":"PLoS ONE","2":"2014-03-18","3":"2018-10-22","4":"2014-03-18","5":"10.1371/journal.pone.0091498","6":"2020-03-30","7":"1932-6203","8":"3","9":"2014-03-18","10":"340","11":"e91498","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"0","17":"0","18":"0","19":"Correction: MicroRNA-130b Suppresses Migration and Invasion of Colorectal Cancer Cells through Downregulation of Integrin β1","20":"journal-article","21":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","22":"http://dx.doi.org/10.1371/journal.pone.0091498","23":"9","24":"en","25":"PLoS ONE","26":"<tibble[,2]>","27":"<NULL>","28":"<df[,2] [1 × 2]>","29":"2014-03-18","30":"http://creativecommons.org/licenses/by/4.0/","31":"0","32":"unspecified","33":"<NULL>","34":"<tibble[,4]>","35":"<NULL>","36":"<NULL>"},{"1":"PLoS ONE","2":"2014-03-14","3":"2018-10-23","4":"2014-03-14","5":"10.1371/journal.pone.0090927","6":"2020-03-30","7":"1932-6203","8":"3","9":"2014-03-14","10":"340","11":"e90927","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"22","17":"22","18":"4","19":"Oral Health Training Programs for Community and Professional Health Care Workers in Nairobi East District Increases Identification of HIV-Infected Patients","20":"journal-article","21":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","22":"http://dx.doi.org/10.1371/journal.pone.0090927","23":"9","24":"en","25":"PLoS ONE","26":"<tibble[,3]>","27":"<tibble[,4]>","28":"<df[,2] [1 × 2]>","29":"2014-03-14","30":"http://creativecommons.org/licenses/by/4.0/","31":"0","32":"unspecified","33":"<tibble[,10]>","34":"<NULL>","35":"<tibble[,2]>","36":"<NULL>"},{"1":"PLoS ONE","2":"2014-03-14","3":"2018-10-23","4":"2014-03-14","5":"10.1371/journal.pone.0091218","6":"2020-03-30","7":"1932-6203","8":"3","9":"2014-03-14","10":"340","11":"e91218","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"0","17":"0","18":"0","19":"Correction: Association of Polymorphisms and Haplotypes in the Insulin-Like Growth Factor 1 Receptor (IGF1R) Gene with the Risk of Breast Cancer in Korean Women","20":"journal-article","21":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","22":"http://dx.doi.org/10.1371/journal.pone.0091218","23":"9","24":"en","25":"PLoS ONE","26":"<tibble[,2]>","27":"<NULL>","28":"<df[,2] [1 × 2]>","29":"2014-03-14","30":"http://creativecommons.org/licenses/by/4.0/","31":"0","32":"unspecified","33":"<NULL>","34":"<tibble[,4]>","35":"<NULL>","36":"<NULL>"},{"1":"PLoS ONE","2":"2014-03-14","3":"2018-10-22","4":"2014-03-14","5":"10.1371/journal.pone.0091166","6":"2020-03-30","7":"1932-6203","8":"3","9":"2014-03-14","10":"340","11":"e91166","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"26","17":"26","18":"10","19":"Epidemiological and Etiological Characteristics of Fever, Thrombocytopenia and Leukopenia Syndrome in Henan Province, China, 2011–2012","20":"journal-article","21":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","22":"http://dx.doi.org/10.1371/journal.pone.0091166","23":"9","24":"en","25":"PLoS ONE","26":"<tibble[,3]>","27":"<tibble[,4]>","28":"<df[,2] [1 × 2]>","29":"2014-03-14","30":"http://creativecommons.org/licenses/by/4.0/","31":"0","32":"unspecified","33":"<tibble[,10]>","34":"<NULL>","35":"<NULL>","36":"<NULL>"},{"1":"PLoS ONE","2":"2014-03-14","3":"2018-10-23","4":"2014-03-14","5":"10.1371/journal.pone.0091389","6":"2020-03-30","7":"1932-6203","8":"3","9":"2014-03-14","10":"340","11":"e91389","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"0","17":"0","18":"0","19":"Correction: Network Analysis Reveals Ecological Links between N-Fixing Bacteria and Wood-Decaying Fungi","20":"journal-article","21":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","22":"http://dx.doi.org/10.1371/journal.pone.0091389","23":"9","24":"en","25":"PLoS ONE","26":"<tibble[,2]>","27":"<NULL>","28":"<df[,2] [1 × 2]>","29":"2014-03-14","30":"http://creativecommons.org/licenses/by/4.0/","31":"0","32":"unspecified","33":"<NULL>","34":"<tibble[,4]>","35":"<NULL>","36":"<NULL>"},{"1":"PLoS ONE","2":"2014-03-14","3":"2018-10-23","4":"2014-03-14","5":"10.1371/journal.pone.0091365","6":"2020-03-30","7":"1932-6203","8":"3","9":"2014-03-14","10":"340","11":"e91365","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"0","17":"0","18":"0","19":"Correction: CXCR7 Is Highly Expressed in Acute Lymphoblastic Leukemia and Potentiates CXCR4 Response to CXCL12","20":"journal-article","21":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","22":"http://dx.doi.org/10.1371/journal.pone.0091365","23":"9","24":"en","25":"PLoS ONE","26":"<tibble[,2]>","27":"<NULL>","28":"<df[,2] [1 × 2]>","29":"2014-03-14","30":"http://creativecommons.org/licenses/by/4.0/","31":"0","32":"unspecified","33":"<NULL>","34":"<tibble[,4]>","35":"<NULL>","36":"<NULL>"},{"1":"PLoS ONE","2":"2014-03-14","3":"2019-08-08","4":"2014-03-14","5":"10.1371/journal.pone.0091345","6":"2020-03-30","7":"1932-6203","8":"3","9":"2014-03-14","10":"340","11":"e91345","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"63","17":"63","18":"12","19":"Stochastic Resonance with Colored Noise for Neural Signal Detection","20":"journal-article","21":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","22":"http://dx.doi.org/10.1371/journal.pone.0091345","23":"9","24":"en","25":"PLoS ONE","26":"<tibble[,3]>","27":"<tibble[,4]>","28":"<df[,2] [1 × 2]>","29":"2014-03-14","30":"http://creativecommons.org/licenses/by/4.0/","31":"0","32":"unspecified","33":"<tibble[,10]>","34":"<NULL>","35":"<NULL>","36":"<NULL>"},{"1":"PLoS ONE","2":"2014-03-14","3":"2019-08-08","4":"2014-03-14","5":"10.1371/journal.pone.0091707","6":"2020-03-30","7":"1932-6203","8":"3","9":"2014-03-14","10":"340","11":"e91707","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"26","17":"26","18":"2","19":"Scaling and Predictability in Stock Markets: A Comparative Study","20":"journal-article","21":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","22":"http://dx.doi.org/10.1371/journal.pone.0091707","23":"9","24":"en","25":"PLoS ONE","26":"<tibble[,3]>","27":"<tibble[,4]>","28":"<df[,2] [1 × 2]>","29":"2014-03-14","30":"http://creativecommons.org/licenses/by/4.0/","31":"0","32":"unspecified","33":"<tibble[,10]>","34":"<NULL>","35":"<NULL>","36":"<NULL>"},{"1":"PLoS ONE","2":"2014-03-14","3":"2018-10-22","4":"2014-03-14","5":"10.1371/journal.pone.0091485","6":"2020-03-30","7":"1932-6203","8":"3","9":"2014-03-14","10":"340","11":"e91485","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"0","17":"0","18":"0","19":"Correction: Density, Destinations or Both? A Comparison of Measures of Walkability in Relation to Transportation Behaviors, Obesity and Diabetes in Toronto, Canada","20":"journal-article","21":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","22":"http://dx.doi.org/10.1371/journal.pone.0091485","23":"9","24":"en","25":"PLoS ONE","26":"<tibble[,2]>","27":"<NULL>","28":"<df[,2] [1 × 2]>","29":"2014-03-14","30":"http://creativecommons.org/licenses/by/4.0/","31":"0","32":"unspecified","33":"<NULL>","34":"<tibble[,4]>","35":"<NULL>","36":"<NULL>"},{"1":"PLoS ONE","2":"2014-03-14","3":"2019-08-08","4":"2014-03-14","5":"10.1371/journal.pone.0091717","6":"2020-03-30","7":"1932-6203","8":"3","9":"2014-03-14","10":"340","11":"e91717","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"74","17":"74","18":"2","19":"Molecular Characterization of Adipose Tissue in the African Elephant (Loxodonta africana)","20":"journal-article","21":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","22":"http://dx.doi.org/10.1371/journal.pone.0091717","23":"9","24":"en","25":"PLoS ONE","26":"<tibble[,3]>","27":"<tibble[,4]>","28":"<df[,2] [1 × 2]>","29":"2014-03-14","30":"http://creativecommons.org/licenses/by/4.0/","31":"0","32":"unspecified","33":"<tibble[,10]>","34":"<NULL>","35":"<NULL>","36":"<NULL>"},{"1":"PLoS ONE","2":"2014-03-14","3":"2019-08-08","4":"2014-03-14","5":"10.1371/journal.pone.0091632","6":"2020-03-30","7":"1932-6203","8":"3","9":"2014-03-14","10":"340","11":"e91632","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"34","17":"34","18":"10","19":"Immunization with H7-HCP-Tir-Intimin Significantly Reduces Colonization and Shedding of Escherichia coli O157:H7 in Goats","20":"journal-article","21":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","22":"http://dx.doi.org/10.1371/journal.pone.0091632","23":"9","24":"en","25":"PLoS ONE","26":"<tibble[,3]>","27":"<tibble[,4]>","28":"<df[,2] [1 × 2]>","29":"2014-03-14","30":"http://creativecommons.org/licenses/by/4.0/","31":"0","32":"unspecified","33":"<tibble[,11]>","34":"<NULL>","35":"<NULL>","36":"<NULL>"},{"1":"PLoS ONE","2":"2014-03-14","3":"2019-08-08","4":"2014-03-14","5":"10.1371/journal.pone.0091728","6":"2020-03-30","7":"1932-6203","8":"3","9":"2014-03-14","10":"340","11":"e91728","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"71","17":"71","18":"5","19":"Characterization of the Altered Gene Expression Profile in Early Porcine Embryos Generated from Parthenogenesis and Somatic Cell Chromatin Transfer","20":"journal-article","21":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","22":"http://dx.doi.org/10.1371/journal.pone.0091728","23":"9","24":"en","25":"PLoS ONE","26":"<tibble[,3]>","27":"<tibble[,4]>","28":"<df[,2] [1 × 2]>","29":"2014-03-14","30":"http://creativecommons.org/licenses/by/4.0/","31":"0","32":"unspecified","33":"<tibble[,10]>","34":"<NULL>","35":"<NULL>","36":"<NULL>"},{"1":"PLoS ONE","2":"2014-03-14","3":"2019-08-08","4":"2014-03-14","5":"10.1371/journal.pone.0091820","6":"2020-03-30","7":"1932-6203","8":"3","9":"2014-03-14","10":"340","11":"e91820","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"81","17":"81","18":"9","19":"Preferential Occupancy of R2 Retroelements on the B Chromosomes of the Grasshopper Eyprepocnemis plorans","20":"journal-article","21":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","22":"http://dx.doi.org/10.1371/journal.pone.0091820","23":"9","24":"en","25":"PLoS ONE","26":"<tibble[,3]>","27":"<tibble[,4]>","28":"<df[,2] [1 × 2]>","29":"2014-03-14","30":"http://creativecommons.org/licenses/by/4.0/","31":"0","32":"unspecified","33":"<tibble[,10]>","34":"<NULL>","35":"<NULL>","36":"<NULL>"},{"1":"PLoS ONE","2":"2014-03-14","3":"2018-10-20","4":"2014-03-14","5":"10.1371/journal.pone.0091873","6":"2020-03-30","7":"1932-6203","8":"3","9":"2014-03-14","10":"340","11":"e91873","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"51","17":"51","18":"17","19":"Quantitative Analysis of Cone Photoreceptor Distribution and Its Relationship with Axial Length, Age, and Early Age-Related Macular Degeneration","20":"journal-article","21":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","22":"http://dx.doi.org/10.1371/journal.pone.0091873","23":"9","24":"en","25":"PLoS ONE","26":"<tibble[,3]>","27":"<tibble[,4]>","28":"<df[,2] [1 × 2]>","29":"2014-03-14","30":"http://creativecommons.org/licenses/by/4.0/","31":"0","32":"unspecified","33":"<tibble[,10]>","34":"<NULL>","35":"<NULL>","36":"<NULL>"},{"1":"PLoS ONE","2":"2014-03-14","3":"2019-08-08","4":"2014-03-14","5":"10.1371/journal.pone.0090249","6":"2020-03-30","7":"1932-6203","8":"3","9":"2014-03-14","10":"340","11":"e90249","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"84","17":"84","18":"6","19":"Can Training Enhance Face Cognition Abilities in Middle-Aged Adults?","20":"journal-article","21":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","22":"http://dx.doi.org/10.1371/journal.pone.0090249","23":"9","24":"en","25":"PLoS ONE","26":"<tibble[,3]>","27":"<tibble[,4]>","28":"<df[,2] [1 × 2]>","29":"2014-03-14","30":"http://creativecommons.org/licenses/by/4.0/","31":"0","32":"unspecified","33":"<tibble[,10]>","34":"<NULL>","35":"<NULL>","36":"<NULL>"},{"1":"PLoS ONE","2":"2014-03-14","3":"2019-08-08","4":"2014-03-14","5":"10.1371/journal.pone.0090821","6":"2020-03-30","7":"1932-6203","8":"3","9":"2014-03-14","10":"340","11":"e90821","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"85","17":"85","18":"12","19":"A Spiking Network Model of Decision Making Employing Rewarded STDP","20":"journal-article","21":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","22":"http://dx.doi.org/10.1371/journal.pone.0090821","23":"9","24":"en","25":"PLoS ONE","26":"<tibble[,3]>","27":"<tibble[,4]>","28":"<df[,2] [1 × 2]>","29":"2014-03-14","30":"http://creativecommons.org/licenses/by/4.0/","31":"0","32":"unspecified","33":"<tibble[,10]>","34":"<NULL>","35":"<NULL>","36":"<NULL>"},{"1":"PLoS ONE","2":"2014-03-14","3":"2018-10-23","4":"2014-03-14","5":"10.1371/journal.pone.0090019","6":"2020-03-30","7":"1932-6203","8":"3","9":"2014-03-14","10":"340","11":"e90019","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"28","17":"28","18":"12","19":"Association between a C8orf13–BLK Polymorphism and Polymyositis/Dermatomyositis in the Japanese Population: An Additive Effect with STAT4 on Disease Susceptibility","20":"journal-article","21":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","22":"http://dx.doi.org/10.1371/journal.pone.0090019","23":"9","24":"en","25":"PLoS ONE","26":"<tibble[,3]>","27":"<tibble[,4]>","28":"<df[,2] [1 × 2]>","29":"2014-03-14","30":"http://creativecommons.org/licenses/by/4.0/","31":"0","32":"unspecified","33":"<tibble[,9]>","34":"<NULL>","35":"<NULL>","36":"<NULL>"},{"1":"PLoS ONE","2":"2014-03-14","3":"2019-08-08","4":"2014-03-14","5":"10.1371/journal.pone.0090860","6":"2020-03-30","7":"1932-6203","8":"3","9":"2014-03-14","10":"340","11":"e90860","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"45","17":"45","18":"5","19":"Signaling Role of Prokineticin 2 on the Estrous Cycle of Female Mice","20":"journal-article","21":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","22":"http://dx.doi.org/10.1371/journal.pone.0090860","23":"9","24":"en","25":"PLoS ONE","26":"<tibble[,3]>","27":"<tibble[,4]>","28":"<df[,2] [1 × 2]>","29":"2014-03-14","30":"http://creativecommons.org/licenses/by/4.0/","31":"0","32":"unspecified","33":"<tibble[,9]>","34":"<NULL>","35":"<NULL>","36":"<NULL>"},{"1":"PLOS ONE","2":"2018-05-30","3":"2018-10-23","4":"2018-05-30","5":"10.1371/journal.pone.0189548","6":"2020-03-30","7":"1932-6203","8":"5","9":"2018-05-30","10":"340","11":"e0189548","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"1","17":"1","18":"0","19":"Correction: Morphological Characterization and Quantification of the Mycelial Growth of the Brown-Rot Fungus Postia placenta for Modeling Purposes","20":"journal-article","21":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","22":"http://dx.doi.org/10.1371/journal.pone.0189548","23":"13","24":"en","25":"PLoS ONE","26":"<tibble[,3]>","27":"<tibble[,4]>","28":"<df[,2] [1 × 2]>","29":"2018-05-30","30":"http://creativecommons.org/licenses/by/4.0/","31":"0","32":"vor","33":"<tibble[,10]>","34":"<tibble[,4]>","35":"<NULL>","36":"<NULL>"},{"1":"PLOS ONE","2":"2018-05-30","3":"2018-10-23","4":"2018-05-30","5":"10.1371/journal.pone.0197014","6":"2020-03-30","7":"1932-6203","8":"5","9":"2018-05-30","10":"340","11":"e0197014","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"1","17":"1","18":"0","19":"Correction: Effects of resource-dependent cannibalism on population size distribution and individual life history in a case-bearing caddisfly","20":"journal-article","21":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","22":"http://dx.doi.org/10.1371/journal.pone.0197014","23":"13","24":"en","25":"PLoS ONE","26":"<tibble[,3]>","27":"<tibble[,4]>","28":"<df[,2] [1 × 2]>","29":"2018-05-30","30":"http://creativecommons.org/licenses/by/4.0/","31":"0","32":"vor","33":"<tibble[,10]>","34":"<tibble[,4]>","35":"<NULL>","36":"<NULL>"},{"1":"PLOS ONE","2":"2018-05-30","3":"2019-10-18","4":"2018-05-30","5":"10.1371/journal.pone.0194133","6":"2020-03-30","7":"1932-6203","8":"5","9":"2018-05-30","10":"340","11":"e0194133","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"71","17":"71","18":"0","19":"Internists’ dilemmas in their interactions with chronically ill patients; A comparison of their interaction strategies and dilemmas in two different medical contexts","20":"journal-article","21":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","22":"http://dx.doi.org/10.1371/journal.pone.0194133","23":"13","24":"en","25":"PLoS ONE","26":"<tibble[,5]>","27":"<tibble[,4]>","28":"<df[,2] [1 × 2]>","29":"2018-05-30","30":"http://creativecommons.org/licenses/by/4.0/","31":"0","32":"vor","33":"<tibble[,11]>","34":"<NULL>","35":"<NULL>","36":"<NULL>"},{"1":"PLOS ONE","2":"2018-05-30","3":"2018-10-24","4":"2018-05-30","5":"10.1371/journal.pone.0197429","6":"2020-03-30","7":"1932-6203","8":"5","9":"2018-05-30","10":"340","11":"e0197429","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"42","17":"42","18":"0","19":"Managing the 1920s’ Chilean educational crisis: A historical view combined with machine learning","20":"journal-article","21":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","22":"http://dx.doi.org/10.1371/journal.pone.0197429","23":"13","24":"en","25":"PLoS ONE","26":"<tibble[,5]>","27":"<tibble[,4]>","28":"<df[,2] [1 × 2]>","29":"2018-05-30","30":"http://creativecommons.org/licenses/by/4.0/","31":"0","32":"vor","33":"<tibble[,10]>","34":"<NULL>","35":"<NULL>","36":"<tibble[,2]>"},{"1":"PLOS ONE","2":"2018-05-30","3":"2019-10-18","4":"2018-05-30","5":"10.1371/journal.pone.0196841","6":"2020-03-30","7":"1932-6203","8":"5","9":"2018-05-30","10":"340","11":"e0196841","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"24","17":"24","18":"4","19":"Can excreted thiocyanate be used to detect cyanide exposure in live reef fish?","20":"journal-article","21":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","22":"http://dx.doi.org/10.1371/journal.pone.0196841","23":"13","24":"en","25":"PLoS ONE","26":"<tibble[,5]>","27":"<tibble[,4]>","28":"<df[,2] [1 × 2]>","29":"2018-05-30","30":"http://creativecommons.org/licenses/by/4.0/","31":"0","32":"vor","33":"<tibble[,10]>","34":"<NULL>","35":"<NULL>","36":"<tibble[,4]>"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>This adds four columns all the way to the right: * <strong>date</strong> (Date on which this license begins to take effect) * <strong>URL</strong> (Link to a web page describing this license–in this case, Creative Commons) * <strong>delay in days</strong> (Number of days between the publication date of the work and the start date of this license), and * <strong>content.version</strong>, which specifies the version of the article the licensing data pertains to (VOR = Version of Record, AM = Accepted Manuscript, TDM = Text and Data Mining).</p>
<p>Browsing the rows, we see all are CC BY 4.0, which stands to reason given <em>PLOS ONE</em> is an open access publisher and <a href="https://journals.plos.org/plosone/s/licenses-and-copyright">applies the CC BY license</a> to the articles they publish.</p>
</div>
</div>
<div id="filtering-rows-and-selecting-columns-with-dplyr" class="section level2">
<h2>Filtering rows and selecting columns with <code>dplyr</code></h2>
<p>You can use the <code>filter()</code> and <code>select()</code> functions from the <code>dplyr</code> package if you want to get subsets of this data after you have made the query. Note that this is a completely different <code>filter</code> than the one used above inside the <code>cr_journals()</code> function. That one was an argument sent with the API call that filtered the results before they were returned. This is a separate function that is part of <code>dplyr</code> to help you filter a data frame in R.</p>
<p>To learn more about the <code>dplyr</code> package, read the <a href="https://r4ds.had.co.nz/transform.html">“Data Transformation” chapter in <em>R For Data Science</em></a>.</p>
<p>Above, we retrieved all articles from the <em>Journal of Librarianship &amp; Scholarly Communication</em> published after January 1, 2019. Let’s say you want only volume 8, issue 1:</p>
<pre class="r"><code># assign the JLSC ISSN and get all publications after January 1, 2019
jlsc_issn &lt;- &quot;2162-3309&quot;
jlsc_publications_2019 &lt;- rcrossref::cr_journals(issn = jlsc_issn, works = T, limit = 25,
    filter = c(from_pub_date = &quot;2019-01-01&quot;)) %&gt;%
    purrr::pluck(&quot;data&quot;)

# print the data frame with select columns
jlsc_publications_2019 %&gt;%
    dplyr::select(title, doi, volume, issue, page, issued, url, publisher, reference.count,
        type, issn)</code></pre>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["title"],"name":[1],"type":["chr"],"align":["left"]},{"label":["doi"],"name":[2],"type":["chr"],"align":["left"]},{"label":["volume"],"name":[3],"type":["chr"],"align":["left"]},{"label":["issue"],"name":[4],"type":["chr"],"align":["left"]},{"label":["page"],"name":[5],"type":["chr"],"align":["left"]},{"label":["issued"],"name":[6],"type":["chr"],"align":["left"]},{"label":["url"],"name":[7],"type":["chr"],"align":["left"]},{"label":["publisher"],"name":[8],"type":["chr"],"align":["left"]},{"label":["reference.count"],"name":[9],"type":["chr"],"align":["left"]},{"label":["type"],"name":[10],"type":["chr"],"align":["left"]},{"label":["issn"],"name":[11],"type":["chr"],"align":["left"]}],"data":[{"1":"Review of Creative Commons for Educators and Librarians","2":"10.31274/jlsc.12904","3":"9","4":"1","5":"NA","6":"2021-07-16","7":"http://dx.doi.org/10.31274/jlsc.12904","8":"Iowa State University","9":"0","10":"journal-article","11":"2162-3309"},{"1":"The Changing Landscape of Open Access Publishing: Can Open Access Publishing Make the Scholarly World More Equitable and Productive?","2":"10.7710/2162-3309.2345","3":"9","4":"1","5":"2345","6":"2021-02-21","7":"http://dx.doi.org/10.7710/2162-3309.2345","8":"Iowa State University","9":"0","10":"journal-article","11":"2162-3309"},{"1":"Knowledge and Practice changes following a student data focused data management education program.","2":"10.31274/jlsc.12906","3":"NA","4":"NA","5":"NA","6":"2021-07-07","7":"http://dx.doi.org/10.31274/jlsc.12906","8":"Iowa State University","9":"0","10":"journal-article","11":"2162-3309"},{"1":"Using Law School Faculty Author Profiles to Promote Impact: The U.S. News & World Report Saga Continues","2":"10.7710/2162-3309.2386","3":"8","4":"1","5":"2386","6":"2020-12-14","7":"http://dx.doi.org/10.7710/2162-3309.2386","8":"Iowa State University","9":"0","10":"journal-article","11":"2162-3309"},{"1":"Creating a Library Publishing Program: Your Options Are Limited","2":"10.7710/2162-3309.2262","3":"7","4":"1","5":"NA","6":"2019-01-10","7":"http://dx.doi.org/10.7710/2162-3309.2262","8":"Iowa State University","9":"0","10":"journal-article","11":"2162-3309"},{"1":"Bibliometrics and Research Evaluation: Uses and Abuses","2":"10.7710/2162-3309.2286","3":"7","4":"1","5":"NA","6":"2019-01-10","7":"http://dx.doi.org/10.7710/2162-3309.2286","8":"Iowa State University","9":"0","10":"journal-article","11":"2162-3309"},{"1":"Meaningful Metrics: A 21st Century Librarian's Guide to Bibliometrics, Altmetrics, and Research Impact [Book Review]","2":"10.7710/2162-3309.2290","3":"7","4":"1","5":"NA","6":"2019-01-15","7":"http://dx.doi.org/10.7710/2162-3309.2290","8":"Iowa State University","9":"0","10":"journal-article","11":"2162-3309"},{"1":"Online Safety and Academic Scholarship: Exploring Researchers’ Concerns from Ghana","2":"10.7710/2162-3309.2263","3":"7","4":"1","5":"NA","6":"2019-01-15","7":"http://dx.doi.org/10.7710/2162-3309.2263","8":"Iowa State University","9":"0","10":"journal-article","11":"2162-3309"},{"1":"When a Repository Is Not Enough: Redesigning a Digital Ecosystem to Serve Scholarly Communication","2":"10.7710/2162-3309.2225","3":"7","4":"1","5":"NA","6":"2019-02-26","7":"http://dx.doi.org/10.7710/2162-3309.2225","8":"Iowa State University","9":"0","10":"journal-article","11":"2162-3309"},{"1":"Strategies for Supporting OER Adoption through Faculty and Instructor Use of a Federated Search Tool","2":"10.7710/2162-3309.2279","3":"7","4":"1","5":"NA","6":"2019-02-22","7":"http://dx.doi.org/10.7710/2162-3309.2279","8":"Iowa State University","9":"0","10":"journal-article","11":"2162-3309"},{"1":"Opening Up Open Access Institutional Repositories to Demonstrate Value: Two Universities’ Pilots on Including Metadata-Only Records","2":"10.7710/2162-3309.2220","3":"7","4":"1","5":"NA","6":"2019-03-08","7":"http://dx.doi.org/10.7710/2162-3309.2220","8":"Iowa State University","9":"0","10":"journal-article","11":"2162-3309"},{"1":"Course Review: CopyrightX","2":"10.7710/2162-3309.2297","3":"7","4":"1","5":"NA","6":"2019-04-11","7":"http://dx.doi.org/10.7710/2162-3309.2297","8":"Iowa State University","9":"0","10":"journal-article","11":"2162-3309"},{"1":"Dynamic Research Support for Academic Libraries [Book Review]","2":"10.7710/2162-3309.2307","3":"7","4":"1","5":"NA","6":"2019-04-15","7":"http://dx.doi.org/10.7710/2162-3309.2307","8":"Iowa State University","9":"0","10":"journal-article","11":"2162-3309"},{"1":"Whose Research is it Anyway? Academic Social Networks Versus Institutional Repositories","2":"10.7710/2162-3309.2243","3":"7","4":"1","5":"NA","6":"2019-04-25","7":"http://dx.doi.org/10.7710/2162-3309.2243","8":"Iowa State University","9":"0","10":"journal-article","11":"2162-3309"},{"1":"Library Publishers as Educators: Crafting Curriculum for Undergraduate Research Journals","2":"10.7710/2162-3309.2296","3":"7","4":"1","5":"NA","6":"2019-05-06","7":"http://dx.doi.org/10.7710/2162-3309.2296","8":"Iowa State University","9":"0","10":"journal-article","11":"2162-3309"},{"1":"Avoiding the “Axe”: Advancing Affordable and Open Education Resources at a Midsize University","2":"10.7710/2162-3309.2259","3":"7","4":"1","5":"NA","6":"2019-05-20","7":"http://dx.doi.org/10.7710/2162-3309.2259","8":"Iowa State University","9":"0","10":"journal-article","11":"2162-3309"},{"1":"The Development of the Journal Evaluation Tool to Evaluate the Credibility of Publication Venues","2":"10.7710/2162-3309.2250","3":"7","4":"1","5":"NA","6":"2019-06-04","7":"http://dx.doi.org/10.7710/2162-3309.2250","8":"Iowa State University","9":"0","10":"journal-article","11":"2162-3309"},{"1":"Embracing New Trends in Scholarly Communication: From Competency Requirements in the Workplace to LIS Curriculum Presence","2":"10.7710/2162-3309.2291","3":"7","4":"1","5":"NA","6":"2019-07-19","7":"http://dx.doi.org/10.7710/2162-3309.2291","8":"Iowa State University","9":"0","10":"journal-article","11":"2162-3309"},{"1":"The Five Laws of OER: Observations from Ranganathan","2":"10.7710/2162-3309.2299","3":"7","4":"1","5":"NA","6":"2019-08-20","7":"http://dx.doi.org/10.7710/2162-3309.2299","8":"Iowa State University","9":"0","10":"journal-article","11":"2162-3309"},{"1":"We Can Do I.T.: Women in Library Information Technology","2":"10.7710/2162-3309.2320","3":"7","4":"1","5":"NA","6":"2019-09-16","7":"http://dx.doi.org/10.7710/2162-3309.2320","8":"Iowa State University","9":"0","10":"journal-article","11":"2162-3309"},{"1":"JLSC Board Editorial 2019","2":"10.7710/2162-3309.2334","3":"7","4":"1","5":"NA","6":"2019-09-12","7":"http://dx.doi.org/10.7710/2162-3309.2334","8":"Iowa State University","9":"0","10":"journal-article","11":"2162-3309"},{"1":"A Team Approach: Library Publishing Partnerships with Scholarly Societies","2":"10.7710/2162-3309.2326","3":"7","4":"1","5":"NA","6":"2019-11-21","7":"http://dx.doi.org/10.7710/2162-3309.2326","8":"Iowa State University","9":"0","10":"journal-article","11":"2162-3309"},{"1":"Scholarly Publishing Literacy at the University of South Florida Libraries: From Advising to Active Involvement","2":"10.7710/2162-3309.2310","3":"7","4":"1","5":"NA","6":"2019-12-04","7":"http://dx.doi.org/10.7710/2162-3309.2310","8":"Iowa State University","9":"0","10":"journal-article","11":"2162-3309"},{"1":"Volume 7 Masthead","2":"10.7710/2162-3309.2351","3":"7","4":"1","5":"NA","6":"2019-12-04","7":"http://dx.doi.org/10.7710/2162-3309.2351","8":"Iowa State University","9":"0","10":"journal-article","11":"2162-3309"},{"1":"From Meow to ROAR: Expanding Open Access Repository Services at the University of Houston Libraries","2":"10.7710/2162-3309.2309","3":"7","4":"1","5":"NA","6":"2019-12-13","7":"http://dx.doi.org/10.7710/2162-3309.2309","8":"Iowa State University","9":"0","10":"journal-article","11":"2162-3309"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<pre class="r"><code># use filter from dplyr to get only volume 8, issue 1
jlsc_8_1 &lt;- jlsc_publications_2019 %&gt;%
    dplyr::filter(volume == &quot;8&quot;, issue == &quot;1&quot;)

# print the data frame with select columns
jlsc_8_1 %&gt;%
    dplyr::select(title, doi, volume, issue, page, issued, url, publisher, reference.count,
        type, issn)</code></pre>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["title"],"name":[1],"type":["chr"],"align":["left"]},{"label":["doi"],"name":[2],"type":["chr"],"align":["left"]},{"label":["volume"],"name":[3],"type":["chr"],"align":["left"]},{"label":["issue"],"name":[4],"type":["chr"],"align":["left"]},{"label":["page"],"name":[5],"type":["chr"],"align":["left"]},{"label":["issued"],"name":[6],"type":["chr"],"align":["left"]},{"label":["url"],"name":[7],"type":["chr"],"align":["left"]},{"label":["publisher"],"name":[8],"type":["chr"],"align":["left"]},{"label":["reference.count"],"name":[9],"type":["chr"],"align":["left"]},{"label":["type"],"name":[10],"type":["chr"],"align":["left"]},{"label":["issn"],"name":[11],"type":["chr"],"align":["left"]}],"data":[{"1":"Using Law School Faculty Author Profiles to Promote Impact: The U.S. News & World Report Saga Continues","2":"10.7710/2162-3309.2386","3":"8","4":"1","5":"2386","6":"2020-12-14","7":"http://dx.doi.org/10.7710/2162-3309.2386","8":"Iowa State University","9":"0","10":"journal-article","11":"2162-3309"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p><code>filter()</code> will go through each row of your existing <code>jlsc_publications_2019</code> data frame, and keep only those rows with values matching the filters you input. <strong>Note:</strong> be careful of filtering by ISSN. If a journal has multiple ISSNs they’ll be combined in a single cell with a comma and the <code>filter()</code> will fail, as with JAMA above. In this case it may be wiser to use <code>str_detect()</code>, as described a couple code chunks down.</p>
<pre class="r"><code>jah_jama_publications$issn[1]</code></pre>
<pre><code>## [1] &quot;0098-7484,1538-3598&quot;</code></pre>
<p>We can use <code>filter()</code> to get a single article from within this data frame if we need, either by DOI:</p>
<pre class="r"><code># filter to get &#39;The Five Laws of OER&#39; article by DOI
jlsc_article &lt;- jlsc_publications_2019 %&gt;%
    dplyr::filter(doi == &quot;10.7710/2162-3309.2299&quot;)

# print data frame with select columns
jlsc_article %&gt;%
    dplyr::select(title, doi, volume, issue, page, issued, url, publisher, reference.count,
        type, issn)</code></pre>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["title"],"name":[1],"type":["chr"],"align":["left"]},{"label":["doi"],"name":[2],"type":["chr"],"align":["left"]},{"label":["volume"],"name":[3],"type":["chr"],"align":["left"]},{"label":["issue"],"name":[4],"type":["chr"],"align":["left"]},{"label":["page"],"name":[5],"type":["chr"],"align":["left"]},{"label":["issued"],"name":[6],"type":["chr"],"align":["left"]},{"label":["url"],"name":[7],"type":["chr"],"align":["left"]},{"label":["publisher"],"name":[8],"type":["chr"],"align":["left"]},{"label":["reference.count"],"name":[9],"type":["chr"],"align":["left"]},{"label":["type"],"name":[10],"type":["chr"],"align":["left"]},{"label":["issn"],"name":[11],"type":["chr"],"align":["left"]}],"data":[{"1":"The Five Laws of OER: Observations from Ranganathan","2":"10.7710/2162-3309.2299","3":"7","4":"1","5":"NA","6":"2019-08-20","7":"http://dx.doi.org/10.7710/2162-3309.2299","8":"Iowa State University","9":"0","10":"journal-article","11":"2162-3309"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>Or by title:</p>
<pre class="r"><code># use str_detect to search the title column for articles that include the term
# OER
jlsc_article &lt;- jlsc_publications_2019 %&gt;%
    dplyr::filter(stringr::str_detect(title, &quot;OER&quot;))

# print the data frame with select column
jlsc_article %&gt;%
    dplyr::select(title, doi, volume, issue, page, issued, url, publisher, reference.count,
        type, issn)</code></pre>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["title"],"name":[1],"type":["chr"],"align":["left"]},{"label":["doi"],"name":[2],"type":["chr"],"align":["left"]},{"label":["volume"],"name":[3],"type":["chr"],"align":["left"]},{"label":["issue"],"name":[4],"type":["chr"],"align":["left"]},{"label":["page"],"name":[5],"type":["chr"],"align":["left"]},{"label":["issued"],"name":[6],"type":["chr"],"align":["left"]},{"label":["url"],"name":[7],"type":["chr"],"align":["left"]},{"label":["publisher"],"name":[8],"type":["chr"],"align":["left"]},{"label":["reference.count"],"name":[9],"type":["chr"],"align":["left"]},{"label":["type"],"name":[10],"type":["chr"],"align":["left"]},{"label":["issn"],"name":[11],"type":["chr"],"align":["left"]}],"data":[{"1":"Strategies for Supporting OER Adoption through Faculty and Instructor Use of a Federated Search Tool","2":"10.7710/2162-3309.2279","3":"7","4":"1","5":"NA","6":"2019-02-22","7":"http://dx.doi.org/10.7710/2162-3309.2279","8":"Iowa State University","9":"0","10":"journal-article","11":"2162-3309"},{"1":"The Five Laws of OER: Observations from Ranganathan","2":"10.7710/2162-3309.2299","3":"7","4":"1","5":"NA","6":"2019-08-20","7":"http://dx.doi.org/10.7710/2162-3309.2299","8":"Iowa State University","9":"0","10":"journal-article","11":"2162-3309"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>Here, we use the <code>str_detect()</code> function from the <code>stringr</code> package, which is loaded as part of the <code>tidyverse</code>, in order to find a single term (OER) in the title.</p>
<p>Remember that these <code>dplyr</code> and <code>stringr</code> functions are searching through our existing data frame <code>jlsc_publications_2019</code>, not issuing new API calls.</p>
</div>
<div id="field-queries" class="section level2">
<h2>Field queries</h2>
<p>There is yet another way of making your query more precise, and that is to use a field query (<code>flq</code>) argument to <code>cr_journals()</code>. This allows you to search in specific bibliographic fields such as author, editor, titles, ISSNs, and author affiliation (not widely available). These are listed in the <a href="https://github.com/CrossRef/rest-api-doc#field-queries">Crossref documentation</a> and reproduced below. You <em>must</em> provide an ISSN–in other words, you can’t run a field query for authors across all journals.</p>
<table>
<colgroup>
<col width="63%" />
<col width="36%" />
</colgroup>
<thead>
<tr class="header">
<th>Field query parameter</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><code>query.container-title</code></td>
<td>Query <code>container-title</code> aka. publication name</td>
</tr>
<tr class="even">
<td><code>query.author</code></td>
<td>Query author given and family names</td>
</tr>
<tr class="odd">
<td><code>query.editor</code></td>
<td>Query editor given and family names</td>
</tr>
<tr class="even">
<td><code>query.chair</code></td>
<td>Query chair given and family names</td>
</tr>
<tr class="odd">
<td><code>query.translator</code></td>
<td>Query translator given and family names</td>
</tr>
<tr class="even">
<td><code>query.contributor</code></td>
<td>Query author, editor, chair and translator given and family names</td>
</tr>
<tr class="odd">
<td><code>query.bibliographic</code></td>
<td>Query bibliographic information, useful for citation look up. Includes titles, authors, ISSNs and publication years</td>
</tr>
<tr class="even">
<td><code>query.affiliation</code></td>
<td>Query contributor affiliations</td>
</tr>
</tbody>
</table>
<div id="field-query-by-title" class="section level3">
<h3>Field query by title</h3>
<p>Here, we get all publications from the Journal of Librarianship and Scholarly Communication with the term “open access” in the title.</p>
<pre class="r"><code># assign JLSC ISSN and query the bibliographic field for terms mentioning open
# access.
jlsc_issn &lt;- &quot;2162-3309&quot;
jlsc_publications_oa &lt;- rcrossref::cr_journals(issn = jlsc_issn, works = T, limit = 25,
    flq = c(query.bibliographic = &quot;open access&quot;)) %&gt;%
    purrr::pluck(&quot;data&quot;)

# print the data frame with select columns
jlsc_publications_oa %&gt;%
    dplyr::select(title, doi, volume, issue, page, issued, issn, author)</code></pre>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["title"],"name":[1],"type":["chr"],"align":["left"]},{"label":["doi"],"name":[2],"type":["chr"],"align":["left"]},{"label":["volume"],"name":[3],"type":["chr"],"align":["left"]},{"label":["issue"],"name":[4],"type":["chr"],"align":["left"]},{"label":["page"],"name":[5],"type":["chr"],"align":["left"]},{"label":["issued"],"name":[6],"type":["chr"],"align":["left"]},{"label":["issn"],"name":[7],"type":["chr"],"align":["left"]},{"label":["author"],"name":[8],"type":["list"],"align":["right"]}],"data":[{"1":"The Changing Landscape of Open Access Publishing: Can Open Access Publishing Make the Scholarly World More Equitable and Productive?","2":"10.7710/2162-3309.2345","3":"9","4":"1","5":"2345","6":"2021-02-21","7":"2162-3309","8":"<tibble[,3]>"},{"1":"Campus Open Access Funds: Experiences of the KU “One University” Open\\n                        Access Author Fund","2":"10.7710/2162-3309.1252","3":"3","4":"1","5":"1252","6":"2015-09-18","7":"2162-3309","8":"<tibble[,3]>"},{"1":"Book Review: Open Divide: Critical Studies on Open Access","2":"10.7710/2162-3309.2352","3":"8","4":"1","5":"2352","6":"2020-06-24","7":"2162-3309","8":"<tibble[,3]>"},{"1":"Open Access and the Author-Pays Problem: Assuring Access for Readers and\\n                        Authors in the Global Academic Community","2":"10.7710/2162-3309.1064","3":"1","4":"3","5":"1064","6":"2013-02-28","7":"2162-3309","8":"<tibble[,3]>"},{"1":"“Open Don’t Mean Free”: A Reflection on the Potential Advantages and\\n                        Disadvantages of Publishing Research via Open Access","2":"10.7710/2162-3309.1049","3":"1","4":"2","5":"1049","6":"2012-09-28","7":"2162-3309","8":"<tibble[,3]>"},{"1":"Open Data, [Open] Access: Linking Data Sharing and Article Sharing in the Earth Sciences","2":"10.7710/2162-3309.2150","3":"5","4":"General Issue","5":"NA","6":"2017","7":"2162-3309","8":"<tibble[,3]>"},{"1":"Reassembling Scholarly Communications: Histories, Infrastructures, and\\n                        Global Politics of Open Access [Book Review]","2":"10.7710/2162-3309.2412","3":"9","4":"1","5":"2412","6":"2021-04-11","7":"2162-3309","8":"<tibble[,3]>"},{"1":"Developing Open Access Journals: A Practical Guide","2":"10.7710/2162-3309.1028","3":"1","4":"1","5":"1028","6":"2012-05-15","7":"2162-3309","8":"<tibble[,3]>"},{"1":"Innovation Fair Abstracts, SPARC 2012 Open Access Meeting","2":"10.7710/2162-3309.1042","3":"1","4":"1","5":"1042","6":"2012-05-15","7":"2162-3309","8":"<tibble[,3]>"},{"1":"Point & Counterpoint Is CC BY the Best Open Access License?","2":"10.7710/2162-3309.1043","3":"1","4":"1","5":"1043","6":"2012-05-15","7":"2162-3309","8":"<tibble[,3]>"},{"1":"Up from Under the “Open Access” Bus","2":"10.7710/2162-3309.1045","3":"1","4":"2","5":"1045","6":"2012-09-28","7":"2162-3309","8":"<tibble[,3]>"},{"1":"Understanding and Making Use of Academic Authors’ Open Access\\n                        Rights","2":"10.7710/2162-3309.1050","3":"1","4":"2","5":"1050","6":"2012-09-28","7":"2162-3309","8":"<tibble[,3]>"},{"1":"Response to “Is CC BY the Best Open Access License?”","2":"10.7710/2162-3309.1055","3":"1","4":"2","5":"1055","6":"2012-09-28","7":"2162-3309","8":"<tibble[,3]>"},{"1":"Tenure-Track Science Faculty and the ‘Open Access Citation\\n                        Effect’","2":"10.7710/2162-3309.1052","3":"1","4":"3","5":"1052","6":"2013-02-28","7":"2162-3309","8":"<tibble[,3]>"},{"1":"Addressing Faculty Publishing Concerns with Open Access Journal Quality\\n                        Indicators","2":"10.7710/2162-3309.1133","3":"2","4":"2","5":"1133","6":"2014-04-10","7":"2162-3309","8":"<tibble[,3]>"},{"1":"“Free to All”: Library Publishing and the Challenge of Open\\n                        Access","2":"10.7710/2162-3309.1181","3":"2","4":"4","5":"1181","6":"2014-12-11","7":"2162-3309","8":"<tibble[,3]>"},{"1":"Evaluating an Open Access Publishing Fund at a Comprehensive\\n                        University","2":"10.7710/2162-3309.1204","3":"3","4":"3","5":"1204","6":"2016-02-05","7":"2162-3309","8":"<tibble[,3]>"},{"1":"Academic Librarians, Open Access, and the Ethics of Care","2":"10.31274/jlsc.12914","3":"9","4":"1","5":"NA","6":"2021-07-09","7":"2162-3309","8":"<tibble[,4]>"},{"1":"Effective Practices and Strategies for Open Access Outreach: A Qualitative Study","2":"10.7710/2162-3309.2216","3":"6","4":"General Issue","5":"NA","6":"2018","7":"2162-3309","8":"<tibble[,3]>"},{"1":"Engaged Citizenship through Campus-Level Democratic Processes: A Librarian and Graduate Student Collaboration on Open Access Policy Adoption","2":"10.7710/2162-3309.2229","3":"6","4":"2","5":"2229","6":"2018-08-31","7":"2162-3309","8":"<tibble[,3]>"},{"1":"Measuring Open Access Policy Compliance: Results of a Survey","2":"10.7710/2162-3309.2247","3":"6","4":"1","5":"NA","6":"2018-10-05","7":"2162-3309","8":"<tibble[,3]>"},{"1":"Opening Up Open Access Institutional Repositories to Demonstrate Value: Two Universities’ Pilots on Including Metadata-Only Records","2":"10.7710/2162-3309.2220","3":"7","4":"1","5":"NA","6":"2019-03-08","7":"2162-3309","8":"<tibble[,3]>"},{"1":"From Meow to ROAR: Expanding Open Access Repository Services at the University of Houston Libraries","2":"10.7710/2162-3309.2309","3":"7","4":"1","5":"NA","6":"2019-12-13","7":"2162-3309","8":"<tibble[,3]>"},{"1":"Publications and Evaluations: Conducting a Baseline Assessment of Open Access Adoption and Support at an R2 University","2":"10.7710/2162-3309.2392","3":"8","4":"1","5":"2392","6":"2020-11-09","7":"2162-3309","8":"<tibble[,3]>"},{"1":"The Factors Significant to the Introduction of Institutional Open Access Policies: Two Case Studies of R-1 Universities","2":"10.7710/2162-3309.2400","3":"9","4":"1","5":"2400","6":"2021-02-25","7":"2162-3309","8":"<tibble[,3]>"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="field-query-by-author-contributor-or-editor" class="section level3">
<h3>Field query by author, contributor, or editor</h3>
<p>The <code>flq</code> argument can also be used for authors, contributors, or editors. Here we search the same journal for authors with the name Salo (looking for all articles written by Dorothea Salo).</p>
<pre class="r"><code># Use the query.author field query to find JLSC articles with author name Salo
jlsc_publications_auth &lt;- rcrossref::cr_journals(issn = jlsc_issn, works = T, limit = 25,
    flq = c(query.author = &quot;salo&quot;)) %&gt;%
    purrr::pluck(&quot;data&quot;)

# print the data frame with select columns
jlsc_publications_auth %&gt;%
    dplyr::select(title, doi, volume, issue, page, issued, url, publisher, reference.count,
        type, issn)</code></pre>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["title"],"name":[1],"type":["chr"],"align":["left"]},{"label":["doi"],"name":[2],"type":["chr"],"align":["left"]},{"label":["volume"],"name":[3],"type":["chr"],"align":["left"]},{"label":["issue"],"name":[4],"type":["chr"],"align":["left"]},{"label":["page"],"name":[5],"type":["chr"],"align":["left"]},{"label":["issued"],"name":[6],"type":["chr"],"align":["left"]},{"label":["url"],"name":[7],"type":["chr"],"align":["left"]},{"label":["publisher"],"name":[8],"type":["chr"],"align":["left"]},{"label":["reference.count"],"name":[9],"type":["chr"],"align":["left"]},{"label":["type"],"name":[10],"type":["chr"],"align":["left"]},{"label":["issn"],"name":[11],"type":["chr"],"align":["left"]}],"data":[{"1":"How to Scuttle a Scholarly Communication Initiative","2":"10.7710/2162-3309.1075","3":"1","4":"4","5":"1075","6":"2013-08-15","7":"http://dx.doi.org/10.7710/2162-3309.1075","8":"Iowa State University","9":"0","10":"journal-article","11":"2162-3309"},{"1":"On the Mark? Responses to a Sting","2":"10.7710/2162-3309.1116","3":"2","4":"1","5":"1116","6":"2013-12-09","7":"http://dx.doi.org/10.7710/2162-3309.1116","8":"Iowa State University","9":"0","10":"journal-article","11":"2162-3309"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<hr />
<p><strong>TRY IT YOURSELF</strong></p>
<ol style="list-style-type: decimal">
<li>Assign the ISSN for <em>College &amp; Research Libraries</em> to a value - 2150-6701</li>
<li>Use the <code>query.author</code> field query (<code>flq</code>) to find all articles written by Lisa Janicke Hinchliffe</li>
</ol>
<pre class="r"><code># assign the C&amp;RL ISSN 2150-6701</code></pre>
<pre class="r"><code># use the query.author field query to search for articles written by Lisa
# Hinchliffe.</code></pre>
</div>
</div>
</div>
<div id="viewing-the-json-file" class="section level1">
<h1>Viewing the JSON file</h1>
<p>You can view these files in a JSON viewer using <code>jsonedit()</code> from the <code>listviewer</code> package. Note especially the last few variables. These are nested lists, as a single article can have multiple authors, and each author has a given name, family name, and sequence of authorship.</p>
<pre class="r"><code># assign the PLOS ISSN and get the last 25 articles on deposit
plosone_issn &lt;- &quot;1932-6203&quot;
plosone_publications &lt;- cr_journals(issn = plosone_issn, works = TRUE, limit = 5) %&gt;%
    pluck(&quot;data&quot;)

listviewer::jsonedit(plosone_publications, mode = &quot;view&quot;)</code></pre>
<div id="htmlwidget-f127905823c4eabc59c1" style="width:672px;height:480px;" class="jsonedit html-widget"></div>
<script type="application/json" data-for="htmlwidget-f127905823c4eabc59c1">{"x":{"data":{"container.title":["PLoS ONE","PLoS ONE","PLoS ONE","PLoS ONE","PLoS ONE"],"created":["2014-03-18","2014-03-18","2014-03-18","2014-03-14","2014-03-14"],"deposited":["2019-08-08","2018-10-22","2018-10-22","2018-10-23","2018-10-23"],"published.online":["2014-03-18","2014-03-18","2014-03-18","2014-03-14","2014-03-14"],"doi":["10.1371/journal.pone.0091033","10.1371/journal.pone.0091432","10.1371/journal.pone.0091498","10.1371/journal.pone.0090927","10.1371/journal.pone.0091218"],"indexed":["2020-03-30","2020-03-30","2020-03-30","2020-03-30","2020-03-30"],"issn":["1932-6203","1932-6203","1932-6203","1932-6203","1932-6203"],"issue":["3","3","3","3","3"],"issued":["2014-03-18","2014-03-18","2014-03-18","2014-03-14","2014-03-14"],"member":["340","340","340","340","340"],"page":["e91033","e91432","e91498","e90927","e91218"],"prefix":["10.1371","10.1371","10.1371","10.1371","10.1371"],"publisher":["Public Library of Science (PLoS)","Public Library of Science (PLoS)","Public Library of Science (PLoS)","Public Library of Science (PLoS)","Public Library of Science (PLoS)"],"score":["1","1","1","1","1"],"source":["Crossref","Crossref","Crossref","Crossref","Crossref"],"reference.count":["42","24","0","22","0"],"references.count":["42","24","0","22","0"],"is.referenced.by.count":["6","3","0","4","0"],"title":["Multiple Sporadic Colorectal Cancers Display a Unique Methylation Phenotype","Need for Space: The Key Distance Effect Depends on Spatial Stimulus Configurations","Correction: MicroRNA-130b Suppresses Migration and Invasion of Colorectal Cancer Cells through Downregulation of Integrin β1","Oral Health Training Programs for Community and Professional Health Care Workers in Nairobi East District Increases Identification of HIV-Infected Patients","Correction: Association of Polymorphisms and Haplotypes in the Insulin-Like Growth Factor 1 Receptor (IGF1R) Gene with the Risk of Breast Cancer in Korean Women"],"type":["journal-article","journal-article","journal-article","journal-article","journal-article"],"update.policy":["http://dx.doi.org/10.1371/journal.pone.corrections_policy","http://dx.doi.org/10.1371/journal.pone.corrections_policy","http://dx.doi.org/10.1371/journal.pone.corrections_policy","http://dx.doi.org/10.1371/journal.pone.corrections_policy","http://dx.doi.org/10.1371/journal.pone.corrections_policy"],"url":["http://dx.doi.org/10.1371/journal.pone.0091033","http://dx.doi.org/10.1371/journal.pone.0091432","http://dx.doi.org/10.1371/journal.pone.0091498","http://dx.doi.org/10.1371/journal.pone.0090927","http://dx.doi.org/10.1371/journal.pone.0091218"],"volume":["9","9","9","9","9"],"language":["en","en","en","en","en"],"short.container.title":["PLoS ONE","PLoS ONE","PLoS ONE","PLoS ONE","PLoS ONE"],"author":[{"given":["Victoria","Juan Jose","Virginia","Leticia","Jenifer","Maria","Sergi","Xavier","Montserrat","Rosa M.","Xavier","Clara","Angel","Rodrigo","Antoni","Francesc",null],"family":["Gonzalo","Lozano","Alonso-Espinaco","Moreira","Muñoz","Pellisé","Castellví-Bel","Bessa","Andreu","Xicola","Llor","Ruiz-Ponte","Carracedo","Jover","Castells","Balaguer",null],"sequence":["first","additional","additional","additional","additional","additional","additional","additional","additional","additional","additional","additional","additional","additional","additional","additional","additional"],"name":[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"for the Gastrointestinal Oncology Group of the Spanish Gastroenterological Association"]},{"given":["Melanie","Owino","Julia","Volker H."],"family":["Jonas","Eloka","Stephan","Franz"],"sequence":["first","additional","additional","additional"]},{"name":["The PLOS ONE Staff"],"sequence":["first"]},{"given":["Lucina N.","Wil J. M. van der","Elizabeth","Jan","Nico H. J.","Matthias A. W.","Andre","Jo E."],"family":["Koyio","Sanden","Dimba","Mulder","Creugers","Merkx","van der Ven","Frencken"],"sequence":["first","additional","additional","additional","additional","additional","additional","additional"]},{"name":["The PLOS ONE Staff"],"sequence":["first"]}],"link":[{"URL":["http://dx.plos.org/10.1371/journal.pone.0091033"],"content.type":["unspecified"],"content.version":["vor"],"intended.application":["similarity-checking"]},{"URL":["http://dx.plos.org/10.1371/journal.pone.0091432"],"content.type":["unspecified"],"content.version":["vor"],"intended.application":["similarity-checking"]},null,{"URL":["http://dx.plos.org/10.1371/journal.pone.0090927"],"content.type":["unspecified"],"content.version":["vor"],"intended.application":["similarity-checking"]},null],"content_domain":[{"domain":["www.plosone.org"],"crossmark_restriction":[false]},{"domain":["www.plosone.org"],"crossmark_restriction":[false]},{"domain":["www.plosone.org"],"crossmark_restriction":[false]},{"domain":["www.plosone.org"],"crossmark_restriction":[false]},{"domain":["www.plosone.org"],"crossmark_restriction":[false]}],"license":[{"date":["2014-03-18"],"URL":["http://creativecommons.org/licenses/by/4.0/"],"delay.in.days":[0],"content.version":["unspecified"]},{"date":["2014-03-18"],"URL":["http://creativecommons.org/licenses/by/4.0/"],"delay.in.days":[0],"content.version":["unspecified"]},{"date":["2014-03-18"],"URL":["http://creativecommons.org/licenses/by/4.0/"],"delay.in.days":[0],"content.version":["unspecified"]},{"date":["2014-03-14"],"URL":["http://creativecommons.org/licenses/by/4.0/"],"delay.in.days":[0],"content.version":["unspecified"]},{"date":["2014-03-14"],"URL":["http://creativecommons.org/licenses/by/4.0/"],"delay.in.days":[0],"content.version":["unspecified"]}],"reference":[{"key":["ref1","ref2","ref3","ref4","ref5","ref6","ref7","ref8","ref9","ref10","ref11","ref12","ref13","ref14","ref15","ref16","ref17","ref18","ref19","ref20","ref21","ref22","ref23","ref24","ref25","ref26","ref27","ref28","ref29","ref30","ref31","ref32","ref33","ref34","ref35","ref36","ref37","ref38","ref39","ref40","ref41","ref42"],"doi.asserted.by":["crossref","crossref","crossref","crossref","crossref","crossref","crossref","crossref","crossref","crossref","crossref","crossref","crossref","crossref","crossref","crossref","crossref","crossref",null,"crossref","crossref","crossref","crossref","crossref","crossref","crossref","crossref","crossref","crossref","crossref","crossref","crossref","crossref","crossref","crossref","crossref","crossref","crossref",null,"crossref","crossref","crossref"],"first.page":["1093","1528","1192","1609","1083","1330","1778","48","472","1","379","919","529","5029","1467","1317","e8777","814","37","676","16365","489","275","788","177","197","271","E32","865","1854","5402","1805","787","847","652","594","427","669",null,"709","4597","781"],"DOI":["10.1007/BF02236556","10.1002/bjs.6382","10.1007/s10350-004-0562-7","10.1053/j.gastro.2009.08.002","10.1038/modpathol.3800618","10.1093/jnci/dji275","10.1158/1055-9965.EPI-06-0164","10.1002/cncr.11445","10.1038/sj.bjc.6602708","10.1002/(SICI)1097-0215(19990331)81:1<1::AID-IJC1>3.0.CO;2-K","10.1016/j.cgh.2006.12.025","10.1056/NEJMra012242","10.1016/S0002-9440(10)64872-9","10.1038/onc.2011.652","10.1136/gut.2005.082859","10.1093/jnci/dji305","10.1371/journal.pone.0008777","10.1158/1940-6207.CAPR-09-0054",null,"10.3390/cancers5020676","10.3390/ijms140816365","10.1126/science.1083558","10.1016/j.ccr.2005.09.007","10.1053/j.gastro.2010.05.072","10.2217/epi.09.14","10.1007/978-1-59745-538-1_13","10.1101/gr.117523.110","10.1093/nar/28.8.e32","10.1136/gutjnl-2011-300041","10.1053/j.gastro.2010.01.035","10.1158/1078-0432.CCR-10-1491","10.1093/bioinformatics/bts251","10.1038/ng1834","10.1136/gutjnl-2011-300865","10.1016/j.jmoldx.2013.03.009","10.1002/path.2318","10.1158/1078-0432.CCR-09-3069","10.1111/j.1365-2559.2011.03818.x",null,"10.1093/jnci/djs187","10.1158/0008-5472.CAN-07-6645","10.1002/gcc.20341"],"article.title":["Synchronous and “early” metachronous colorectal adenocarcinoma: analysis of prognosis and current trends","Epidemiology and prognosis of synchronous colorectal cancers","Synchronous colorectal neoplasms in patients with colorectal cancer: predisposing individual and familial factors","A prospective cohort study shows unique epigenetic, genetic, and prognostic features of synchronous colorectal cancers","Epigenetic profiling of synchronous colorectal neoplasias by quantitative DNA methylation analysis","MGMT promoter methylation and field defect in sporadic colorectal cancer","The case for a genetic predisposition to serrated neoplasia in the colorectum: hypothesis and review of the literature","Evidence of a preferred molecular pathway in patients with synchronous colorectal cancer","The role of MLH1, MSH2 and MSH6 in the development of multiple colorectal cancers","Microsatellite instability in multiple colorectal tumors","Identification of MYH mutation carriers in colorectal cancer: a multicenter, case-control, population-based study","Hereditary colorectal cancer","Concordant CpG island methylation in hyperplastic polyposis","DNA demethylation in normal colon tissue predicts predisposition to multiple cancers","Extensive DNA methylation in normal colorectal mucosa in hyperplastic polyposis","DNA methylation, field effects, and colorectal cancer","Aberrant gene promoter methylation associated with sporadic multiple colorectal cancer","Concordant DNA methylation in synchronous colorectal carcinomas","[Epigenetics of cancer]","Epigenetics and colorectal cancer pathogenesis","Molecular pathways involved in colorectal cancer: implications for disease behavior and prevention","Induction of tumors in mice by genomic hypomethylation","Global loss of imprinting leads to widespread tumorigenesis in adult mice","Susceptibility genetic variants associated with colorectal cancer risk correlate with cancer phenotype","Genome-wide DNA methylation profiling using Infinium(R) assay","Whole-genome genotyping on bead arrays","Genome-scale analysis of aberrant DNA methylation in colorectal cancer","MethyLight: a high-throughput assay to measure DNA methylation","Comparison between universal molecular screening for Lynch syndrome and revised Bethesda guidelines in a large population-based cohort of patients with colorectal cancer","Aberrant DNA methylation in hereditary nonpolyposis colorectal cancer without mismatch repair deficiency","MSH6 and MUTYH deficiency is a frequent event in early-onset colorectal cancer","DAVID-WS: a stateful web service to facilitate gene/protein list analysis","CpG island methylator phenotype underlies sporadic microsatellite instability and is tightly associated with BRAF mutation in colorectal cancer","Assessment of colorectal cancer molecular features along bowel subsites challenges the conception of distinct dichotomy of proximal versus distal colorectum","Similarities of molecular genetic changes in synchronous and metachronous colorectal cancers are limited and related to the cancers&apos; proximities to each other","The methylator phenotype in microsatellite stable colorectal cancers is characterized by a distinct gene expression profile","Expression and functional significance of HtrA1 loss in endometrial cancer","HtrA1, a potential predictor of response to cisplatin-based combination chemotherapy in gastric cancer","HtrA1 in human urothelial bladder cancer: A secreted protein and a potential novel biomarker","Effects of gut-targeted 15-LOX-1 transgene expression on colonic tumorigenesis in mice","Epigenetic signatures of familial cancer are characteristic of tumor type and family category","CpG island methylation of genes accumulates during the adenoma progression step of the multistep pathogenesis of colorectal cancer"],"volume":["43","95","47","137","19","97","15","98","93","81","5","348","160","31","55","97","5","2","31","5","14","300","8","139","1","529","22","28","61","138","16","28","38","61","15","214","17","58",null,"104","68","45"],"author":["HS Chen","M Latournerie","V Pinol","K Nosho","S Ogino","L Shen","J Young","SL Dykes","DA Lawes","M Pedroni","F Balaguer","HT Lynch","AO Chan","H Kamiyama","P Minoo","E Giovannucci","V Gonzalo","K Konishi","V Gonzalo","K Bardhan","D Colussi","F Gaudet","TM Holm","A Abuli","M Bibikova","KL Gunderson","T Hinoue","CA Eads","L Perez-Carbonell","A Goel","MD Giraldez","X Jiao","DJ Weisenberger","M Yamauchi","P Zauber","M Ferracin","SA Mullany","V Catalano","T Lorenzi","X Zuo","EI Joensuu","YH Kim"],"year":["2000","2008","2004","2009","2006","2005","2006","2003","2005","1999","2007","2003","2002","2012","2006","2005","2010","2009","2008","2013","2013","2003","2005","2010","2009","2009","2012","2000","2012","2010","2010","2012","2006","2012","2013","2008","2011","2011","2013","2012","2008","2006"],"journal.title":["Dis Colon Rectum","Br J Surg","Dis Colon Rectum","Gastroenterology","Mod Pathol","J Natl Cancer Inst","Cancer Epidemiol Biomarkers Prev","Cancer","Br J Cancer","Int J Cancer","Clin Gastroenterol Hepatol","N Engl J Med","Am J Pathol","Oncogene","Gut","J Natl Cancer Inst","PLoS One","Cancer Prev Res (Phila)","Gastroenterol Hepatol","Cancers (Basel)","Int J Mol Sci","Science","Cancer Cell","Gastroenterology","Epigenomics","Methods Mol Biol","Genome Res","Nucleic Acids Res","Gut","Gastroenterology","Clin Cancer Res","Bioinformatics","Nat Genet","Gut","J Mol Diagn","J Pathol","Clin Cancer Res","Histopathology","Int J Cancer","J Natl Cancer Inst","Cancer Res","Genes Chromosomes Cancer"]},{"key":["ref1","ref2","ref3","ref4","ref5","ref6","ref7","ref8","ref9","ref10","ref11","ref12","ref13","ref14","ref15","ref16","ref17","ref18","ref19","ref20","ref21","ref22","ref23","ref24"],"doi.asserted.by":["crossref","crossref","crossref","crossref","crossref","crossref",null,"crossref","crossref","crossref","crossref","crossref",null,"crossref","crossref","crossref","crossref","crossref",null,"crossref","crossref","crossref","crossref","crossref"],"first.page":["643","383","887","39","371","300",null,null,"221",null,"569","302",null,"593","353","433","437","107",null,"250","416","815","399","395"],"DOI":["10.1037/h0054651","10.1016/S1364-6613(00)01530-8","10.1177/0956797611412391","10.1016/j.actpsy.2012.06.005","10.1037/0096-3445.122.3.371","10.1037/h0020586",null,"10.1027/1618-3169/a000234","10.1037/0096-1523.11.2.221","10.1016/S0166-4115(08)61220-0","10.1080/713755894","10.1016/S0010-0285(02)00516-9",null,"10.1037/0096-3445.136.4.593","10.3758/BF03194393","10.1163/156856897X00357","10.1163/156856897X00366","10.1177/001316447303300111",null,"10.3758/BF03195452","10.1037/0033-2909.132.3.416","10.3758/BF03194748","10.1016/j.actpsy.2011.01.006","10.3758/s13423-012-0230-1"],"article.title":["Studies of interference in serial verbal reactions","Interdimensional interference in the Stroop effect: uncovering the cognitive and neural anatomy of attention","Telling things apart: the distance between response keys influences categorization times","Dissociating influences of key and hand separation on the Stroop color-identification effect","The mental representation of parity and number magnitude","Auditory S-R compatibility - Effect of an irrelevant cue on information processing",null,null,"A hand advantage in preparation of simple keypress responses - Reply to Reeve and Proctor (1984)",null,"Stimulus-response compatibility for absolute and relative spatial correspondence in reaching and in button pressing","Preparing for perception and action (I): the role of grouping in the response-cuing paradigm",null,"On the regulation of cognitive control: action orientation moderates the impact of high demands in stroop interference tasks","On the role of stimulus-response and stimulus-stimulus compatibility in the Stroop effect","The psychophysics toolbox","The VideoToolbox software for visual psychophysics: transforming numbers into movies","Eta-squared and partial eta-squared in fixed factor anova designs",null,"Measuring keyboard response delays by comparing keyboard and joystick inputs","Polarity correspondence: a general principle for performance of speeded binary classification tasks","The prevalence effect in two-dimensional stimulus-response compatibility is a function of the relative salience of the dimensions","Response-repetition effects in task switching - Dissociating effects of anatomical and spatial response discriminability","Standard errors and confidence intervals in within-subjects designs: generalizing Loftus and Masson (1994) and avoiding the biases of alternative accounts"],"volume":["18","4","22","141","122","51","88",null,"11",null,"53","46","286","136","31","10","10","33",null,"34","132","64","136","19"],"author":["JR Stroop","CM MacLeod","D Lakens","RW Proctor","S Dehaene","JR Simon",null,null,"J Miller",null,"JF Stins","JJ Adam",null,"NB Jostmann","J De Houwer","DH Brainard","DG Pelli","J Cohen",null,"H Shimizu","RW Proctor","KPL Vu","I Koch","VH Franz"],"year":["1935","2000","2011","2012","1993","1967",null,null,"1985",null,"2000","2003",null,"2007","2003","1997","1997","1973",null,"2002","2006","2002","2011","2012"],"journal.title":["J Exp Psychol","Trends Cogn Sci","Psychol Sci","Acta Psychol","J Exp Psychol Gen","J Appl Psychol",null,null,"J Exp Psychol Human",null,"Q J Exp Psychol-A","Cognitive Psychol",null,"J Exp Psychol Gen","Mem Cognition","Spatial Vision","Spatial Vision","Educ Psychol Meas",null,"Behav Res Meth Ins C","Psychol Bull","Percept Psychophys","Acta Psychol","Psychon B Rev"],"unstructured":[null,null,null,null,null,null,"Wallace RJ (1971) S-R compatibility and idea of a response code. J Exp Psychol <volume>88<\/volume>: : 354-&amp;.","Nett N, Frings C (2013, October 24th) Things can be cold apart: no influence of response categories and labels on the distance effect in Stroop tasks. Exp Psychol. Advance online publication. doi: 10.1027/1618-3169/a000234",null,"Heister G, Schroeder-Heister P, Ehrenstein WH (1990) Spatial coding and spatio-anatomical mapping: evidence for a hierarchical model of spatial stimulus–response compatibility. In: R. W Proctor and T. G Reeve, editors. Stimulus–response compatibility: An integrated perspective. Amsterdam: North-Holland. pp. 117–143.",null,null,"Clark A (2008) Supersizing the mind: embodiment, action, and cognitive extension. pOxford; New York: Oxford University Press. <volume>286<\/volume> p.",null,null,null,null,null,"Cohen J (1988) Statistical power analysis for the behavioral sciences. Hillsdale, N.J.: L. Erlbaum Associates. 567 p. p.",null,null,null,null,null]},null,{"key":["ref1","ref2","ref3","ref4","ref5","ref6","ref7","ref8","ref9","ref10","ref11","ref12","ref13","ref14","ref15","ref16","ref17","ref18","ref19","ref20","ref21","ref22"],"doi.asserted.by":["crossref",null,"crossref","crossref",null,null,null,"crossref","crossref","crossref",null,null,null,null,"crossref",null,"crossref","crossref",null,"crossref",null,null],"first.page":["370","93","12","99",null,null,null,"e5","137","a2390","63","795",null,null,"127","289","720","345",null,"34",null,null],"DOI":["10.1111/j.1601-0825.2005.01132.x",null,"10.1186/1472-6831-6-12","10.1111/j.1600-0714.2007.00598.x",null,null,null,"10.4081/jphr.2013.e5","10.4081/jphr.2012.e20","10.1136/bmj.a2390",null,null,null,null,"10.1111/j.1752-7325.2012.00353.x",null,"10.1016/j.tripleo.2009.11.019","10.1136/sti.74.5.345",null,"10.1186/1478-4491-10-34",null,null],"article.title":["Oral lesions as indicators of HIV infection among routine dental patients in Lagos, Nigeria","HIV related oral manifestations of children and adolescents aged 2–15 years living in Nairobi and Mombasa","Oral manifestations of HIV infection in children and adults receiving highly active anti-retroviral therapy (HAART) in Dar es Salaam","Oral lesions as predictors of highly active antiretroviral therapy failure in Brazilian HIV-infected children",null,null,null,"A community-based oral health promotion model for HIV patients in Nairobi East District in Kenya: a study protocol","Study protocol: Effect of education of primary health care workers on HIV-related oral lesions in Nairobi East district","Improving the reporting of pragmatic trials: an extension of the CONSORT statement","Oral Lesions in HIV Infection in Developing Countries: an Overview. Adv. Dent. Res","The effect of mobile phone text-message reminders on Kenyan health workers’ adherence to malaria treatment guidelines: a cluster randomised trial",null,null,"Attitude, subjective norms and intention to do routine oral examination for oro-pharyngeal candidiasis as perceived by primary health care providers in Nairobi Province","European Economic Community (EEC) Clearinghouse on Oral Problems Related to HIV infection and WHO Collaborating Centre on Oral Manifestation of Human Immunodeficiency Virus. Classification and diagnostic criteria of oral lesions in HIV infection. J. Oral Pathol","Predictive value of group I oral lesions for HIV infection","Oral examination: a screening tool for HIVinfection?",null,"Task-shifting: experiences and opinions of health workers in Mozambique and Zambia",null,null],"volume":["11","1","6","37",null,null,null,"2","1","337","19","378",null,null,"73","22","109","74",null,"10",null,null],"author":["GA Agbelusi","M Anver","OJM Hamza","ID Miziara",null,null,null,"LN Koyio","LN Koyio","M Zwarenstein","K Ranganathan","D Zurovac",null,null,"LN Koyio",null,"A Bhayat","PG Robinson",null,"P Ferrinho",null,null],"year":["2005","2010","2006","2008",null,null,null,"2013","2012","2008","2006","2011",null,null,"2012","1993","2010","1998",null,"2012",null,null],"journal.title":["Oral Diseases","J Kenya Dent Ass","BMC Oral Health","J Oral Pathol Med 2008",null,null,null,"Journal of Public Health Research, 2013","Journal of Public Health Research","BMJ",null,"Lancet 2011",null,null,"Journal of Public Health Dentistry","Med","Oral Surg Oral Med Oral Pathol Oral Radiol Endod","Sex Transm Inf",null,"Human Resources for Health",null,null],"unstructured":[null,null,null,null,"Koyio LN (2014) Knowledge of primary health care providers regarding HIV-related oral facial, and other, common oral diseases and conditions. In: Koyio LN, thesis. Oropharyngeal candidiasis as a marker for HIV infection: impact of community and professional health care workers in Kenya. Nijmegen: Radboud University Nijmegen. 59–72.","Koyio LN (2014) Knowledge of Nairobi East District Community Health Workers on HIV related oral facial lesions and other common oral lesions. In: Koyio LN, thesis. Oropharyngeal candidiasis as a marker for HIV infection: impact of community and professional health care workers in Kenya. Nijmegen: Radboud University Nijmegen. 93–108.","MoH (2006) Taking the Kenya Essential Package for Health to the community. A Strategy for the delivery of level one services. Ministry of Health, 2006. Available: <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"http://marsgroupkenya.org/pdfs/2011/01/Ministry_PDFS/Ministry_of_Public_Health_and_Sanitation/Documents/Taking_the_Kenya_Essential_Package_for_Health_to_the_Community.pdf\" xlink:type=\"simple\">http://marsgroupkenya.org/pdfs/2011/01/Ministry_PDFS/Ministry_of_Public_Health_and_Sanitation/Documents/Taking_the_Kenya_Essential_Package_for_Health_to_the_Community.pdf<\/ext-link>.",null,null,null,null,null,"KDHS (2008–09). Kenya National Bureau of Statistics (KNBS) and ICF Macro. 2010. Kenya Demographic and Health Survey 2008–09. Calverton, Maryland: KNBS and ICF Macro. Kenya demographic and health survey 2008–09. Available: <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"http://www.unfpa.org/sowmy/resources/docs/library/R313_KNBS_2010_Kenya_DHS_2009_final_report.pdfhttp://nascop.or.ke/aboutus.php\" xlink:type=\"simple\">http://www.unfpa.org/sowmy/resources/docs/library/R313_KNBS_2010_Kenya_DHS_2009_final_report.pdfhttp://nascop.or.ke/aboutus.php<\/ext-link>. Accessed 20 July 2013.","MoH (2007) Ministry of Health, Kenya. National AIDS and STI Control Programme. Kenya AIDS Indicator Survey 2007: Preliminary Report. Nairobi, Kenya. July 2008. Available: <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"http://www.prb.org/pdf09/kaiskenyadatasheet.pdf\" xlink:type=\"simple\">http://www.prb.org/pdf09/kaiskenyadatasheet.pdf<\/ext-link>. Accessed 20 August 2013.",null,null,null,null,"Koyio LN (2014) High level drug resistance in patients on chronic antiretroviral treatment presenting with oropharyngeal candidiasis in Kenya. In: Koyio LN, thesis. Oropharyngeal candidiasis as a marker for HIV infection: impact of community and professional health care workers in Kenya. Nijmegen: Radboud University Nijmegen. 135–146.",null,"KESP (2010) Kenya Economic Stimulus Program: Available: <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"http://en.wikipedia.org/wiki/Kenya_Economic_Stimulus_Program\" xlink:type=\"simple\">http://en.wikipedia.org/wiki/Kenya_Economic_Stimulus_Program<\/ext-link>.Accessed 13 August 2013.","Kenya (2011) Kenya: The Sinai Slum Fire Tragedy – Kenya Pipeline Issues Warning.: Available: <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"http://allafrica.com/stories/201109140129.html\" xlink:type=\"simple\">http://allafrica.com/stories/201109140129.html<\/ext-link>. Accessed 25 April 2013."]},null],"update_to":[null,null,{"date":["2014-03-18"],"DOI":["10.1371/journal.pone.0087938"],"type":["correction"],"label":["Correction"]},null,{"date":["2014-03-14"],"DOI":["10.1371/journal.pone.0084532"],"type":["correction"],"label":["Correction"]}],"clinical-trial-number":[null,null,null,{"clinical.trial.number":["ntr2627","ntr2697"],"registry":["10.18810/dutch-trial-register","10.18810/dutch-trial-register"]},null]},"options":{"mode":"view","modes":["code","form","text","tree","view"]}},"evals":[],"jsHooks":[]}</script>
<p>To save the JSON output, first use <code>toJSON()</code>, then <code>write_json()</code> from <code>jsonlite</code>.</p>
<pre class="r"><code># use the toJSON function to convert the output to JSON
plosone_publications_json &lt;- jsonlite::toJSON(plosone_publications)

# write a JSON file
jsonlite::write_json(plosone_publications_json, &quot;plosone_publications.json&quot;)</code></pre>
<p>You can also open the json in an online json viewer such as <a href="https://codebeautify.org/jsonviewer">Code Beautify</a>. Either open a file or paste the JSON output on the left side. Click <strong>Tree Viewer</strong> to view the data.</p>
</div>
<div id="using-cr_works-to-get-data-on-articles" class="section level1">
<h1>Using <code>cr_works()</code> to get data on articles</h1>
<p><code>cr_works()</code> allows you to search by DOI or a general query in order to return the Crossref metadata.</p>
<p>It is important to note, as Crossref does <a href="https://github.com/CrossRef/rest-api-doc/blob/master/demos/crossref-api-demo.ipynb">in the documentation</a>:</p>
<blockquote>
<p>Crossref does not use “works” in the FRBR sense of the word. In Crossref parlance, a “work” is just a thing identified by a DOI. In practice, Crossref DOIs are used as citation identifiers. So, in FRBR terms, this means, that a Crossref DOI tends to refer to one <em>expression</em> which might include multiple <em>manifestations</em>. So, for example, the ePub, HTML and PDF version of an article will share a Crossref DOI because the differences between them should not effect the interpretation or crediting of the content. In short, they can be cited interchangeably. The same is true of the “accepted manuscript” and the “version-of-record” of that accepted manuscript.</p>
</blockquote>
<div id="searching-by-doi" class="section level2">
<h2>Searching by DOI</h2>
<p>You can pass a DOI directly to <code>cr_works()</code> using the <code>dois</code> argument:</p>
<pre class="r"><code># Get metadata for a single article by DOI
jlsc_ku_oa &lt;- cr_works(dois = &quot;10.7710/2162-3309.1252&quot;) %&gt;%
    purrr::pluck(&quot;data&quot;)

# print the data frame with select columns
jlsc_ku_oa %&gt;%
    dplyr::select(title, doi, volume, issue, page, issued, url, publisher, reference.count,
        type, issn)</code></pre>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["title"],"name":[1],"type":["chr"],"align":["left"]},{"label":["doi"],"name":[2],"type":["chr"],"align":["left"]},{"label":["volume"],"name":[3],"type":["chr"],"align":["left"]},{"label":["issue"],"name":[4],"type":["chr"],"align":["left"]},{"label":["page"],"name":[5],"type":["chr"],"align":["left"]},{"label":["issued"],"name":[6],"type":["chr"],"align":["left"]},{"label":["url"],"name":[7],"type":["chr"],"align":["left"]},{"label":["publisher"],"name":[8],"type":["chr"],"align":["left"]},{"label":["reference.count"],"name":[9],"type":["chr"],"align":["left"]},{"label":["type"],"name":[10],"type":["chr"],"align":["left"]},{"label":["issn"],"name":[11],"type":["chr"],"align":["left"]}],"data":[{"1":"Campus Open Access Funds: Experiences of the KU “One University” Open\\n                        Access Author Fund","2":"10.7710/2162-3309.1252","3":"3","4":"1","5":"1252","6":"2015-09-18","7":"http://dx.doi.org/10.7710/2162-3309.1252","8":"Iowa State University","9":"0","10":"journal-article","11":"2162-3309"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>You can also pass more than one DOI. Here we start by assigning our DOIs to a variable <code>my_dois</code>, then pass it to <code>cr_works()</code> in the <code>doi</code> argument:</p>
<pre class="r"><code># Use c() to create a vector of DOIs
my_dois &lt;- c(&quot;10.2139/ssrn.2697412&quot;, &quot;10.1016/j.joi.2016.08.002&quot;, &quot;10.1371/journal.pone.0020961&quot;,
    &quot;10.3389/fpsyg.2018.01487&quot;, &quot;10.1038/d41586-018-00104-7&quot;, &quot;10.12688/f1000research.8460.2&quot;,
    &quot;10.7551/mitpress/9286.001.0001&quot;)

# pass the my_dois vector to cr_works()
my_dois_works &lt;- rcrossref::cr_works(dois = my_dois) %&gt;%
    pluck(&quot;data&quot;)

# print the data frame with select columns
my_dois_works %&gt;%
    dplyr::select(title, doi, volume, issue, page, issued, url, publisher, reference.count,
        type, issn)</code></pre>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["title"],"name":[1],"type":["chr"],"align":["left"]},{"label":["doi"],"name":[2],"type":["chr"],"align":["left"]},{"label":["volume"],"name":[3],"type":["chr"],"align":["left"]},{"label":["issue"],"name":[4],"type":["chr"],"align":["left"]},{"label":["page"],"name":[5],"type":["chr"],"align":["left"]},{"label":["issued"],"name":[6],"type":["chr"],"align":["left"]},{"label":["url"],"name":[7],"type":["chr"],"align":["left"]},{"label":["publisher"],"name":[8],"type":["chr"],"align":["left"]},{"label":["reference.count"],"name":[9],"type":["chr"],"align":["left"]},{"label":["type"],"name":[10],"type":["chr"],"align":["left"]},{"label":["issn"],"name":[11],"type":["chr"],"align":["left"]}],"data":[{"1":"Open Access Publishing: A Literature Review","2":"10.2139/ssrn.2697412","3":"NA","4":"NA","5":"NA","6":"2014","7":"http://dx.doi.org/10.2139/ssrn.2697412","8":"Elsevier BV","9":"0","10":"journal-article","11":"1556-5068"},{"1":"Hybrid open access—A longitudinal study","2":"10.1016/j.joi.2016.08.002","3":"10","4":"4","5":"919-932","6":"2016-11","7":"http://dx.doi.org/10.1016/j.joi.2016.08.002","8":"Elsevier BV","9":"33","10":"journal-article","11":"1751-1577"},{"1":"The Development of Open Access Journal Publishing from 1993 to 2009","2":"10.1371/journal.pone.0020961","3":"6","4":"6","5":"e20961","6":"2011-06-13","7":"http://dx.doi.org/10.1371/journal.pone.0020961","8":"Public Library of Science (PLoS)","9":"31","10":"journal-article","11":"1932-6203"},{"1":"The Impact Factor Fallacy","2":"10.3389/fpsyg.2018.01487","3":"9","4":"NA","5":"NA","6":"2018-08-20","7":"http://dx.doi.org/10.3389/fpsyg.2018.01487","8":"Frontiers Media SA","9":"52","10":"journal-article","11":"1664-1078"},{"1":"Funders should mandate open citations","2":"10.1038/d41586-018-00104-7","3":"553","4":"7687","5":"129-129","6":"2018-01","7":"http://dx.doi.org/10.1038/d41586-018-00104-7","8":"Springer Science and Business Media LLC","9":"0","10":"journal-article","11":"0028-0836,1476-4687"},{"1":"The academic, economic and societal impacts of Open Access: an evidence-based review","2":"10.12688/f1000research.8460.2","3":"5","4":"NA","5":"632","6":"2016-06-09","7":"http://dx.doi.org/10.12688/f1000research.8460.2","8":"F1000 Research Ltd","9":"184","10":"journal-article","11":"2046-1402"},{"1":"Open Access","2":"10.7551/mitpress/9286.001.0001","3":"NA","4":"NA","5":"NA","6":"2012","7":"http://dx.doi.org/10.7551/mitpress/9286.001.0001","8":"The MIT Press","9":"0","10":"book","11":"NA"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="unnesting-list-columns" class="section level2">
<h2>Unnesting list columns</h2>
<p>Authors, links, licenses, funders, and some other values can appear in nested lists when you call <code>cr_journals</code> because there can be, and often are, multiple of each of these items per article. You can check the data classes on all variables by running <code>typeof()</code> across all columns using the <code>map_chr()</code> function from <code>purrr</code>:</p>
<pre class="r"><code># query to get data on a specific PLOS article
plos_article &lt;- cr_works(dois = &quot;10.1371/journal.pone.0228782&quot;) %&gt;%
    purrr::pluck(&quot;data&quot;)

# print the type of each column (e.g. character, numeric, logical, list)
purrr::map_chr(plos_article, typeof)</code></pre>
<pre><code>##        container.title                created              deposited 
##            &quot;character&quot;            &quot;character&quot;            &quot;character&quot; 
##       published.online                    doi                indexed 
##            &quot;character&quot;            &quot;character&quot;            &quot;character&quot; 
##                   issn                  issue                 issued 
##            &quot;character&quot;            &quot;character&quot;            &quot;character&quot; 
##                 member                   page                 prefix 
##            &quot;character&quot;            &quot;character&quot;            &quot;character&quot; 
##              publisher                  score                 source 
##            &quot;character&quot;            &quot;character&quot;            &quot;character&quot; 
##        reference.count       references.count is.referenced.by.count 
##            &quot;character&quot;            &quot;character&quot;            &quot;character&quot; 
##                subject                  title                   type 
##            &quot;character&quot;            &quot;character&quot;            &quot;character&quot; 
##          update.policy                    url                 volume 
##            &quot;character&quot;            &quot;character&quot;            &quot;character&quot; 
##               language  short.container.title                 author 
##            &quot;character&quot;            &quot;character&quot;                 &quot;list&quot; 
##                 funder                   link         content_domain 
##                 &quot;list&quot;                 &quot;list&quot;                 &quot;list&quot; 
##                license              reference 
##                 &quot;list&quot;                 &quot;list&quot;</code></pre>
<p>Our <code>plos_article</code> data frame has a nested list for <strong>author</strong>. We can unnest this column using <code>unnest()</code> from the <code>tidyr</code> package. The <code>.drop = TRUE</code> argument will drop any other list columns.</p>
<pre class="r"><code># unnest author column
plos_article %&gt;%
    tidyr::unnest(author, .drop = TRUE)</code></pre>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["container.title"],"name":[1],"type":["chr"],"align":["left"]},{"label":["created"],"name":[2],"type":["chr"],"align":["left"]},{"label":["deposited"],"name":[3],"type":["chr"],"align":["left"]},{"label":["published.online"],"name":[4],"type":["chr"],"align":["left"]},{"label":["doi"],"name":[5],"type":["chr"],"align":["left"]},{"label":["indexed"],"name":[6],"type":["chr"],"align":["left"]},{"label":["issn"],"name":[7],"type":["chr"],"align":["left"]},{"label":["issue"],"name":[8],"type":["chr"],"align":["left"]},{"label":["issued"],"name":[9],"type":["chr"],"align":["left"]},{"label":["member"],"name":[10],"type":["chr"],"align":["left"]},{"label":["page"],"name":[11],"type":["chr"],"align":["left"]},{"label":["prefix"],"name":[12],"type":["chr"],"align":["left"]},{"label":["publisher"],"name":[13],"type":["chr"],"align":["left"]},{"label":["score"],"name":[14],"type":["chr"],"align":["left"]},{"label":["source"],"name":[15],"type":["chr"],"align":["left"]},{"label":["reference.count"],"name":[16],"type":["chr"],"align":["left"]},{"label":["references.count"],"name":[17],"type":["chr"],"align":["left"]},{"label":["is.referenced.by.count"],"name":[18],"type":["chr"],"align":["left"]},{"label":["subject"],"name":[19],"type":["chr"],"align":["left"]},{"label":["title"],"name":[20],"type":["chr"],"align":["left"]},{"label":["type"],"name":[21],"type":["chr"],"align":["left"]},{"label":["update.policy"],"name":[22],"type":["chr"],"align":["left"]},{"label":["url"],"name":[23],"type":["chr"],"align":["left"]},{"label":["volume"],"name":[24],"type":["chr"],"align":["left"]},{"label":["language"],"name":[25],"type":["chr"],"align":["left"]},{"label":["short.container.title"],"name":[26],"type":["chr"],"align":["left"]},{"label":["ORCID"],"name":[27],"type":["chr"],"align":["left"]},{"label":["authenticated.orcid"],"name":[28],"type":["lgl"],"align":["right"]},{"label":["given"],"name":[29],"type":["chr"],"align":["left"]},{"label":["family"],"name":[30],"type":["chr"],"align":["left"]},{"label":["sequence"],"name":[31],"type":["chr"],"align":["left"]},{"label":["funder"],"name":[32],"type":["list"],"align":["right"]},{"label":["link"],"name":[33],"type":["list"],"align":["right"]},{"label":["content_domain"],"name":[34],"type":["list"],"align":["right"]},{"label":["license"],"name":[35],"type":["list"],"align":["right"]},{"label":["reference"],"name":[36],"type":["list"],"align":["right"]}],"data":[{"1":"PLOS ONE","2":"2020-03-04","3":"2020-03-04","4":"2020-03-04","5":"10.1371/journal.pone.0228782","6":"2020-11-02","7":"1932-6203","8":"3","9":"2020-03-04","10":"340","11":"e0228782","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"73","17":"73","18":"1","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Learning about the Ellsberg Paradox reduces, but does not abolish, ambiguity aversion","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0228782","24":"15","25":"en","26":"PLoS ONE","27":"http://orcid.org/0000-0001-5870-9336","28":"TRUE","29":"Ruonan","30":"Jia","31":"first","32":"<tibble[,4]>","33":"<tibble[,4]>","34":"<df[,2] [1 × 2]>","35":"<tibble[,4]>","36":"<tibble[,10]>"},{"1":"PLOS ONE","2":"2020-03-04","3":"2020-03-04","4":"2020-03-04","5":"10.1371/journal.pone.0228782","6":"2020-11-02","7":"1932-6203","8":"3","9":"2020-03-04","10":"340","11":"e0228782","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"73","17":"73","18":"1","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Learning about the Ellsberg Paradox reduces, but does not abolish, ambiguity aversion","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0228782","24":"15","25":"en","26":"PLoS ONE","27":"NA","28":"NA","29":"Ellen","30":"Furlong","31":"additional","32":"<tibble[,4]>","33":"<tibble[,4]>","34":"<df[,2] [1 × 2]>","35":"<tibble[,4]>","36":"<tibble[,10]>"},{"1":"PLOS ONE","2":"2020-03-04","3":"2020-03-04","4":"2020-03-04","5":"10.1371/journal.pone.0228782","6":"2020-11-02","7":"1932-6203","8":"3","9":"2020-03-04","10":"340","11":"e0228782","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"73","17":"73","18":"1","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Learning about the Ellsberg Paradox reduces, but does not abolish, ambiguity aversion","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0228782","24":"15","25":"en","26":"PLoS ONE","27":"NA","28":"NA","29":"Sean","30":"Gao","31":"additional","32":"<tibble[,4]>","33":"<tibble[,4]>","34":"<df[,2] [1 × 2]>","35":"<tibble[,4]>","36":"<tibble[,10]>"},{"1":"PLOS ONE","2":"2020-03-04","3":"2020-03-04","4":"2020-03-04","5":"10.1371/journal.pone.0228782","6":"2020-11-02","7":"1932-6203","8":"3","9":"2020-03-04","10":"340","11":"e0228782","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"73","17":"73","18":"1","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Learning about the Ellsberg Paradox reduces, but does not abolish, ambiguity aversion","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0228782","24":"15","25":"en","26":"PLoS ONE","27":"NA","28":"NA","29":"Laurie R.","30":"Santos","31":"additional","32":"<tibble[,4]>","33":"<tibble[,4]>","34":"<df[,2] [1 × 2]>","35":"<tibble[,4]>","36":"<tibble[,10]>"},{"1":"PLOS ONE","2":"2020-03-04","3":"2020-03-04","4":"2020-03-04","5":"10.1371/journal.pone.0228782","6":"2020-11-02","7":"1932-6203","8":"3","9":"2020-03-04","10":"340","11":"e0228782","12":"10.1371","13":"Public Library of Science (PLoS)","14":"1","15":"Crossref","16":"73","17":"73","18":"1","19":"General Biochemistry, Genetics and Molecular Biology,General Agricultural and Biological Sciences,General Medicine","20":"Learning about the Ellsberg Paradox reduces, but does not abolish, ambiguity aversion","21":"journal-article","22":"http://dx.doi.org/10.1371/journal.pone.corrections_policy","23":"http://dx.doi.org/10.1371/journal.pone.0228782","24":"15","25":"en","26":"PLoS ONE","27":"NA","28":"NA","29":"Ifat","30":"Levy","31":"additional","32":"<tibble[,4]>","33":"<tibble[,4]>","34":"<df[,2] [1 × 2]>","35":"<tibble[,4]>","36":"<tibble[,10]>"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>We can see this has added 5 rows and 3 new columns: <strong>given</strong> (first name), <strong>family</strong> (last name), and <strong>sequence</strong> (order in which they appeared).</p>
<p>See <a href="https://ciakovx.github.io/rcrossref.html#unnesting_list_columns" class="uri">https://ciakovx.github.io/rcrossref.html#unnesting_list_columns</a> for more detailed strategies in unnesting nested lists in Crossref. For more details, call <code>?unnest</code> and read the <a href="https://r4ds.had.co.nz/many-models.html#unnesting">R for Data Science section on Unnesting</a>.</p>
<hr />
<p><strong>TRY IT YOURSELF</strong></p>
<ol style="list-style-type: decimal">
<li>Refer back to the type of each column above. Try unnesting one of the other list columns. Scroll all the way to the right. What new columns have appeared? Have any rows been duplicated?</li>
</ol>
</div>
<div id="getting-more-than-1000-results-with-the-cursor-argument-to-cr_journals" class="section level2">
<h2>Getting more than 1000 results with the <code>cursor</code> argument to <code>cr_journals</code></h2>
<p>If our result will have more than 1000 results, we have to use the <code>cursor</code> argument. Here we will look at the journal <em>Philosophical Transactions</em>, the longest running scientific journal in the world. We first run <code>cr_journals</code> with <code>works</code> set to <code>FALSE</code> in order to get the journal details, specifically to find out how many articles from this journal are indexed in Crossref.</p>
<pre class="r"><code>philo_issn &lt;- &quot;2053-9223&quot;
philo_details &lt;- rcrossref::cr_journals(philo_issn, works = FALSE) %&gt;%
    pluck(&quot;data&quot;)
philo_details$total_dois</code></pre>
<pre><code>## [1] 8536</code></pre>
<p>Because there are 8,534 articles, we need to pass the <code>cursor</code> argument to <code>cr_journals</code>, which is called “deep paging.” As described by <a href="https://poldham.github.io/abs/crossref.html">Paul Oldham</a>:</p>
<blockquote>
<p>“the CrossRef API also permits deep paging (fetching results from multiple pages). We can retrieve all the results by setting the cursor to the wildcard * and the cursor_max to the total results. Note that the cursor and the cursor_max arguments must appear together for this to work.”</p>
</blockquote>
<p>See more in the <a href="https://github.com/CrossRef/rest-api-doc#deep-paging-with-cursors">API documentation</a></p>
<p>We set <code>cursor</code> to <code>*</code> and <code>max</code> to the maximum number of results to return.</p>
<pre class="r"><code>philo_articles &lt;- rcrossref::cr_journals(philo_issn, works = TRUE, cursor = &quot;*&quot;,
    cursor_max = 8534) %&gt;%
    pluck(&quot;data&quot;)</code></pre>
</div>
<div id="running-general-queries-on-cr_works" class="section level2">
<h2>Running general queries on <code>cr_works()</code></h2>
<p>You can also use <code>cr_works()</code> to run a query based on very simple text keywords. For example, you can run <code>oa_works &amp;lt;- rcrossref::cr_works(query = "open+access")</code>. Paul Oldham <a href="https://poldham.github.io/abs/crossref.html#searching_crossref">gives a great example of this</a>, but does make the comment:</p>
<p>&gt; CrossRef is not a text based search engine and the ability to conduct text based searches is presently crude. Furthermore, we can only search a very limited number of fields and this will inevitably result in lower returns than commercial databases such as Web of Science (where abstracts and author keywords are available). Unfortunately there is no boolean AND for Crossref queries (see <a href="https://github.com/CrossRef/rest-api-doc/issues/135" class="uri">https://github.com/CrossRef/rest-api-doc/issues/135</a> and <a href="https://twitter.com/CrossrefSupport/status/1073601263659610113" class="uri">https://twitter.com/CrossrefSupport/status/1073601263659610113</a>). However, as discussed above, the Crossref API assigns a score to each item returned giving a measure of the API’s confidence in the match, and if you connect words using <code>+</code> the Crossref API will give items with those terms a higher score.</p>
</div>
<div id="specifying-field-queries-to-cr_works-with-flq" class="section level2">
<h2>Specifying field queries to <code>cr_works()</code> with <code>flq</code></h2>
<p>As with <code>cr_journals</code>, you can use <code>flq</code> to pass field queries on to <code>cr_works()</code>, such as author.</p>
<p>Here we search for the book <em>Open Access</em> by Peter Suber by doing a general keyword search for “open access” and an author search for “suber”:</p>
<pre class="r"><code># do a general query for the term open access and a field query to return
# results where the author name includes Suber
suber_oa &lt;- cr_works(query = &quot;open+access&quot;, flq = c(query.author = &quot;suber&quot;)) %&gt;%
    pluck(&quot;data&quot;)

# print the data frame with select columns
suber_oa %&gt;%
    dplyr::select(title, doi, volume, issue, page, issued, url, publisher, reference.count,
        type, issn)</code></pre>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["title"],"name":[1],"type":["chr"],"align":["left"]},{"label":["doi"],"name":[2],"type":["chr"],"align":["left"]},{"label":["volume"],"name":[3],"type":["chr"],"align":["left"]},{"label":["issue"],"name":[4],"type":["chr"],"align":["left"]},{"label":["page"],"name":[5],"type":["chr"],"align":["left"]},{"label":["issued"],"name":[6],"type":["chr"],"align":["left"]},{"label":["url"],"name":[7],"type":["chr"],"align":["left"]},{"label":["publisher"],"name":[8],"type":["chr"],"align":["left"]},{"label":["reference.count"],"name":[9],"type":["chr"],"align":["left"]},{"label":["type"],"name":[10],"type":["chr"],"align":["left"]},{"label":["issn"],"name":[11],"type":["chr"],"align":["left"]}],"data":[{"1":"Open access in the USA","2":"10.1016/b978-1-84334-203-8.50015-7","3":"NA","4":"NA","5":"149-160","6":"2006","7":"http://dx.doi.org/10.1016/b978-1-84334-203-8.50015-7","8":"Elsevier","9":"0","10":"book-chapter","11":"NA"},{"1":"Open Access","2":"10.7551/mitpress/9286.001.0001","3":"NA","4":"NA","5":"NA","6":"2012","7":"http://dx.doi.org/10.7551/mitpress/9286.001.0001","8":"The MIT Press","9":"0","10":"book","11":"NA"},{"1":"Gratis Open Access","2":"10.32388/878441","3":"NA","4":"NA","5":"NA","6":"2019-07-22","7":"http://dx.doi.org/10.32388/878441","8":"Qeios","9":"1","10":"reference-entry","11":"NA"},{"1":"Open Access","2":"10.32388/499151","3":"NA","4":"NA","5":"NA","6":"2019-07-19","7":"http://dx.doi.org/10.32388/499151","8":"Qeios","9":"1","10":"reference-entry","11":"NA"},{"1":"Open Access in 2008","2":"10.3998/3336451.0012.104","3":"12","4":"1","5":"NA","6":"2009-02-15","7":"http://dx.doi.org/10.3998/3336451.0012.104","8":"University of Michigan Library","9":"0","10":"journal-article","11":"1080-2711"},{"1":"Open Access in 2007","2":"10.3998/3336451.0011.110","3":"11","4":"1","5":"NA","6":"2008-01-30","7":"http://dx.doi.org/10.3998/3336451.0011.110","8":"University of Michigan Library","9":"0","10":"journal-article","11":"1080-2711"},{"1":"Knowledge Unbound","2":"10.7551/mitpress/8479.001.0001","3":"NA","4":"NA","5":"NA","6":"2016","7":"http://dx.doi.org/10.7551/mitpress/8479.001.0001","8":"The MIT Press","9":"0","10":"book","11":"NA"},{"1":"Open access: other ways","2":"10.1038/426015b","3":"426","4":"6962","5":"15-15","6":"2003-11","7":"http://dx.doi.org/10.1038/426015b","8":"Springer Science and Business Media LLC","9":"0","10":"journal-article","11":"0028-0836,1476-4687"},{"1":"Promoting Open Access in the Humanities","2":"10.1353/syl.2005.0001","3":"16","4":"1","5":"231-246","6":"2005","7":"http://dx.doi.org/10.1353/syl.2005.0001","8":"Project Muse","9":"0","10":"journal-article","11":"2160-5157"},{"1":"Ensuring open access for publicly funded research","2":"10.1136/bmj.e5184","3":"345","4":"aug08 1","5":"e5184-e5184","6":"2012-08-08","7":"http://dx.doi.org/10.1136/bmj.e5184","8":"BMJ","9":"0","10":"journal-article","11":"1756-1833"},{"1":"Open access?","2":"10.1038/ngeo450","3":"2","4":"3","5":"155-155","6":"2009-03","7":"http://dx.doi.org/10.1038/ngeo450","8":"Springer Science and Business Media LLC","9":"1","10":"journal-article","11":"1752-0894,1752-0908"},{"1":"Open Access to Electronic Theses and Dissertations","2":"10.14429/djlit.28.1.149","3":"28","4":"1","5":"25-34","6":"2008-01-01","7":"http://dx.doi.org/10.14429/djlit.28.1.149","8":"Defence Scientific Information and Documentation Centre","9":"0","10":"journal-article","11":"0974-0643,0976-4658"},{"1":"Open Access and Quality","2":"10.14429/djlit.28.1.152","3":"28","4":"1","5":"49-56","6":"2008-01-01","7":"http://dx.doi.org/10.14429/djlit.28.1.152","8":"Defence Scientific Information and Documentation Centre","9":"0","10":"journal-article","11":"0974-0643,0976-4658"},{"1":"Thoughts on prestige, quality, and open access","2":"10.1163/095796510x546959","3":"21","4":"1-2","5":"115-128","6":"2010","7":"http://dx.doi.org/10.1163/095796510x546959","8":"Brill","9":"0","10":"journal-article","11":"0957-9656,1878-4712"},{"1":"Author rights and the Harvard open access policies: a response to Patrick Alexander","2":"10.1629/uksg.543","3":"34","4":"NA","5":"NA","6":"2021","7":"http://dx.doi.org/10.1629/uksg.543","8":"Ubiquity Press, Ltd.","9":"4","10":"journal-article","11":"2048-7754"},{"1":"Open access, impact, and demand","2":"10.1136/bmj.330.7500.1097","3":"330","4":"7500","5":"1097-1098","6":"2005-05-12","7":"http://dx.doi.org/10.1136/bmj.330.7500.1097","8":"BMJ","9":"0","10":"journal-article","11":"0959-8138,1468-5833"},{"1":"SCHOLARLY COMMUNICATION: Removing barriers to research: An introduction to open access for librarians","2":"10.5860/crln.64.2.92","3":"64","4":"2","5":"92-113","6":"2003-02-01","7":"http://dx.doi.org/10.5860/crln.64.2.92","8":"American Library Association","9":"0","10":"journal-article","11":"2150-6698,0099-0086"},{"1":"A living open book","2":"10.5334/bal.l","3":"NA","4":"NA","5":"113-117","6":"2014-11-28","7":"http://dx.doi.org/10.5334/bal.l","8":"Ubiquity Press","9":"0","10":"book-chapter","11":"NA"},{"1":"Public access to federally funded research: The Cornyn-Lieberman and CURES bills","2":"10.5860/crln.67.6.7637","3":"67","4":"6","5":"362-365","6":"2006-06-01","7":"http://dx.doi.org/10.5860/crln.67.6.7637","8":"American Library Association","9":"0","10":"journal-article","11":"2150-6698,0099-0086"},{"1":"Support for the NIH Public Access Policy","2":"10.1126/science.313.5793.1572a","3":"313","4":"5793","5":"1572a-1572a","6":"2006-09-15","7":"http://dx.doi.org/10.1126/science.313.5793.1572a","8":"American Association for the Advancement of Science (AAAS)","9":"0","10":"journal-article","11":"0036-8075,1095-9203"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>Dr. Suber has written lots of materials that includes the term “open access.” We can use the <code>filter()</code> function from <code>dplyr</code> to look only at books, from the <strong>type</strong> column:</p>
<pre class="r"><code># use filter() from dplyr to filter that result to include only books
suber_oa_books &lt;- suber_oa %&gt;%
    filter(type == &quot;book&quot;)

# print the data frame with select columns
suber_oa_books %&gt;%
    dplyr::select(title, doi, volume, issue, page, issued, url, publisher, reference.count,
        type, issn)</code></pre>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["title"],"name":[1],"type":["chr"],"align":["left"]},{"label":["doi"],"name":[2],"type":["chr"],"align":["left"]},{"label":["volume"],"name":[3],"type":["chr"],"align":["left"]},{"label":["issue"],"name":[4],"type":["chr"],"align":["left"]},{"label":["page"],"name":[5],"type":["chr"],"align":["left"]},{"label":["issued"],"name":[6],"type":["chr"],"align":["left"]},{"label":["url"],"name":[7],"type":["chr"],"align":["left"]},{"label":["publisher"],"name":[8],"type":["chr"],"align":["left"]},{"label":["reference.count"],"name":[9],"type":["chr"],"align":["left"]},{"label":["type"],"name":[10],"type":["chr"],"align":["left"]},{"label":["issn"],"name":[11],"type":["chr"],"align":["left"]}],"data":[{"1":"Open Access","2":"10.7551/mitpress/9286.001.0001","3":"NA","4":"NA","5":"NA","6":"2012","7":"http://dx.doi.org/10.7551/mitpress/9286.001.0001","8":"The MIT Press","9":"0","10":"book","11":"NA"},{"1":"Knowledge Unbound","2":"10.7551/mitpress/8479.001.0001","3":"NA","4":"NA","5":"NA","6":"2016","7":"http://dx.doi.org/10.7551/mitpress/8479.001.0001","8":"The MIT Press","9":"0","10":"book","11":"NA"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>One is the book from MIT Press that we’re looking for; the other is <em>Knowledge Unbound</em>, which is a collection of his writings.</p>
<p>We could be more specific from the outset by adding bibliographic information in <code>query.bibliographic</code>, such as ISBN (or ISSN, if it’s a journal):</p>
<pre class="r"><code># run a different cr_works() query with author set to Suber and his book&#39;s ISBN
# passed to query.bibliographic
suber_isbn &lt;- cr_works(flq = c(query.author = &quot;suber&quot;, query.bibliographic = &quot;9780262301732&quot;)) %&gt;%
    pluck(&quot;data&quot;)

# print the data frame with select columns
suber_isbn %&gt;%
    dplyr::select(title, doi, issued, url, publisher, type, author)</code></pre>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["title"],"name":[1],"type":["chr"],"align":["left"]},{"label":["doi"],"name":[2],"type":["chr"],"align":["left"]},{"label":["issued"],"name":[3],"type":["chr"],"align":["left"]},{"label":["url"],"name":[4],"type":["chr"],"align":["left"]},{"label":["publisher"],"name":[5],"type":["chr"],"align":["left"]},{"label":["type"],"name":[6],"type":["chr"],"align":["left"]},{"label":["author"],"name":[7],"type":["list"],"align":["right"]}],"data":[{"1":"Open Access","2":"10.7551/mitpress/9286.001.0001","3":"2012","4":"http://dx.doi.org/10.7551/mitpress/9286.001.0001","5":"The MIT Press","6":"book","7":"<tibble[,3]>"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>You can combine the <code>filter</code> argument with <code>flq</code> to return only items of <strong>type</strong> <code>book</code> published in 2012.</p>
</div>
<div id="getting-formatted-references-in-a-text-file" class="section level2">
<h2>Getting formatted references in a text file</h2>
<p>We can use the <code>cr_cn()</code> function from the <code>rcrossref</code> package to get the citations to those articles in text form in the style you specify. We’ll put it into Chicago. The <code>cr_cn()</code> function returns each citation into a list element. We can use the <code>map_chr</code> and the <code>pluck</code> functions from <code>purrr</code> to instead assign them to a character vector.</p>
<pre class="r"><code># Use c() to create a vector of DOIs
my_dois &lt;- c(&quot;10.2139/ssrn.2697412&quot;, &quot;10.1016/j.joi.2016.08.002&quot;, &quot;10.1371/journal.pone.0020961&quot;,
    &quot;10.3389/fpsyg.2018.01487&quot;, &quot;10.1038/d41586-018-00104-7&quot;, &quot;10.12688/f1000research.8460.2&quot;,
    &quot;10.7551/mitpress/9286.001.0001&quot;)

# Use cr_cn to get back citations formatted in Chicago for those DOIs
my_citations &lt;- rcrossref::cr_cn(my_dois, format = &quot;text&quot;, style = &quot;chicago-note-bibliography&quot;) %&gt;%
    purrr::map_chr(., purrr::pluck, 1)

# print the formatted citations
my_citations</code></pre>
<pre><code>## [1] &quot;Frosio, Giancarlo F. “Open Access Publishing: A Literature Review.” SSRN Electronic Journal (2014). doi:10.2139/ssrn.2697412.&quot;                                                                                                                                                              
## [2] &quot;Laakso, Mikael, and Bo-Christer Björk. “Hybrid Open access—A Longitudinal Study.” Journal of Informetrics 10, no. 4 (November 2016): 919–932. doi:10.1016/j.joi.2016.08.002.&quot;                                                                                                               
## [3] &quot;Laakso, Mikael, Patrik Welling, Helena Bukvova, Linus Nyman, Bo-Christer Björk, and Turid Hedlund. “The Development of Open Access Journal Publishing from 1993 to 2009.” Edited by Marcelo Hermes-Lima. PLoS ONE 6, no. 6 (June 13, 2011): e20961. doi:10.1371/journal.pone.0020961.&quot;      
## [4] &quot;Paulus, Frieder M., Nicole Cruz, and Sören Krach. “The Impact Factor Fallacy.” Frontiers in Psychology 9 (August 20, 2018). doi:10.3389/fpsyg.2018.01487.&quot;                                                                                                                                  
## [5] &quot;Shotton, David. “Funders Should Mandate Open Citations.” Nature 553, no. 7687 (January 2018): 129–129. doi:10.1038/d41586-018-00104-7.&quot;                                                                                                                                                     
## [6] &quot;Tennant, Jonathan P., François Waldner, Damien C. Jacques, Paola Masuzzo, Lauren B. Collister, and Chris. H. J. Hartgerink. “The Academic, Economic and Societal Impacts of Open Access: An Evidence-Based Review.” F1000Research 5 (June 9, 2016): 632. doi:10.12688/f1000research.8460.2.&quot;
## [7] &quot;Suber, Peter. “Open Access” (2012). doi:10.7551/mitpress/9286.001.0001.&quot;</code></pre>
<p>Beautiful formatted citations from simply a list of DOIs! You can then write this to a text file using <code>writeLines</code>.</p>
<pre class="r"><code># write the formatted citations to a text file
writeLines(my_citations, &quot;my_citations_text.txt&quot;)</code></pre>
<p>The above is helpful if you need to paste the references somewhere, and there are loads of other citation styles included in <code>rcrossref</code>–view them by calling <code>rcrossref::get_styles()</code> and it will print a vector of these styles to your console. I’ll just print the first 15 below:</p>
<pre class="r"><code># look at the first 15 styles Crossref offers
rcrossref::get_styles()[1:15]</code></pre>
<pre><code>##  [1] &quot;academy-of-management-review&quot;                                
##  [2] &quot;accident-analysis-and-prevention&quot;                            
##  [3] &quot;aci-materials-journal&quot;                                       
##  [4] &quot;acm-sig-proceedings-long-author-list&quot;                        
##  [5] &quot;acm-sig-proceedings&quot;                                         
##  [6] &quot;acm-sigchi-proceedings-extended-abstract-format&quot;             
##  [7] &quot;acm-sigchi-proceedings&quot;                                      
##  [8] &quot;acm-siggraph&quot;                                                
##  [9] &quot;acme-an-international-journal-for-critical-geographies&quot;      
## [10] &quot;acta-amazonica&quot;                                              
## [11] &quot;acta-anaesthesiologica-scandinavica&quot;                         
## [12] &quot;acta-anaesthesiologica-taiwanica&quot;                            
## [13] &quot;acta-botanica-croatica&quot;                                      
## [14] &quot;acta-chiropterologica&quot;                                       
## [15] &quot;acta-chirurgiae-orthopaedicae-et-traumatologiae-cechoslovaca&quot;</code></pre>
</div>
<div id="getting-formatted-references-in-a-bibtex-or-ris-file" class="section level2">
<h2>Getting formatted references in a BibTeX or RIS file</h2>
<p>In addition to a text file, you can also write it to BibTeX or RIS:</p>
<pre class="r"><code># Use cr_cn() to get BibTeX files for my DOIs
my_citations_bibtex &lt;- rcrossref::cr_cn(my_dois, format = &quot;bibtex&quot;) %&gt;%
    purrr::map_chr(., purrr::pluck, 1)</code></pre>
<p>Write it to a .bib file using <code>writeLines()</code>:</p>
<pre class="r"><code># write to bibtex file
writeLines(my_citations_bibtex, &quot;my_citations_bibtex.bib&quot;)</code></pre>
</div>
<div id="getting-works-from-a-typed-citation-in-a-word-documenttext-file" class="section level2">
<h2>Getting works from a typed citation in a Word document/text file</h2>
<p>This can be helpful if you have a bibliography in a Word document or text file that you want to get into a reference management tool like Zotero. For instance, you may have written the citations in APA style and need to change to Chicago, but don’t want to rekey it all out. Or perhaps you jotted down your citations hastily and left out volume, issue, or page numbers, and you need a nice, fully-formatted citation.</p>
<p>If each citation is on its own line in your document’s bibliography, then you can probably paste the whole bibliography into an Excel spreadsheet. If it goes as planned, each citation will be in its own cell. You can then save it to a CSV file, which can then be read into R.</p>
<pre class="r"><code># read in a CSV file of citations
my_references &lt;- read.csv(&quot;data/references.txt&quot;, stringsAsFactors = FALSE)

# print the file
my_references</code></pre>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["reference"],"name":[1],"type":["chr"],"align":["left"]}],"data":[{"1":"Frosio, G. F. (2014). Open Access Publishing: A Literature Review. SSRN Electronic Journal. https://doi.org/10.2139/ssrn.2697412"},{"1":"Laakso, M., & Bjork, B.-C. (2016). Hybrid open access: A longitudinal study. Journal of Informetrics, 10(4), 919-932. https://doi.org/10.1016/j.joi.2016.08.002"},{"1":"Laakso, M., Welling, P., Bukvova, H., Nyman, L., Bjork, B.-C., & Hedlund, T. (2011). The Development of Open Access Journal Publishing from 1993 to 2009. PLoS ONE, 6(6), e20961. https://doi.org/10.1371/journal.pone.0020961"},{"1":"Paulus, F. M., Cruz, N., & Krach, S. (2018). The Impact Factor Fallacy. Frontiers in Psychology, 9. https://doi.org/10.3389/fpsyg.2018.01487"},{"1":"Science, Digital; Hook, Daniel; Hahnel, Mark; Calvert, Ian (2019): The Ascent of Open Access. figshare. Journal contribution. https://doi.org/10.6084/m9.figshare.7618751"},{"1":"Shotton, D. (2018). Funders should mandate open citations. Nature, 553(7687), 129-129. https://doi.org/10.1038/d41586-018-00104-7"},{"1":"Suber, P. (2012). Open access. Cambridge, Mass: MIT Press."},{"1":"Tennant, J. P., Waldner, F., Jacques, D. C., Masuzzo, P., Collister, L. B., & Hartgerink, C. H. J. (2016). The academic, economic and societal impacts of Open Access: an evidence-based review. F1000Research, 5, 632. https://doi.org/10.12688/f1000research.8460.2"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>As you can see, these are just raw citations, not divided into variables by their metadata elements (that is, with title in one column, author in another, etc.). But, we can now run a query to get precisely that from Crossref using <code>cr_works</code>. Because <code>cr_works</code> is not vectorized, we will need to build a loop using <code>map()</code> from the <code>purrr</code> package.</p>
<p>Don’t mind the technical details–it is basically saying to take each row and look it up in the Crossref search engine. Basically, this is the equivalent of copy/pasting the whole reference into the Crossref search engine. The loop will <code>print()</code> the citation before searching for it so we can keep track of where it is. We set the <code>limit</code> to 5 because if Crossref didn’t find it in the first 5 results, it’s not likely to be there at all.</p>
<pre class="r"><code># loop through the references column, using cr_works() to look the item up and
# return the top 5 hits
my_references_works_list &lt;- purrr::map(my_references$reference, function(x) {
    print(x)
    my_works &lt;- rcrossref::cr_works(query = x, limit = 5) %&gt;%
        purrr::pluck(&quot;data&quot;)
})</code></pre>
<pre><code>## [1] &quot;Frosio, G. F. (2014). Open Access Publishing: A Literature Review. SSRN Electronic Journal. https://doi.org/10.2139/ssrn.2697412&quot;
## [1] &quot;Laakso, M., &amp; Bjork, B.-C. (2016). Hybrid open access: A longitudinal study. Journal of Informetrics, 10(4), 919-932. https://doi.org/10.1016/j.joi.2016.08.002&quot;
## [1] &quot;Laakso, M., Welling, P., Bukvova, H., Nyman, L., Bjork, B.-C., &amp; Hedlund, T. (2011). The Development of Open Access Journal Publishing from 1993 to 2009. PLoS ONE, 6(6), e20961. https://doi.org/10.1371/journal.pone.0020961&quot;
## [1] &quot;Paulus, F. M., Cruz, N., &amp; Krach, S. (2018). The Impact Factor Fallacy. Frontiers in Psychology, 9. https://doi.org/10.3389/fpsyg.2018.01487&quot;
## [1] &quot;Science, Digital; Hook, Daniel; Hahnel, Mark; Calvert, Ian (2019): The Ascent of Open Access. figshare. Journal contribution. https://doi.org/10.6084/m9.figshare.7618751&quot;
## [1] &quot;Shotton, D. (2018). Funders should mandate open citations. Nature, 553(7687), 129-129. https://doi.org/10.1038/d41586-018-00104-7&quot;
## [1] &quot;Suber, P. (2012). Open access. Cambridge, Mass: MIT Press.&quot;
## [1] &quot;Tennant, J. P., Waldner, F., Jacques, D. C., Masuzzo, P., Collister, L. B., &amp; Hartgerink, C. H. J. (2016). The academic, economic and societal impacts of Open Access: an evidence-based review. F1000Research, 5, 632. https://doi.org/10.12688/f1000research.8460.2&quot;</code></pre>
<p>The Crossref API assigns a score to each item returned within each query, giving a measure of the API’s confidence in the match. The item with the highest score is returned first in the datasets. We can return the first result in each item in the <code>my_references_works_list</code> by using <code>map_dfr()</code>, which is like <code>map()</code> except it returns the results into a data frame rather than a list:</p>
<pre class="r"><code># for each reference looked up, get back the first result
my_references_works_df &lt;- my_references_works_list %&gt;%
    purrr::map_dfr(., function(x) {
        x[1, ]
    })

# print the data frame with select columns
my_references_works_df %&gt;%
    dplyr::select(title, doi, volume, issue, page, issued, url, publisher, reference.count,
        type, issn)</code></pre>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["title"],"name":[1],"type":["chr"],"align":["left"]},{"label":["doi"],"name":[2],"type":["chr"],"align":["left"]},{"label":["volume"],"name":[3],"type":["chr"],"align":["left"]},{"label":["issue"],"name":[4],"type":["chr"],"align":["left"]},{"label":["page"],"name":[5],"type":["chr"],"align":["left"]},{"label":["issued"],"name":[6],"type":["chr"],"align":["left"]},{"label":["url"],"name":[7],"type":["chr"],"align":["left"]},{"label":["publisher"],"name":[8],"type":["chr"],"align":["left"]},{"label":["reference.count"],"name":[9],"type":["chr"],"align":["left"]},{"label":["type"],"name":[10],"type":["chr"],"align":["left"]},{"label":["issn"],"name":[11],"type":["chr"],"align":["left"]}],"data":[{"1":"Open Access Publishing: A Literature Review","2":"10.2139/ssrn.2697412","3":"NA","4":"NA","5":"NA","6":"2014","7":"http://dx.doi.org/10.2139/ssrn.2697412","8":"Elsevier BV","9":"0","10":"journal-article","11":"1556-5068"},{"1":"Hybrid open access—A longitudinal study","2":"10.1016/j.joi.2016.08.002","3":"10","4":"4","5":"919-932","6":"2016-11","7":"http://dx.doi.org/10.1016/j.joi.2016.08.002","8":"Elsevier BV","9":"33","10":"journal-article","11":"1751-1577"},{"1":"The Development of Open Access Journal Publishing from 1993 to 2009","2":"10.1371/journal.pone.0020961","3":"6","4":"6","5":"e20961","6":"2011-06-13","7":"http://dx.doi.org/10.1371/journal.pone.0020961","8":"Public Library of Science (PLoS)","9":"31","10":"journal-article","11":"1932-6203"},{"1":"The Impact Factor Fallacy","2":"10.3389/fpsyg.2018.01487","3":"9","4":"NA","5":"NA","6":"2018-08-20","7":"http://dx.doi.org/10.3389/fpsyg.2018.01487","8":"Frontiers Media SA","9":"52","10":"journal-article","11":"1664-1078"},{"1":"Table S1: Digital copies of visualizations, alignments, and phylogenetic trees deposited at Figshare (also accessible under Figshare Collection doi:10.6084/m9.figshare.c.3521787)","2":"10.7717/peerj.2951/supp-3","3":"NA","4":"NA","5":"NA","6":"NA","7":"http://dx.doi.org/10.7717/peerj.2951/supp-3","8":"PeerJ","9":"0","10":"component","11":"NA"},{"1":"Funders should mandate open citations","2":"10.1038/d41586-018-00104-7","3":"553","4":"7687","5":"129-129","6":"2018-01","7":"http://dx.doi.org/10.1038/d41586-018-00104-7","8":"Springer Science and Business Media LLC","9":"0","10":"journal-article","11":"0028-0836,1476-4687"},{"1":"Open Access by Peter Suber. MIT Press, Cambridge, MA, U.S.A., 2012. 230 pp. ISBN: 978-0- 262-51763-8","2":"10.1162/leon_r_00535","3":"46","4":"2","5":"180-181","6":"2013-04","7":"http://dx.doi.org/10.1162/leon_r_00535","8":"MIT Press - Journals","9":"3","10":"journal-article","11":"0024-094X,1530-9282"},{"1":"The academic, economic and societal impacts of Open Access: an evidence-based review","2":"10.12688/f1000research.8460.3","3":"5","4":"NA","5":"632","6":"2016-09-21","7":"http://dx.doi.org/10.12688/f1000research.8460.3","8":"F1000 Research Ltd","9":"185","10":"journal-article","11":"2046-1402"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>We can print just the titles to quickly see how well they match with the titles of the works we requested:</p>
<pre class="r"><code># print the title column
my_references_works_df$title</code></pre>
<pre><code>## [1] &quot;Open Access Publishing: A Literature Review&quot;                                                                                                                                       
## [2] &quot;Hybrid open access—A longitudinal study&quot;                                                                                                                                           
## [3] &quot;The Development of Open Access Journal Publishing from 1993 to 2009&quot;                                                                                                               
## [4] &quot;The Impact Factor Fallacy&quot;                                                                                                                                                         
## [5] &quot;Table S1: Digital copies of visualizations, alignments, and phylogenetic trees deposited at Figshare (also accessible under Figshare Collection doi:10.6084/m9.figshare.c.3521787)&quot;
## [6] &quot;Funders should mandate open citations&quot;                                                                                                                                             
## [7] &quot;Open Access by Peter Suber. MIT Press, Cambridge, MA, U.S.A., 2012. 230 pp. ISBN: 978-0- 262-51763-8&quot;                                                                              
## [8] &quot;The academic, economic and societal impacts of Open Access: an evidence-based review&quot;</code></pre>
<p>Not bad! Not bad! Looks like we got 6 out of 8, with problems on number 5 and 7.</p>
<p>Let’s deal with 5 first. This was the result for “The Ascent of Open Access”, which was a report by Digital Science posted to figshare, didn’t come back. Even though this report does have a DOI (<a href="https://doi.org/10.6084/m9.figshare.7618751.v2" class="uri">https://doi.org/10.6084/m9.figshare.7618751.v2</a>) assigned via figshare, the <code>cr_works()</code> function searches only for Crossref DOIs. We should check to see if it came back in any of the 5 items we pulled. We do this by calling <code>pluck()</code> on the titles of the fifth item in the list:</p>
<pre class="r"><code>my_references_works_list %&gt;%
    purrr::pluck(5, &quot;title&quot;)</code></pre>
<pre><code>## [1] &quot;Table S1: Digital copies of visualizations, alignments, and phylogenetic trees deposited at Figshare (also accessible under Figshare Collection doi:10.6084/m9.figshare.c.3521787)&quot;                                          
## [2] &quot;Figure 4 from: Miko I, Ernst A, Deans A (2013) Morphology and function of the ovipositor mechanism in Ceraphronoidea (Hymenoptera, Apocrita). Journal of Hymenoptera Research 33: 25-61. https://doi.org/10.3897/jhr.33.5204&quot;
## [3] &quot;Figure 1 from: Miko I, Ernst A, Deans A (2013) Morphology and function of the ovipositor mechanism in Ceraphronoidea (Hymenoptera, Apocrita). Journal of Hymenoptera Research 33: 25-61. https://doi.org/10.3897/jhr.33.5204&quot;
## [4] &quot;Figure 6 from: Miko I, Popovici O, Seltmann K, Deans A (2014) The maxillo-labial complex of Sparasion (Hymenoptera, Platygastroidea). Journal of Hymenoptera Research 37: 77-111. https://doi.org/10.3897/jhr.37.5206&quot;       
## [5] &quot;Figure 3 from: Miko I, Ernst A, Deans A (2013) Morphology and function of the ovipositor mechanism in Ceraphronoidea (Hymenoptera, Apocrita). Journal of Hymenoptera Research 33: 25-61. https://doi.org/10.3897/jhr.33.5204&quot;</code></pre>
<p>Nope, unfortunately none of these are “The Ascent of Open Access”, so we’re out of luck. We can just throw this row out entirely using <code>slice()</code> from <code>dplyr</code>. We’ll overwrite our existing <code>my_references_works_df</code> because we have no future use for it in this R session.</p>
<pre class="r"><code>my_references_works_df &lt;- my_references_works_df %&gt;%
    dplyr::slice(-5)</code></pre>
<p>For row 7, it’s giving us the full citation for Peter Suber’s book when we asked for the title only, so something is fishy.</p>
<p>When we look at it more closely (we can call <code>View(my_references_works_df)</code>), we see the author of this item is not Peter Suber, but Rob Harle, and, checking the <strong>type</strong> column, it’s a journal article, not a book. This is a book review published in the journal <em>Leonardo</em>, not Peter Suber’s book. So let’s go back to <code>my_references_works_list</code> and pull data from all 5 items that came back with the API call and see if Suber’s book is in there somewhere:</p>
<pre class="r"><code>suber &lt;- my_references_works_list %&gt;%
    purrr::pluck(7)
suber</code></pre>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["alternative.id"],"name":[1],"type":["chr"],"align":["left"]},{"label":["container.title"],"name":[2],"type":["chr"],"align":["left"]},{"label":["created"],"name":[3],"type":["chr"],"align":["left"]},{"label":["deposited"],"name":[4],"type":["chr"],"align":["left"]},{"label":["published.print"],"name":[5],"type":["chr"],"align":["left"]},{"label":["doi"],"name":[6],"type":["chr"],"align":["left"]},{"label":["indexed"],"name":[7],"type":["chr"],"align":["left"]},{"label":["issn"],"name":[8],"type":["chr"],"align":["left"]},{"label":["issue"],"name":[9],"type":["chr"],"align":["left"]},{"label":["issued"],"name":[10],"type":["chr"],"align":["left"]},{"label":["member"],"name":[11],"type":["chr"],"align":["left"]},{"label":["page"],"name":[12],"type":["chr"],"align":["left"]},{"label":["prefix"],"name":[13],"type":["chr"],"align":["left"]},{"label":["publisher"],"name":[14],"type":["chr"],"align":["left"]},{"label":["score"],"name":[15],"type":["chr"],"align":["left"]},{"label":["source"],"name":[16],"type":["chr"],"align":["left"]},{"label":["reference.count"],"name":[17],"type":["chr"],"align":["left"]},{"label":["references.count"],"name":[18],"type":["chr"],"align":["left"]},{"label":["is.referenced.by.count"],"name":[19],"type":["chr"],"align":["left"]},{"label":["subject"],"name":[20],"type":["chr"],"align":["left"]},{"label":["title"],"name":[21],"type":["chr"],"align":["left"]},{"label":["type"],"name":[22],"type":["chr"],"align":["left"]},{"label":["url"],"name":[23],"type":["chr"],"align":["left"]},{"label":["volume"],"name":[24],"type":["chr"],"align":["left"]},{"label":["language"],"name":[25],"type":["chr"],"align":["left"]},{"label":["short.container.title"],"name":[26],"type":["chr"],"align":["left"]},{"label":["author"],"name":[27],"type":["list"],"align":["right"]},{"label":["link"],"name":[28],"type":["list"],"align":["right"]},{"label":["content_domain"],"name":[29],"type":["list"],"align":["right"]},{"label":["reference"],"name":[30],"type":["list"],"align":["right"]},{"label":["isbn"],"name":[31],"type":["chr"],"align":["left"]},{"label":["published.online"],"name":[32],"type":["chr"],"align":["left"]},{"label":["license"],"name":[33],"type":["list"],"align":["right"]}],"data":[{"1":"10.1162/LEON_r_00535","2":"Leonardo","3":"2013-01-17","4":"2021-03-12","5":"2013-04","6":"10.1162/leon_r_00535","7":"2021-03-12","8":"0024-094X,1530-9282","9":"2","10":"2013-04","11":"281","12":"180-181","13":"10.1162","14":"MIT Press - Journals","15":"41.597294","16":"Crossref","17":"3","18":"3","19":"0","20":"Engineering (miscellaneous),Visual Arts and Performing Arts,Music,Computer Science Applications","21":"Open Access by Peter Suber. MIT Press, Cambridge, MA, U.S.A., 2012. 230 pp. ISBN: 978-0- 262-51763-8","22":"journal-article","23":"http://dx.doi.org/10.1162/leon_r_00535","24":"46","25":"en","26":"Leonardo","27":"<tibble[,4]>","28":"<tibble[,4]>","29":"<df[,2] [1 × 2]>","30":"<tibble[,5]>","31":"NA","32":"NA","33":"<NULL>"},{"1":"NA","2":"NA","3":"2019-01-11","4":"2019-01-11","5":"2012","6":"10.7551/mitpress/9286.001.0001","7":"2021-06-29","8":"NA","9":"NA","10":"2012","11":"281","12":"NA","13":"10.7551","14":"The MIT Press","15":"39.16521","16":"Crossref","17":"0","18":"0","19":"254","20":"NA","21":"Open Access","22":"book","23":"http://dx.doi.org/10.7551/mitpress/9286.001.0001","24":"NA","25":"NA","26":"NA","27":"<tibble[,3]>","28":"<NULL>","29":"<df[,2] [1 × 2]>","30":"<NULL>","31":"9780262301732","32":"NA","33":"<NULL>"},{"1":"NA","2":"Definitions","3":"2019-07-22","4":"2019-11-11","5":"NA","6":"10.32388/878441","7":"2021-07-16","8":"NA","9":"NA","10":"2019-07-22","11":"17262","12":"NA","13":"10.32388","14":"Qeios","15":"34.18894","16":"Crossref","17":"1","18":"1","19":"2","20":"NA","21":"Gratis Open Access","22":"reference-entry","23":"http://dx.doi.org/10.32388/878441","24":"NA","25":"NA","26":"NA","27":"<tibble[,3]>","28":"<NULL>","29":"<df[,2] [1 × 2]>","30":"<tibble[,2]>","31":"NA","32":"2019-07-22","33":"<NULL>"},{"1":"NA","2":"Definitions","3":"2019-07-19","4":"2019-11-11","5":"NA","6":"10.32388/499151","7":"2021-02-25","8":"NA","9":"NA","10":"2019-07-19","11":"17262","12":"NA","13":"10.32388","14":"Qeios","15":"34.18894","16":"Crossref","17":"1","18":"1","19":"2","20":"NA","21":"Open Access","22":"reference-entry","23":"http://dx.doi.org/10.32388/499151","24":"NA","25":"NA","26":"NA","27":"<tibble[,3]>","28":"<NULL>","29":"<df[,2] [1 × 2]>","30":"<tibble[,2]>","31":"NA","32":"2019-07-19","33":"<NULL>"},{"1":"NA","2":"Open Access","3":"2014-03-28","4":"2018-10-05","5":"2006","6":"10.1016/b978-1-84334-203-8.50015-7","7":"2020-04-09","8":"NA","9":"NA","10":"2006","11":"78","12":"149-160","13":"10.1016","14":"Elsevier","15":"33.466602","16":"Crossref","17":"0","18":"0","19":"5","20":"NA","21":"Open access in the USA","22":"book-chapter","23":"http://dx.doi.org/10.1016/b978-1-84334-203-8.50015-7","24":"NA","25":"NA","26":"NA","27":"<tibble[,3]>","28":"<tibble[,4]>","29":"<df[,2] [1 × 2]>","30":"<NULL>","31":"9781843342038","32":"NA","33":"<tibble[,4]>"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>It looks like it is the second item, confirming by seeing the <strong>author</strong> is Peter Suber, the <strong>publisher</strong> is MIT Press, the <strong>type</strong> is book, and the <strong>ISBN</strong> is “9780262301732”.</p>
<p>We do the following to correct it:</p>
<ul>
<li>use <code>filter()</code> with the isbn to assign the correct row from <code>suber</code> to a variable called <code>suber_correct</code></li>
<li>remove the incorrect row with <code>slice</code> (double checking that it is the 6th row)</li>
<li>use <code>bind_rows()</code> to add the correct one to our <code>my_references_works_df</code> data frame. We can just overwrite the existing <code>my_references_works_df</code> again</li>
</ul>
<pre class="r"><code>suber_correct &lt;- suber %&gt;%
    dplyr::filter(isbn == &quot;9780262301732&quot;)
my_references_works_df &lt;- my_references_works_df %&gt;%
    dplyr::slice(-6) %&gt;%
    bind_rows(suber_correct)</code></pre>
</div>
<div id="writing-publications-to-disk" class="section level2">
<h2>Writing publications to disk</h2>
<p>We will use the <code>write_csv()</code> function from the <code>readr</code> package to write our data to disk. This package was loaded when you called <code>library(tidyverse)</code> at the beginning of the session.</p>
<p>First we’re going to create a vector that represents file path to where you want to save the file:</p>
<pre class="r"><code>my_filepath &lt;- &quot;C:/EXAMPLE/crossref/data&quot;</code></pre>
<p>Unfortunately, you cannot simply write the <code>jah_jama_publications</code> data frame to a CSV, due to the nested lists. It will throw an error: <code>"Error in stream_delim_(df, path, ...) : Don't know how to handle vector of type list."</code></p>
<p>You have a few choices here:</p>
<ol style="list-style-type: decimal">
<li>You can unnest one of the columns and leave <code>.drop</code> set to <code>TRUE</code>. This will add rows for all the values in the nested lists, and drop the additional nested lists.</li>
</ol>
<pre class="r"><code>jah_jama_publications_auth &lt;- jah_jama_publications %&gt;%
    dplyr::filter(!purrr::map_lgl(author, is.null)) %&gt;%
    tidyr::unnest(author, .drop = TRUE) %&gt;%
    dplyr::bind_rows(jah_jama_publications %&gt;%
        dplyr::filter(map_lgl(author, is.null)) %&gt;%
        dplyr::select(-author, -link))
readr::write_csv(jah_jama_publications_auth, file.path(my_filepath, &quot;jah_jama_publications_auth.csv&quot;))</code></pre>
<ol start="2" style="list-style-type: decimal">
<li>You can drop the nested lists altogether using a combination of <code>select_if()</code> from <code>dplyr</code> and <code>negate()</code> from <code>purrr</code> to drop all lists in the data frame. This only works if you don’t need author or link.</li>
</ol>
<pre class="r"><code>jah_jama_publications_short &lt;- jah_jama_publications %&gt;%
    dplyr::select_if(purrr::negate(is.list))
readr::write_csv(jah_jama_publications_author, file.path(my_filepath, &quot;jah_jama_publications_short.csv&quot;))</code></pre>
<p>We go from 24 to 22 variables because it dropped <strong>author</strong> and <strong>list</strong>.</p>
<ol start="3" style="list-style-type: decimal">
<li>You can use <code>mutate()</code> from <code>dplyr</code> to coerce the list columns into character vectors:</li>
</ol>
<pre class="r"><code>jah_jama_publications_mutated &lt;- jah_jama_publications %&gt;%
    dplyr::mutate(author = as.character(author)) %&gt;%
    dplyr::mutate(link = as.character(link))
readr::write_csv(jah_jama_publications_mutated, file.path(my_filepath, &quot;jah_jama_publications_mutated.csv&quot;))</code></pre>
</div>
</div>
<div id="using-roadoi-to-check-for-open-access" class="section level1">
<h1>Using <code>roadoi</code> to check for open access</h1>
<p><code>roadoi</code> was developed by Najko Jahn, with reviews from Tuija Sonkkila and Ross Mounce. It interfaces with <a href="https://unpaywall.org">Unpaywall</a> (which used to be called oaDOI), an important tool developed by <a href="http://unpaywall.org/team">ImpactStory</a> (Heather Piwowar and Jason Priem) for locating open access versions of scholarship–read more in this <a href="https://www.nature.com/articles/d41586-018-05968-3"><em>Nature</em> article</a>. See here for <a href="https://cran.r-project.org/web/packages/roadoi/roadoi.pdf">the <code>roadoi</code> documentation</a>.</p>
<p>This incredible <a href="https://cran.r-project.org/web/packages/roadoi/vignettes/intro.html">Introduction to <code>roadoi</code></a> by Najko Jahn provides much of what you need to know to use the tool, as well as an interesting use case. Also see his recently published article <a href="https://subugoe.github.io/scholcomm_analytics/posts/unpaywall_evidence/">Open Access Evidence in Unpaywall</a>, running deep analysis on Unpaywall data.</p>
<p>First install the package and load it.</p>
<pre class="r"><code># install the roadoi package
install.packages(&quot;roadoi&quot;)</code></pre>
<pre class="r"><code># load the roadoi package
library(roadoi)</code></pre>
<div id="setting-up-roadoi" class="section level2">
<h2>Setting up <code>roadoi</code></h2>
<p>Your API calls to Unpaywall must include a valid email address where you can be reached in order to keep the service open and free for everyone.</p>
</div>
<div id="checking-oa-status-with-oadoi_fetch" class="section level2">
<h2>Checking OA status with <code>oadoi_fetch</code></h2>
<p>We then create DOI vector and use the <code>oadoi_fetch()</code> function from <code>roadoi</code>.</p>
<p><strong>Be sure to replace the email below with your own</strong></p>
<pre class="r"><code># Use c() to create a vector of DOIs
my_dois &lt;- c(&quot;10.2139/ssrn.2697412&quot;, &quot;10.1016/j.joi.2016.08.002&quot;, &quot;10.1371/journal.pone.0020961&quot;,
    &quot;10.3389/fpsyg.2018.01487&quot;, &quot;10.1038/d41586-018-00104-7&quot;, &quot;10.12688/f1000research.8460.2&quot;,
    &quot;10.7551/mitpress/9286.001.0001&quot;)

# use oadoi_fetch() to get Unpaywall data on those DOIs
my_dois_oa &lt;- roadoi::oadoi_fetch(dois = my_dois, email = &quot;clarke.iakovakis@okstate.edu&quot;)</code></pre>
<p>Look at the column names.</p>
<pre class="r"><code># print column names
names(my_dois_oa)</code></pre>
<pre><code>##  [1] &quot;doi&quot;                    &quot;best_oa_location&quot;       &quot;oa_locations&quot;          
##  [4] &quot;oa_locations_embargoed&quot; &quot;data_standard&quot;          &quot;is_oa&quot;                 
##  [7] &quot;is_paratext&quot;            &quot;genre&quot;                  &quot;oa_status&quot;             
## [10] &quot;has_repository_copy&quot;    &quot;journal_is_oa&quot;          &quot;journal_is_in_doaj&quot;    
## [13] &quot;journal_issns&quot;          &quot;journal_issn_l&quot;         &quot;journal_name&quot;          
## [16] &quot;publisher&quot;              &quot;published_date&quot;         &quot;year&quot;                  
## [19] &quot;title&quot;                  &quot;updated_resource&quot;       &quot;authors&quot;</code></pre>
<p>The returned variables are described on the <a href="http://unpaywall.org/data-format">Unpaywall Data Format</a> page.</p>
<p>We can see that Unpaywall could not find OA versions for two of the seven of these, so we will filter them out:</p>
<pre class="r"><code># use filter() to overwrite the data frame and keep only items that are
# available OA
my_dois_oa &lt;- my_dois_oa %&gt;%
    dplyr::filter(is_oa == TRUE)</code></pre>
<p>As above, it is easier to use <code>unnest()</code> to more closely view one of the variables:</p>
<pre class="r"><code># print the data frame with best open access location unnested
my_dois_oa %&gt;%
    tidyr::unnest(best_oa_location, names_repair = &quot;universal&quot;)</code></pre>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["doi"],"name":[1],"type":["chr"],"align":["left"]},{"label":["updated"],"name":[2],"type":["chr"],"align":["left"]},{"label":["url"],"name":[3],"type":["chr"],"align":["left"]},{"label":["url_for_pdf"],"name":[4],"type":["chr"],"align":["left"]},{"label":["url_for_landing_page"],"name":[5],"type":["chr"],"align":["left"]},{"label":["evidence"],"name":[6],"type":["chr"],"align":["left"]},{"label":["license"],"name":[7],"type":["chr"],"align":["left"]},{"label":["version"],"name":[8],"type":["chr"],"align":["left"]},{"label":["host_type"],"name":[9],"type":["chr"],"align":["left"]},{"label":["is_best"],"name":[10],"type":["lgl"],"align":["right"]},{"label":["pmh_id"],"name":[11],"type":["chr"],"align":["left"]},{"label":["endpoint_id"],"name":[12],"type":["chr"],"align":["left"]},{"label":["repository_institution"],"name":[13],"type":["chr"],"align":["left"]},{"label":["oa_date"],"name":[14],"type":["chr"],"align":["left"]},{"label":["oa_locations"],"name":[15],"type":["list"],"align":["right"]},{"label":["oa_locations_embargoed"],"name":[16],"type":["list"],"align":["right"]},{"label":["data_standard"],"name":[17],"type":["int"],"align":["right"]},{"label":["is_oa"],"name":[18],"type":["lgl"],"align":["right"]},{"label":["is_paratext"],"name":[19],"type":["lgl"],"align":["right"]},{"label":["genre"],"name":[20],"type":["chr"],"align":["left"]},{"label":["oa_status"],"name":[21],"type":["chr"],"align":["left"]},{"label":["has_repository_copy"],"name":[22],"type":["lgl"],"align":["right"]},{"label":["journal_is_oa"],"name":[23],"type":["lgl"],"align":["right"]},{"label":["journal_is_in_doaj"],"name":[24],"type":["lgl"],"align":["right"]},{"label":["journal_issns"],"name":[25],"type":["chr"],"align":["left"]},{"label":["journal_issn_l"],"name":[26],"type":["chr"],"align":["left"]},{"label":["journal_name"],"name":[27],"type":["chr"],"align":["left"]},{"label":["publisher"],"name":[28],"type":["chr"],"align":["left"]},{"label":["published_date"],"name":[29],"type":["chr"],"align":["left"]},{"label":["year"],"name":[30],"type":["chr"],"align":["left"]},{"label":["title"],"name":[31],"type":["chr"],"align":["left"]},{"label":["updated_resource"],"name":[32],"type":["chr"],"align":["left"]},{"label":["authors"],"name":[33],"type":["list"],"align":["right"]}],"data":[{"1":"10.2139/ssrn.2697412","2":"2020-08-01T22:09:50.925791","3":"https://zenodo.org/record/8381/files/CREATe-Working-Paper-2014-01.pdf","4":"https://zenodo.org/record/8381/files/CREATe-Working-Paper-2014-01.pdf","5":"https://zenodo.org/record/8381","6":"oa repository (via OAI-PMH title and first author match)","7":"cc-by","8":"publishedVersion","9":"repository","10":"TRUE","11":"oai:zenodo.org:8381","12":"kmlgamtfmx4w6tf78ykl","13":"CERN European Organization for Nuclear Research - Zenodo","14":"NA","15":"<tibble[,13]>","16":"<tibble[,0]>","17":"2","18":"TRUE","19":"FALSE","20":"journal-article","21":"green","22":"TRUE","23":"FALSE","24":"FALSE","25":"1556-5068","26":"1556-5068","27":"SSRN Electronic Journal","28":"Elsevier BV","29":"2014-01-01","30":"2014","31":"Open Access Publishing: A Literature Review","32":"2021-04-22T07:05:10.620822","33":"<df[,3] [1 × 3]>"},{"1":"10.1016/j.joi.2016.08.002","2":"2021-07-24T11:24:58.713076","3":"https://doi.org/10.1016/j.joi.2016.08.002","4":"NA","5":"https://doi.org/10.1016/j.joi.2016.08.002","6":"open (via crossref license)","7":"cc-by","8":"publishedVersion","9":"publisher","10":"TRUE","11":"NA","12":"NA","13":"NA","14":"2016-08-06","15":"<tibble[,13]>","16":"<tibble[,0]>","17":"2","18":"TRUE","19":"FALSE","20":"journal-article","21":"hybrid","22":"FALSE","23":"FALSE","24":"FALSE","25":"1751-1577","26":"1751-1577","27":"Journal of Informetrics","28":"Elsevier BV","29":"2016-11-01","30":"2016","31":"Hybrid open access—A longitudinal study","32":"2021-01-14T06:29:22.661456","33":"<df[,5] [2 × 5]>"},{"1":"10.1371/journal.pone.0020961","2":"2021-07-24T11:24:58.900523","3":"https://journals.plos.org/plosone/article/file?id=10.1371/journal.pone.0020961&type=printable","4":"https://journals.plos.org/plosone/article/file?id=10.1371/journal.pone.0020961&type=printable","5":"https://doi.org/10.1371/journal.pone.0020961","6":"oa journal (via doaj)","7":"cc-by","8":"publishedVersion","9":"publisher","10":"TRUE","11":"NA","12":"NA","13":"NA","14":"2011-06-13","15":"<tibble[,13]>","16":"<tibble[,0]>","17":"2","18":"TRUE","19":"FALSE","20":"journal-article","21":"gold","22":"TRUE","23":"TRUE","24":"TRUE","25":"1932-6203","26":"1932-6203","27":"PLoS ONE","28":"Public Library of Science (PLoS)","29":"2011-06-13","30":"2011","31":"The Development of Open Access Journal Publishing from 1993 to 2009","32":"2021-04-19T05:53:22.969933","33":"<df[,3] [6 × 3]>"},{"1":"10.3389/fpsyg.2018.01487","2":"2021-07-24T11:24:59.153950","3":"https://www.frontiersin.org/articles/10.3389/fpsyg.2018.01487/pdf","4":"https://www.frontiersin.org/articles/10.3389/fpsyg.2018.01487/pdf","5":"https://doi.org/10.3389/fpsyg.2018.01487","6":"oa journal (via doaj)","7":"cc-by","8":"publishedVersion","9":"publisher","10":"TRUE","11":"NA","12":"NA","13":"NA","14":"2018-08-20","15":"<tibble[,13]>","16":"<tibble[,0]>","17":"2","18":"TRUE","19":"FALSE","20":"journal-article","21":"gold","22":"TRUE","23":"TRUE","24":"TRUE","25":"1664-1078","26":"1664-1078","27":"Frontiers in Psychology","28":"Frontiers Media SA","29":"2018-08-20","30":"2018","31":"The Impact Factor Fallacy","32":"2021-07-02T07:24:21.285995","33":"<df[,3] [3 × 3]>"},{"1":"10.1038/d41586-018-00104-7","2":"2019-12-23T19:38:29.249435","3":"https://www.nature.com/magazine-assets/d41586-018-00104-7/d41586-018-00104-7.pdf","4":"https://www.nature.com/magazine-assets/d41586-018-00104-7/d41586-018-00104-7.pdf","5":"https://doi.org/10.1038/d41586-018-00104-7","6":"open (via free pdf)","7":"pd","8":"publishedVersion","9":"publisher","10":"TRUE","11":"NA","12":"NA","13":"NA","14":"2018-01-01","15":"<tibble[,13]>","16":"<tibble[,0]>","17":"2","18":"TRUE","19":"FALSE","20":"journal-article","21":"hybrid","22":"FALSE","23":"FALSE","24":"FALSE","25":"0028-0836,1476-4687","26":"0028-0836","27":"Nature","28":"Springer Science and Business Media LLC","29":"2018-01-01","30":"2018","31":"Funders should mandate open citations","32":"2021-03-29T21:00:27.983962","33":"<df[,3] [1 × 3]>"},{"1":"10.12688/f1000research.8460.2","2":"2021-07-24T11:25:00.281951","3":"https://f1000research.com/articles/5-632/v2/pdf","4":"https://f1000research.com/articles/5-632/v2/pdf","5":"https://doi.org/10.12688/f1000research.8460.2","6":"oa journal (via doaj)","7":"cc-by","8":"publishedVersion","9":"publisher","10":"TRUE","11":"NA","12":"NA","13":"NA","14":"2016-06-09","15":"<tibble[,13]>","16":"<tibble[,0]>","17":"2","18":"TRUE","19":"FALSE","20":"journal-article","21":"gold","22":"TRUE","23":"TRUE","24":"TRUE","25":"2046-1402","26":"2046-1402","27":"F1000Research","28":"F1000 Research Ltd","29":"2016-06-09","30":"2016","31":"The academic, economic and societal impacts of Open Access: an evidence-based review","32":"2021-04-21T11:42:18.537593","33":"<df[,5] [6 × 5]>"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
</div>
<div id="next-steps" class="section level1">
<h1>Next steps</h1>
<p>There are several other excellent R packages that interface with publication metadata that can be used in conjunction with this package. Examples:</p>
<ul>
<li><code>bibliometrix</code> “is an open-source tool for quantitative research in scientometrics and bibliometrics that includes all the main bibliometric methods of analysis.” See more information at <a href="https://bibliometrix.org/" class="uri">https://bibliometrix.org/</a>.</li>
<li><code>rromeo</code> is a wrapper for the SHERPA-RoMEO API. You can retrieve a set of publications metadata from <code>rcrossref</code>, then use the ISSN to look up the policies of the journal regarding the archival of preprints, postprints, and publisher versions. <a href="https://cran.r-project.org/web/packages/rromeo/rromeo.pdf" class="uri">https://cran.r-project.org/web/packages/rromeo/rromeo.pdf</a></li>
<li><code>crminer</code> “includes functions for getting getting links to full text of articles, fetching full text articles from those links or Digital Object Identifiers (‘DOIs’), and text extraction from ‘PDFs’.” <a href="https://cran.r-project.org/web/packages/crminer/crminer.pdf" class="uri">https://cran.r-project.org/web/packages/crminer/crminer.pdf</a></li>
</ul>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = false;
    options.smoothScroll = false;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
